<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<title>Sentences that contain Cytoscape</title>
</head>
<body>
<h1>Sentences that contain Cytoscape</h1>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28025995">28025995</a> Prediction of key genes and miRNAs responsible for loss of muscle force in patients during an acute exacerbation of chronic obstructive pulmonary disease.<ul><li>Additionally, key miRNAs were enriched using gene set enrichment analysis (GSEA) software and a miR-gene regulatory network was constructed using <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28003598">28003598</a> An iTRAQ-based quantitative proteomics study of refractory mycoplasma pneumoniae pneumonia patients.<ul><li>Functional classification, sub-cellular localization, and protein interaction network were carried out based on PANTHER and <b>cytoscape</b> analysis. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28000890">28000890</a> Combined analysis of gene expression, miRNA expression and DNA methylation profiles of osteosarcoma.<ul><li>Besides, miRNA-gene regulation network was obtained based on the pairs of involved DEMIs and overlapping genes between DEMs and DEGs and visualized through <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28000864">28000864</a> Identification of therapeutic targets for Parkinson's disease via bioinformatics analysis.<ul><li>Target genes of differentially expressed miRNAs were identified using the starBase database and a miRNA‑gene regulatory network was constructed using <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28000438">28000438</a> Gogadget: an R Package for Interpretation and Visualization of GO Enrichment Results.<ul><li>The filtered GO term enrichment results can also be exported by the package for subsequent analysis in <b>Cytoscape</b> Enrichment Map. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27987172">27987172</a> Exploring Plant Co-Expression and Gene-Gene Interactions with CORNET 3.0.<ul><li>The CORNET platform outputs its results in either text format or through the <b>Cytoscape</b> framework, which is automatically launched by the CORNET website.CORNET 3.0 is the third iteration of the web platform designed for the user exploration of the coexpression space of plant genomes, with a focus on the model species Arabidopsis thaliana. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27964719">27964719</a> Pathway collages: personalized multi-pathway diagrams.<ul><li>Layouts for the individual pathways are generated by the Pathway Tools software, and are sent to a Javascript Pathway Collage application implemented using <b>Cytoscape</b>.js. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27959423">27959423</a> Comparative analysis of hepatocellular carcinoma and cirrhosis gene expression profiles.<ul><li>Functional annotations were assigned to the hub genes using the Database for Annotation, Visualization and Integration Discovery, and functional annotation networks were visualized with <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27956081">27956081</a> Gene markers of fracture healing in early stage and the regulatory mechanism during the process using microarray analysis.<ul><li>Furthermore, protein-protein interaction (PPI) network was established integrating information in STRING database, and visualized by <b>Cytoscape</b> software.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27942860">27942860</a> Comprehensive gene and microRNA expression profiling reveals a role for miRNAs in the oncogenic roles of SphK1 in papillary thyroid cancer.<ul><li>miRNA-mRNA interactions were explored by microT-CDS, and the predicted networks were visualized using <b>CytoScape</b>(®). </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27924018">27924018</a> DisGeNET: a comprehensive platform integrating information on human disease-associated genes and variants.<ul><li>The information is accessible through a web interface, a <b>Cytoscape</b> App, an RDF SPARQL endpoint, scripts in several programming languages and an R package. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27924014">27924014</a> The STRING database in 2017: quality-controlled protein-protein association networks, made broadly accessible.<ul><li>In the latest version 10.5 of STRING, the biggest changes are concerned with data dissemination: the web frontend has been completely redesigned to reduce dependency on outdated browser technologies, and the database can now also be queried from inside the popular <b>Cytoscape</b> software framework. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27922680">27922680</a> Identification of key genes induced by platelet-rich plasma in human dermal papilla cells using bioinformatics methods.<ul><li>Following identification by Cuffdiff software, differentially expressed genes (DEGs) underwent enrichment analyses, and protein-protein interaction networks were constructed using <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27907142">27907142</a> IBiSA_Tools: A Computational Toolkit for Ion-Binding State Analysis in Molecular Dynamics Trajectories of Ion Channels.<ul><li>A complex network named ion-binding state graph is generated in a standard graph format (graph modeling language; GML), which can be visualized by standard network analyzers such as <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27900679">27900679</a> Bioinformatic Profiling Identifies a Glucose-Related Risk Signature for the Malignancy of Glioma and the Survival of Patients.<ul><li>Gene Set Enrichment Analysis (GSEA) and <b>Cytoscape</b> were used to explore the bioinformatic implication. <li><b>Cytoscape</b> visualized the correlation analysis among the genes. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27899966">27899966</a> Analysis of the molecular mechanism of osteosarcoma using a bioinformatics approach.<ul><li>Next, the protein-protein interaction (PPI) network was constructed and module analysis was further assessed by cluster analysis with the overlapping neighborhood expansion (Cluster ONE) <b>cytoscape</b> plug-in. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27896985">27896985</a> A NEW RELEVANCE ESTIMATOR FOR THE COMPILATION AND VISUALIZATION OF DISEASE PATTERNS AND POTENTIAL DRUG TARGETS.<ul><li><b>Cytoscape</b> was used to visualize and to analyze Mrelevance and Srelevance together with the genes and diseases. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27895852">27895852</a> Protein-protein interaction network of celiac disease.<ul><li>The networks of related differentially expressed protein were explored using <b>Cytoscape</b> 3.3 and the PPI analysis methods such as MCODE and ClueGO.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27895744">27895744</a> Identification of targets of miRNA-221 and miRNA-222 in fulvestrant-resistant breast cancer.<ul><li>Furthermore, a miRNA-messenger (m) RNA regulatory network depicting interactions between miRNA-targeted upregulated DEGs was constructed and visualized using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27895742">27895742</a> Comparative analysis of gene expression profiles of gastric cardia adenocarcinoma and gastric non-cardia adenocarcinoma.<ul><li>Subnetworks were extracted from the whole network with <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27853512">27853512</a> Contextual Hub Analysis Tool (CHAT): A <b>Cytoscape</b> app for identifying contextually relevant hubs in biological networks.<ul><li>Here, we report a <b>Cytoscape</b> app, the Contextual Hub Analysis Tool (CHAT), which enables users to easily construct and visualize a network of interactions from a gene or protein list of interest, integrate contextual information, such as gene expression or mass spectrometry data, and identify hub nodes that are more highly connected to contextual nodes (e.g. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27853512">27853512</a> Contextual Hub Analysis Tool (CHAT): A <b>Cytoscape</b> app for identifying contextually relevant hubs in biological networks.<ul><li>CHAT is available for <b>Cytoscape</b> 3.0+ and can be installed via the <b>Cytoscape</b> App Store ( http://apps.<b>cytoscape</b>.org/apps/chat).</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27853510">27853510</a> CoNet app: inference of biological association networks using <b>Cytoscape</b>.<ul><li>Here we present the <b>Cytoscape</b> app version of our association network inference tool CoNet. <li>The CoNet app supports <b>Cytoscape</b> 2.x and 3.x and offers a variety of network inference approaches, which can also be combined. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27855613">27855613</a> HOXB7 and Hsa-miR-222 as the Potential Therapeutic Candidates for Metastatic Colorectal Cancer.<ul><li>Clustering and topological analyses of the regulatory networks were performed using <b>Cytoscape</b> plug-ins.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27853520">27853520</a> dot-app: a Graphviz-<b>Cytoscape</b> conversion plug-in.<ul><li>dot-app is a <b>Cytoscape</b> 3 app that allows <b>Cytoscape</b> to import and export Graphviz (*.dot, *.gv) files, also known as DOT files due to the *.dot extension and their conformance to the DOT language syntax. <li>While DOT-based graph applications are popular, they emphasize general graph layout and styling over the topological and semantic analysis functions available in domain-focused applications such as <b>Cytoscape</b>. <li>dot-app enables the interchange of networks between <b>Cytoscape</b> and DOT-based applications so that users can benefit from the features of both. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27846294">27846294</a> Identification and Verification of Potential Therapeutic Target Genes in Berberine-Treated Zucker Diabetic Fatty Rats through Bioinformatics Analysis.<ul><li>Furthermore, a protein-protein interaction (PPI) network was constructed using STRING and <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27840954">27840954</a> Identfication of key miRNAs in pancreatitis using bioinformatics analysis of microarray data.<ul><li>Finally, pancreatitis‑associated genes among the target genes identified were searched using the Comparative Toxicogenomics Database, and a regulatory network of pancreatitis‑associated genes and their target miRNAs were constructed using <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27840927">27840927</a> Systematic analysis of mRNA expression profiles in NSCLC cell lines to screen metastasis-related genes.<ul><li>Interaction networks between the proteins encoded by the DEGs were identified using the database BioGRID and were visualized by <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27835046">27835046</a> Identifying miRNA regulatory mechanisms in preeclampsia by systems biology approaches.<ul><li>We integrated publicly available miRNA and gene expression profiles and created a network from the significant miRNA-mRNA pairs with the help of MAGIA and <b>Cytoscape</b> softwares. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27834916">27834916</a> Systematic Analysis of Protein Interaction Network Associated with Azoospermia.<ul><li>Using Online Mendelian Inheritance in Man, the Human Protein Reference Database and <b>Cytoscape</b>, we created a novel network consisting of 209 protein nodes and 737 interactions. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27831650">27831650</a> Critical genes of hepatocellular carcinoma revealed by network and module analysis of RNA-seq data.<ul><li>Modules in the PPI network were identified with MCODE method using plugin clusterViz of <b>CytoScape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27830058">27830058</a> AutoAnnotate: A <b>Cytoscape</b> app for summarizing networks with semantic annotations.<ul><li><b>Cytoscape</b> provides the ability to annotate a network with shapes and labels, however these annotations must be created manually one at a time, which can be a laborious process. <li>Annotated clusters may be collapsed into single nodes using the <b>Cytoscape</b> groups feature, which helps simplify a network by making its overall structure more visible. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27829444">27829444</a> Potential molecular characteristics in situ in response to repetitive UVB irradiation.<ul><li>Accordingly, the PPI network and regulatory networks were visualized using <b>Cytoscape</b>, and they were merged together to obtain an integrated network for mining densely connected modules.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27826930">27826930</a> Visualization and Analysis of MiRNA-Targets Interactions Networks.<ul><li>In this chapter, we describe, in a simple way, bioinformatics guidelines to use miRWalk database and <b>Cytoscape</b> software for analyzing microRNA-mRNA interactions through their visualization as a network.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27814735">27814735</a> Gene expression profiles and protein-protein interaction networks in amyotrophic lateral sclerosis patients with C9orf72 mutation.<ul><li>PPI interaction network was divided into subnetworks using MCODE algorithm and was analyzed using <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27802793">27802793</a> Identification of crucial miRNAs and the targets in renal cortex of hypertensive patients by expression profiles.<ul><li>Furthermore, protein-protein interaction (PPI) networks were constructed for up- and down-regulated genes, respectively, using the <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27797457">27797457</a> Graph theoretical analysis, insilico modeling, design, and synthesis of compounds containing benzimidazole skeleton as antidepressant agents.<ul><li>In this study, drug target was identified using KEGG database and network analysis through <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27794559">27794559</a> A <b>Cytoscape</b> app for motif enumeration with ISMAGS.<ul><li>We present a <b>Cytoscape</b> app for the ISMAGS algorithm, which can enumerate all instances of a motif in a graph, making optimal use of the motif's symmetries to make the search more efficient. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27794559">27794559</a> A <b>Cytoscape</b> app for motif enumeration with ISMAGS.<ul><li>The <b>Cytoscape</b> app for ISMAGS can be freely downloaded from the <b>Cytoscape</b> App store http://apps.<b>cytoscape</b>.org/apps/ismags Source code and documentation for ISMAGS are available at https://github.com/biointec/ismags Source code and documentation for the <b>Cytoscape</b> app are available at https://gitlab.psb.ugent.be/thpar/ISMAGS_<b>Cytoscape</b> CONTACTS: Pieter.Audenaert@intec.ugent.be or Yves.VanDePeer@psb.vib-ugent.beSupplementary information: Supplementary data are available at Bioinformatics online.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26510741">26510741</a> Altered Blood Gene Expression of Tumor-Related Genes (PRKCB, BECN1, and CDKN2A) in Alzheimer's Disease.<ul><li>The significant correlations (p &lt; 0.001) were represented in a network analysis with <b>Cytoscape</b> tool, which suggested an uncoupling of mitochondria-related genes in AD group. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27667480">27667480</a> Integrated analysis of DNA methylation and RNA‑sequencing data in Down syndrome.<ul><li>Furthermore, the interactions of the proteins encoded by selected genes were determined using the Search Tool for the Retrieval of Interacting Genes, and a protein‑protein interaction (PPI) network was constructed using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27666388">27666388</a> Transcriptional profile of SH-SY5Y human neuroblastoma cells transfected by Toxoplasma rhoptry protein 16.<ul><li>A protein‑protein interaction network was constructed using <b>Cytoscape</b> software according to the interaction associations determined using the Search Tool for the Retrieval of Interacting Genes/Proteins. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27765602">27765602</a> Spatial and temporal structure of the clinical research based on mesenchymal stromal cells: A network analysis.<ul><li>Networks were generated using <b>Cytoscape</b> 3.2.2, where each center was assimilated to a node, and one trial to an edge connecting two nodes.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27765529">27765529</a> Identification of key genes involved in nasopharyngeal carcinoma.<ul><li>Using the BiNGO plugin in <b>Cytoscape</b> and pathway enrichment analysis in the PGS package, functional and pathway enrichment analyses were performed separately to predict potential functions of the differentially expressed genes. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27757782">27757782</a> Identification of candidate biomarkers and analysis of prognostic values in ovarian cancer by integrated bioinformatics analysis.<ul><li>Protein-protein interaction (PPI) network was established by STRING and visualized by <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27208688">27208688</a> Analysis of the functional aspects and seminal plasma proteomic profile of sperm from smokers.<ul><li>The quantified proteins were used for Venn diagram construction in <b>Cytoscape</b> 3.2.1 software, using the PINA4MS plug-in. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27738870">27738870</a> Differentially Expressed Long Non-Coding RNAs Were Predicted to Be Involved in the Control of Signaling Pathways in Pediatric Astrocytoma.<ul><li>Interactomes were constructed with the HIPPIE database and visualized with the <b>Cytoscape</b> platform. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27716259">27716259</a> Microarray data analysis to identify crucial genes regulated by CEBPB in human SNB19 glioma cells.<ul><li>Furthermore, the protein-protein interaction (PPI) network and transcriptional regulatory network were constructed using <b>Cytoscape</b> software.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27711182">27711182</a> Genome-Wide Identification and Characterization of MicroRNAs and Target Genes in Lonicera japonica.<ul><li>Furthermore, the miRNA-mRNA and mRNA-mRNA regulatory networks were constructed using <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27708692">27708692</a> In silico profiling for secondary metabolites from Lepidium meyenii (maca) by the pharmacophore and ligand-shape-based joint approach.<ul><li>A compounds-targets-diseases network was established using <b>Cytoscape</b> 3.2. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27698802">27698802</a> Identification of genes involved in Epstein-Barr virus-associated nasopharyngeal carcinoma.<ul><li>With the information from the Search Tool for the Retrieval of Interacting Genes/Proteins database, the protein-protein interaction (PPI) network was visualized by <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27698798">27698798</a> Celecoxib exhibits an anti-gastric cancer effect by targeting focal adhesion and leukocyte transendothelial migration-associated genes.<ul><li>Finally, a PPI network was visualized using <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27698747">27698747</a> Co-expressed differentially expressed genes and long non-coding RNAs involved in the celecoxib treatment of gastric cancer: An RNA sequencing analysis.<ul><li>A protein-protein interaction network was constructed using STRING database and module analysis was performed using ClusterONE plugin of <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27682024">27682024</a> Bioinformatics analysis of gene expression profile data to screen key genes involved in pulmonary sarcoidosis.<ul><li>Subsequently, protein-protein interaction (PPI) pairs among the DEGs were searched by STRING or REACTOME databases, and then PPI networks were visualized by <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27667790">27667790</a> cMapper: gene-centric connectivity mapper for EBI-RDF platform.<ul><li>We developed the user interface using HTML5, JQuery and the <b>Cytoscape</b> Graph API. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27655333">27655333</a> Identification potential biomarkers in pulmonary tuberculosis and latent infection based on bioinformatics analysis.<ul><li>The protein-protein interaction (PPI) network was constructed by <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27634459">27634459</a> Systematic network assessment of the carcinogenic activities of cadmium.<ul><li>The protein-protein functional interaction networks were constructed with NetBox software and visualized with <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27621916">27621916</a> Two-Way Gene Interaction From Microarray Data Based on Correlation Methods.<ul><li>Finally, these methods were compared with <b>Cytoscape</b>, based on BIND, and Gene Ontology, based on molecular function visual methods; R software version 3.2 and Bioconductor were used to perform these methods.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27621916">27621916</a> Two-Way Gene Interaction From Microarray Data Based on Correlation Methods.<ul><li>Based on the Pearson and Spearman correlations, the results were the same and were confirmed by <b>Cytoscape</b> and GO visual methods; however, Blomqvist's coefficient was not confirmed by visual methods.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27613243">27613243</a> Crucial genes associated with diabetic nephropathy explored by microarray analysis.<ul><li>Furthermore, miRNAs and transcription factors (TFs) regulating DEGs were predicted by the GeneCoDis tool, and miRNA-DEG-TF regulatory network was visualized by <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27610375">27610375</a> Integrated Analysis of DNA Methylation and mRNA Expression Profiles Data to Identify Key Genes in Lung Adenocarcinoma.<ul><li>Then, protein-protein interaction (PPI) network was constructed using STRING and <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27602130">27602130</a> Hsa-miR-137, hsa-miR-520e and hsa-miR-590-3p perform crucial roles in Lynch syndrome.<ul><li>The regulatory network of the DEMs and their TGs was constructed using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27599581">27599581</a> Bioinformatics analysis of gene expression profiles of dermatomyositis.<ul><li>Subsequently, functional and pathway enrichment analyses were performed using ClueGO from <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27599550">27599550</a> Identification of key player genes in gene regulatory networks.<ul><li>The Java implementation of the heuristic algorithm explained in this paper is available as a <b>Cytoscape</b> plugin at http://apps.<b>cytoscape</b>.org/apps/mcds . </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27595129">27595129</a> Interaction network and mass spectrometry data of Xanthomonas citri subsp. citri surface proteins from differential proteomic analysis of infectious and non-infectious cells.<ul><li>This article contains raw data of proteins detected in the 79 spots analyzed by LC-ESI-MS/MS approach and also an enrichment analysis on the resulting protein-protein interaction network performed with the Integrated Interactome System (IIS) platform and <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27586009">27586009</a> InteGO2: a web tool for measuring and visualizing gene semantic similarities using Gene Ontology.<ul><li>Using HTML5 and <b>cytoscape</b>.js, we provide a graphical interface in InteGO2 to visualize the resulting gene functional association networks.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27581154">27581154</a> Identification of key pathways and genes in colorectal cancer using bioinformatics analysis.<ul><li>The gene ontology (GO) and Kyoto Encyclopedia of Genes and Genomes pathway (KEGG) enrichment analyses were performed, and protein-protein interaction (PPI) network of the differentially expressed genes (DEGs) was constructed by <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27559157">27559157</a> PhenomeScape: a <b>cytoscape</b> app to identify differentially regulated sub-networks using known disease associations.<ul><li>PhenomeScape is a <b>Cytoscape</b> app which provides easy access to the PhenomeExpress algorithm to interpret gene expression data. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27531214">27531214</a> Cross-species Conservation of context-specific networks.<ul><li>The CroCo web-application and <b>Cytoscape</b> plug-in are freely available at: http://services.bio.ifi.lmu.de/croco-web . </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27524402">27524402</a> Using network analysis to discern compositional patterns in ultrahigh-resolution mass spectrometry data of dissolved organic matter.<ul><li>The network of chemical transformations was calculated using the MetaNetter plug-in within <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27518462">27518462</a> Network Pharmacological Screening of Herbal Monomers that Regulate Apoptosis-Associated Genes in Acute Pancreatitis.<ul><li>Based on a database of herbal monomers and a literature search, we constructed a gene-herbal monomer regulatory network using <b>Cytoscape</b> software (San Diego, Calif), and we analyzed the relationships between apoptosis, genes, and herbal monomers.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27516065">27516065</a> Identification of conserved and novel microRNAs in Porphyridium purpureum via deep sequencing and bioinformatics.<ul><li>In addition, a <b>cytoscape</b> network will provide some clues for research into the complex biological processes that occur in this unique alga.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27503052">27503052</a> In silico modeling for tumor growth visualization.<ul><li>Within this paper we present the implementation of the Cellular Potts Model graphically presented through <b>Cytoscape</b>.js within a Web application. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27497640">27497640</a> The effects of Qi Teng Xiao Zhuo granules, traditional Chinese medicine, on the expression of genes in chronic glomerulonephritis rats.<ul><li>KEGG and Gene Ontology (GO) analyses and LIMMA, String and <b>Cytoscape</b> software were used to analyze the functional microarray data and screen differentially expressed genes. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27487266">27487266</a> Network ethnopharmacological evaluation of the immunomodulatory activity of Withania somnifera.<ul><li>The networks were constructed using <b>Cytoscape</b> 3.2.1. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27482299">27482299</a> Effect of a diet enriched with omega-6 and omega-3 fatty acids on the pig liver transcriptome.<ul><li>The biological mechanisms and pathway interaction networks were identified using DAVID and <b>Cytoscape</b> tools. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27466623">27466623</a> NCMine: Core-peripheral based functional module detection using near-clique mining.<ul><li>We implemented this method as a plugin of <b>Cytoscape</b>, which is widely used to visualize and analyze biological networks. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27466623">27466623</a> NCMine: Core-peripheral based functional module detection using near-clique mining.<ul><li>The <b>Cytoscape</b> plugin and tutorial are available at <b>Cytoscape</b> AppStore. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27456030">27456030</a> Bioinformatics exploration of PAK1 (P21-activated kinase-1) revealed potential network gene elements in breast invasive carcinoma.<ul><li>The PAK1 interaction network derived from STRING and <b>Cytoscape</b> revealed that the genes Signal-Transducer-and-Activator-of-Transcription-3 (STAT3), Cyclin-D1 (CCND1), Mitogen-activated protein kinase-1 (MAPK1), Ras-Homolog-Family-Member-A (RHOA) and Catenin-beta-1 have high degrees of interaction where CCND1, MAPK1 and RHOA have direct interaction with PAK1. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27452039">27452039</a> Mechanisms of CCl4-induced liver fibrosis with combined transcriptomic and proteomic analysis.<ul><li>Network of protein-protein interaction (PPI) and key function with their related targets were performed and the degree of network was calculated with <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27450445">27450445</a> Gene expression profile after activation of RIG-I in 5'ppp-dsRNA challenged DF1.<ul><li>Based on the shared genes among different pathways, a network representing crosstalk between RIG-I and other signaling pathways was constructed using <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27446427">27446427</a> Identification of key target genes and pathways in laryngeal carcinoma.<ul><li>Furthermore, the protein-protein interaction (PPI) network of DEGs was constructed using <b>Cytoscape</b> software and modules were analyzed using MCODE plugin from the PPI network. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27446297">27446297</a> Investigation of the molecular mechanisms underlying metastasis in prostate cancer by gene expression profiling.<ul><li>Subsequently, microRNAs (miRNAs) potentially involved in the regulation of PCa metastasis were identified by WebGestalt software, and the miRNA-DEG regulatory network was visualized using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27442190">27442190</a> Gene Expression Profile of Extracellular Matrix and Adhesion Molecules in the Human Normal Corneal Stroma.<ul><li>Function association network was generated via the web tools String and <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27423718">27423718</a> Comparison of Pathogenic Mechanisms Underlying Single and Recurrent Venous Thromboembolism Based on Gene Expression Profiling.<ul><li>Based on the Search Tool for the Retrieval of Interacting Genes/Proteins database, protein-protein interaction network was visualized by <b>Cytoscape</b>, and modules were identified by CFinder. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27420970">27420970</a> Identifying significant pathways of hepatitis B virus-related hepatocellular carcinoma based on crosstalk and network pathways.<ul><li>Finally, pathways and crosstalk were visualized via <b>Cytoscape</b> to construct the final pathway network. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27414486">27414486</a> A five-gene expression signature to predict progression in T1G3 bladder cancer.<ul><li>Interaction networks between the genes of the model were built by GeneMANIA <b>Cytoscape</b> plugin.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27405725">27405725</a> Identification of candidate genes for myeloma-induced osteocyte death based on microarray data.<ul><li>Additionally, protein-protein interaction (PPI) network and sub-network analyses were performed using STRING database and <b>Cytoscape</b> software.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27402693">27402693</a> PetriScape - A plugin for discrete Petri net simulations in <b>Cytoscape</b>.<ul><li><b>Cytoscape</b> is the standard bioinformatics tool offering the community an extensible platform for computational analysis of the emerging cellular network together with experimental omics data sets. <li>Many approaches of varying complexity exist but none of them have been integrated into <b>Cytoscape</b> as app/plugin yet. <li>PetriScape comes with the following main functionalities: (1) import of biological networks in SBML format, (2) conversion into a Petri net, (3) visualization as Petri net, and (4) simulation and visualization of the token flow in <b>Cytoscape</b>. <li>It allows a straightforward Petri net model creation, simulation and visualization with <b>Cytoscape</b>, providing clues about the activity of key components in biological networks.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27396555">27396555</a> Bioinformatic Analysis of Genes and MicroRNAs Associated With Atrioventricular Septal Defect in Down Syndrome Patients.<ul><li>Next, miRNAs that targeted DEGs were predicted based on Webgestalt (criteria: P &lt; 0.05 and target DEGs ≥ 2), and miRNA-DEG regulatory networks were visualized through <b>Cytoscape</b>.A total of 179 DEGs were identified. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27356888">27356888</a> Screening of gene signatures for rheumatoid arthritis and osteoarthritis based on bioinformatics analysis.<ul><li>A protein‑protein interaction network was constructed and the modules were further analyzed using the Molecular Complex Detection plugin of <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27356390">27356390</a> [Mechanism Study on Danggui Shaoyao San and Guizhi Fuling Wan For Treating Primary Dysmenorrheal Based on Biological Network].<ul><li>Then, the &quot;formula-syndromes-disease&quot; network was constructed and analyzed with <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27354160">27354160</a> BioNSi: A Discrete Biological Network Simulator Tool.<ul><li>The Biological Network Simulator (BioNSi) is a tool for modeling biological networks and simulating their discrete-time dynamics, implemented as a <b>Cytoscape</b> App. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27347378">27347378</a> CyLineUp: A <b>Cytoscape</b> app for visualizing data in network small multiples.<ul><li>CyLineUp is a <b>Cytoscape</b> 3 app for the projection of high-throughput measurement data from multiple experiments/samples on a network or pathway map using &quot;small multiples&quot;. <li>The user can import various kinds of measurement data and select any appropriate <b>Cytoscape</b> network or WikiPathways pathway map. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27347171">27347171</a> Investigation of potential molecular biomarkers and small molecule drugs for hepatocellular carcinoma transformed from cirrhosis.<ul><li>Differential co-expression network (DEN) analysis was conducted and the network was visualized using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27347159">27347159</a> Gene expression profiling analysis of ovarian cancer.<ul><li>The interaction associations of the proteins encoded by the DEGs were searched using the Search Tool for the Retrieval of Interacting Genes, and the protein-protein interaction (PPI) network was visualized by <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27347057">27347057</a> Regulatory mechanisms underlying sepsis progression in patients with tumor necrosis factor-α genetic variations.<ul><li>An integrated miRNA-DEG network was constructed using the <b>Cytoscape</b> software, based on the interactions between the DEGs, as identified using the Search Tool for the Retrieval of Interacting Genes/Proteins database, and the correlation between miRNAs and their target genes. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27331400">27331400</a> An Integrative Transcriptomic Analysis for Identifying Novel Target Genes Corresponding to Severity Spectrum in Spinal Muscular Atrophy.<ul><li>The transcriptomic analysis was conducted through combining weighted correlation network analysis (WGCNA) for gene module detection, gene set enrichment analysis (GSEA) for functional categorization and filtration, and <b>Cytoscape</b> (visual interaction gene network analysis) for target gene identification. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27323086">27323086</a> Construction of a protein-protein interaction network of Wilms' tumor and pathway prediction of molecular complexes.<ul><li>We screened the confirmed disease-related genes using the Online Mendelian Inheritance in Man database, created a protein-protein interaction network based on biological function in the <b>Cytoscape</b> software, and detected molecular complexes and relevant pathways that may be included in the network. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27313588">27313588</a> Comparative Proteomic Analysis of the Response of Maize (Zea mays L.) Leaves to Long Photoperiod Condition.<ul><li>The enriched gene ontology categories among the identified proteins were identified statistically with the <b>Cytoscape</b> plugin ClueGO + Cluepedia. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27294188">27294188</a> System Review about Function Role of ESCC Driver Gene KDM6A by Network Biology Approach.<ul><li>We constructed a PPI network with <b>Cytoscape</b> software and performed clustering of network with Clust&amp;See. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27284354">27284354</a> Identification of nondiabetic heart failure-associated genes by bioinformatics approaches in patients with dilated ischemic cardiomyopathy.<ul><li>In addition, the protein-protein interaction (PPI) network of the upregulated and downregulated genes was constructed using the Search Tool for the Retrieval of Interacting Genes/Proteins database and the <b>Cytoscape</b> software platform. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27284300">27284300</a> Differential co-expression analysis of venous thromboembolism based on gene expression profile data.<ul><li>Differential co-expression analysis was performed using R, followed by screening of modules using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27278879">27278879</a> Exosome-derived microRNAs contribute to prostate cancer chemoresistance.<ul><li>To construct potential miRNA regulatory networks, the overlapping parts of miRNAs and their targer genes from the selected KEGG pathway 'PCa progression (hsa05215)' were visualized by <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27262796">27262796</a> Mapping of protein-protein interaction network of Alexander disease.<ul><li>Through system biology approach, the undirected human protein interaction network of ALXD is buildup with the help of <b>Cytoscape</b> tool and its various plugins helps to investigate network further. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27247549">27247549</a> Identification of differentially expressed genes associated with the enhancement of X-ray susceptibility by RITA in a hypopharyngeal squamous cell carcinoma cell line (FaDu).<ul><li>Afterward, the protein-protein interaction (PPI) relationships were obtained from the STRING database and a PPI network was constructed using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27222482">27222482</a> Screening and functional analysis of differentially expressed genes in chronic glomerulonephritis by whole genome microarray.<ul><li>KEGG, Gene Ontology (GO) analyze, LIMMA, String and <b>Cytoscape</b> software were applied to screen and analyze differentially regulated genes. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27222482">27222482</a> Screening and functional analysis of differentially expressed genes in chronic glomerulonephritis by whole genome microarray.<ul><li>According to the results of Generank, String and <b>Cytoscape</b> analyses, 27 genes may be key controlled genes in the pathogenesis of CGN, including 14 up-regulated genes (Fos, Myc, Kng1, Rac2, Pik3r1, Egr1, Icam1, Syk, Anxa1, Lgals3, Ptprc, Runx1, Itgb7, Ccl6) and 13 down-regulated genes (Aldh2, Dpyd, Mthfd1, Gldc, Ppar-α, Igf1, Pomc, Oas1a, Gsr, Acox1, Cyp1a1, Ugt2b15, Hsd3b6), which primarily contribute to biological processes such as, cell cycle, cell proliferation, inflammatory response, immune response, metabolic process, and so on. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27203685">27203685</a> Reactome from a WikiPathways Perspective.<ul><li>The research community at large benefits from the availability of a larger set of pathways for analysis in PathVisio and <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27198219">27198219</a> The RING 2.0 web server for high quality residue interaction networks.<ul><li>The network can be visualized directly in the browser or in <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27193824">27193824</a> Identification and Characterization of miRNAs in Chondrus crispus by High-Throughput Sequencing and Bioinformatics Analysis.<ul><li>crispus by constructing a <b>Cytoscape</b> network. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27184904">27184904</a> Exploring mechanisms of Panax notoginseng saponins in treating coronary heart disease by integrating gene interaction network and functional enrichment analysis.<ul><li>Gene-gene interaction networks of CHD and PNS were built by the GeneMANIA <b>Cytoscape</b> plugin. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27180817">27180817</a> Global quantitative proteomics reveal up-regulation of endoplasmic reticulum stress response proteins upon depletion of eIF5A in HeLa cells.<ul><li>Heatmap analysis and <b>Cytoscape</b> visualization of biological networks identified protein folding as the major cellular process affected by depletion of eIF5A. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27169281">27169281</a> [An exploration in the action targets for antidepressant bioactive components of Xiaoyaosan based on network pharmacology].<ul><li>The <b>Cytoscape</b> software was used to construct the Xiaoyaosan ingredients-targets-pathways network. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27153624">27153624</a> DyNet: visualization and analysis of dynamic molecular interaction networks.<ul><li>Here, we present DyNet, a <b>Cytoscape</b> application that provides a range of functionalities for the visualization, real-time synchronization and analysis of large multi-state dynamic molecular interaction networks enabling users to quickly identify and analyze the most 'rewired' nodes across many network states.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27153624">27153624</a> DyNet: visualization and analysis of dynamic molecular interaction networks.<ul><li>DyNet is available at the <b>Cytoscape</b> (3.2+) App Store (http://apps.<b>cytoscape</b>.org/apps/dynet).</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27153602">27153602</a> Quantitative phosphoproteomics-based molecular network description for high-resolution kinase-substrate interactome analysis.<ul><li>In this study, we developed a <b>Cytoscape</b>-based bioinformatical platform named 'Post Translational Modification mapper (PTMapper)' to integrate PPI data with publicly available kinase-substrate relations at the resolution of phosphorylated amino acid residues. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27148389">27148389</a> How Hepatitis C Virus Leads to Hepatocellular Carcinoma: A Network-Based Study.<ul><li>The topological analysis and network visualization was carried outperformed using <b>Cytoscape</b> software and the top hub genes were identified. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27131904">27131904</a> Transcriptomic gene-network analysis of exposure to silver nanoparticle reveals potentially neurodegenerative progression in mouse brain neural cells.<ul><li>The differentially expressed genes in mouse brain neural cells exposure to AgNPs were further identified using Phalanx Mouse OneArray® chip, and permitted to explore the gene network pathway regulating in neurodegenerative progression according to <b>Cytoscape</b> analysis. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27131791">27131791</a> APID interactomes: providing proteome-based interactomes with controlled quality for multiple species and derived networks.<ul><li>APID also includes a new graph tool (based on <b>Cytoscape</b>.js) for visualization and interactive analyses of PPI networks. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27123054">27123054</a> Identification of potential therapeutic target genes and mechanisms in head and neck squamous cell carcinoma by bioinformatics analysis.<ul><li>Furthermore, the protein-protein interaction (PPI) network of these DEGs was constructed using <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27098040">27098040</a> GAM: a web-service for integrated transcriptional and metabolic network analysis.<ul><li>The subnetworks are then visualized online and also can be downloaded in <b>Cytoscape</b> format for subsequent processing. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27086506">27086506</a> HiQuant: Rapid Postquantification Analysis of Large-Scale MS-Generated Proteomics Data.<ul><li>HiQuant also enables the interpretation of results generated from large-scale data sets by supporting interactive heatmap analysis and also the direct export to <b>Cytoscape</b> and Gephi, two leading network analysis platforms. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27082053">27082053</a> Identification of microRNA-mRNA interactions in atrial fibrillation using microarray expression profiles and bioinformatics analysis.<ul><li>Subsequently, differentially expressed miRNAs and DEGs were searched for in the miRecords database and TarBase, and were used to construct a regulatory network using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27079097">27079097</a> [Protein Interaction Network Construction and Biological Pathway Analysis Related to Atherosclerosis].<ul><li>The interactional data were imported into the software <b>Cytoscape</b> to construct the interaction network, and then the degree characteristics of the network were analyzed for Hub proteins. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27059647">27059647</a> CyTRANSFINDER: a <b>Cytoscape</b> 3.3 plugin for three-component (TF, gene, miRNA) signal transduction pathway construction.<ul><li>We have developed CyTRANSFINDER, a new <b>Cytoscape</b> plugin able to infer three-component signal transduction pathways based on user defined regulatory patterns and including miRNAs, TFs and genes. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27056547">27056547</a> Oral squamous cell carcinoma: microRNA expression profiling and integrative analyses for elucidation of tumourigenesis mechanism.<ul><li>A computational pipeline with sequential integration of data from miRTarBase, <b>CytoScape</b>, UniProtKB and DIANA-miRPath was utilized to map the target genes of deregulated miRNAs and associated molecular pathways.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27052252">27052252</a> Bioinformatic exploration of MTA1-regulated gene networks in colon cancer.<ul><li>GO and KEGG pathway analyses were conducted with DAVID and gene network analysis via STRING and <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27045593">27045593</a> GUIdock: Using Docker Containers with a Common Graphics User Interface to Address the Reproducibility of Research.<ul><li>Our package also includes <b>Cytoscape</b>, a java-based platform with a graphical user interface for visualizing and analyzing gene networks, and the CyNetworkBMA app, a <b>Cytoscape</b> app that allows the use of networkBMA via the user-friendly <b>Cytoscape</b> interface.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27010134">27010134</a> Identification potential biomarkers and therapeutic agents in multiple myeloma based on bioinformatics analysis.<ul><li>Furthermore, co-expression network was constructed by <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27001301">27001301</a> Abscisic acid transcriptomic signaling varies with grapevine organ.<ul><li>The biochemical pathways affected by ABA were identified using the <b>Cytoscape</b> program with the BiNGO plug-in software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27001127">27001127</a> The proteome profile of embryogenic cell suspensions of Coffea arabica L.<ul><li>We present their known GO annotations and show two protein networks sharing the GO annotations related to stress and embryogenic capacity via the free program <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26998004">26998004</a> Expression profile analysis based on DNA microarray for patients undergoing off-pump coronary artery bypass surgery.<ul><li><b>Cytoscape</b> was applied to construct miRNA regulatory networks of DEGs. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26988373">26988373</a> Exploratory Analysis of Biological Networks through Visualization, Clustering, and Functional Annotation in <b>Cytoscape</b>.<ul><li>The visualization procedure described here relies on the open-source network visualization software <b>Cytoscape</b> and includes detailed instructions on formatting and loading the data, clustering networks, and overlaying functional annotations.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26985959">26985959</a> E2F, HSF2, and miR-26 in thyroid carcinoma: bioinformatic analysis of RNA-sequencing data.<ul><li>A co-expression network was constructed using the EBcoexpress package and visualized by <b>Cytoscape</b>, and functional and pathway enrichment of DEGs in the co-expression network was analyzed with DAVID and KOBAS 2.0. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26968890">26968890</a> Study of pathway cross-talk interactions with NF-κB leading to its activation via ubiquitination or phosphorylation: A brief review.<ul><li>For this <b>Cytoscape</b> and its various plugins (Cytocopter, Allegro, AgilentLitSearch and Styles) are employed. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26968041">26968041</a> Identifying candidate genes involved in osteoarthritis through bioinformatics analysis.<ul><li>Moreover, perturbation and pathway enrichment analyses were performed through PerturbationAnalyzer in <b>Cytoscape</b> (iterative criteria &lt;1×e-10) and clusterProfiler package (FDR &lt;0.05), respectively.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26960328">26960328</a> IL-22 Impedes the Proliferation of Schwann cells: Transcriptome Sequencing and Bioinformatics Analysis.<ul><li>Functional and pathway enrichment analyses were performed (tool: GOFunction), and protein-protein interaction (PPI) network was constructed (tool: STRING and <b>Cytoscape</b>). </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26957268">26957268</a> Identification of key genes involved in HER2-positive breast cancer.<ul><li>Then, protein-protein interaction (PPI) network of the DEGs was constructed using <b>Cytoscape</b> and modules of the PPI network were screened by CFinder. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26951766">26951766</a> Quantitative proteomics study of the neuroprotective effects of B12 on hydrogen peroxide-induced apoptosis in SH-SY5Y cells.<ul><li>Bioinformatics analysis using the <b>Cytoscape</b> platform indicated that poly pyrimidine tract binding protein 1 (PTBP1) was highly associated with the protective effect, and western blotting verified that PTBP1 was up-regulated in H2O2 + B12 treatment group, compared with the H2O2 treated group. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26937461">26937461</a> Comprehensive data analysis of human ureter proteome.<ul><li>In addition, <b>Cytoscape</b> GO annotation was examined on the final ureter dataset to better understand proteins molecular function, biological processes, and cellular component. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26925814">26925814</a> Prediction of human protein-protein interaction by a domain-based approach.<ul><li>It provides an opportunity to visualize the PPIs by using network graphs, which were constructed with <b>Cytoscape</b>, so that we can indicate underlying pathways possible.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26914116">26914116</a> Expression profile analysis of new candidate genes for the therapy of primary osteoporosis.<ul><li>The protein-protein interaction (PPI) network was constructed by STRING and <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26909922">26909922</a> Construction of polycythemia vera protein interaction network and prediction of related biological functions.<ul><li>Here, polycythemia vera (PV)-related genes were screened by the Online Mendelian Inheritance in Man (OMIM), and literature pertaining to the identified genes was extracted and a protein-protein interaction network was constructed using various <b>Cytoscape</b> plugins. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26907161">26907161</a> Structures of Trypanosome Vacuolar Soluble Pyrophosphatases: Antiparasitic Drug Targets.<ul><li>On the basis of <b>Cytoscape</b> and other bioinformatics analyses, it is apparent that similar folds will be found in most if not all trypanosomatid VSP1s, including those found in insects (Angomonas deanei, Strigomonas culicis), plant pathogens (Phytomonas spp.), and Leishmania spp. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26881536">26881536</a> Construction and analysis of the regulatory network disturbed by the silenced Sp1 transcription factor in HeLa cells.<ul><li>Combining the DEGs with the data from the microRNA (miRNA) databases and transcription factor databases, an integrated regulatory network was established with <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26870752">26870752</a> Proteomic data on the nuclear interactome of human MCM9.<ul><li>A set of 22 interacting proteins identified from the proteomic data was used to create an MCM9-centered interactive network diagram, using the <b>Cytoscape</b> program. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26870242">26870242</a> Identification of key genes associated with gastric cancer based on DNA microarray data.<ul><li>The gene co-expression network was constructed based on the DEGs using <b>Cytoscape</b> software, and modules in the network were analyzed by ClusterOne and Bingo. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26870222">26870222</a> Signal transduction by M3 muscarinic acetylcholine receptor in prostate cancer.<ul><li>Finally, the PPI network and the miRNA-target DEG network were integrated using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26870179">26870179</a> Gene expression profiling analysis of MENX-associated rat pituitary adenomas contributes to understand molecular mechanisms of human pituitary adenomas.<ul><li>Furthermore, the interaction associations of the up- and downregulated DEGs obtained from the Search Tool for the Retrieval of Interacting Genes database were respectively revealed by the protein-protein interaction networks visualized with <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26868022">26868022</a> Identification of Molecular Targets for Predicting Colon Adenocarcinoma.<ul><li>A regulation network of miRNA-gene was constructed and analyzed by <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26867676">26867676</a> Exposure of CCRF-CEM cells to acridone derivative 8a triggers tumor death via multiple mechanisms.<ul><li>Data analyses based on PLGS, STRING, <b>Cytoscape</b>, and database for annotation, visualization, and integrated discovery identified 55 proteins that were differentially expressed in response to 8a exposure. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26858982">26858982</a> Data in support of FSH induction of IRS-2 in human granulosa cells: Mapping the transcription factor binding sites in human IRS-2 promoter.<ul><li>Further analysis was carried out for binary interactions of TF regulatory genes in IRS-2 network using <b>Cytoscape</b> software tool and R-code. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26850698">26850698</a> Human urinary exosome proteome unveils its aerobic respiratory ability.<ul><li>By <b>cytoscape</b> analysis, the biological process gene ontology was correlated to their probability (P ≤ 0.05) to be functional. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26846964">26846964</a> CABeRNET: a <b>Cytoscape</b> app for augmented Boolean models of gene regulatory NETworks.<ul><li>We here introduce CABERNET, a <b>Cytoscape</b> app for the generation, simulation and analysis of Boolean models of GRNs, specifically focused on their augmentation when a only partial topological and functional characterization of the network is available. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26846964">26846964</a> CABeRNET: a <b>Cytoscape</b> app for augmented Boolean models of gene regulatory NETworks.<ul><li>The integration within the widely used <b>Cytoscape</b> framework for the visualization and analysis of biological networks, makes CABERNET a new essential instrument for both the bioinformatician and the computational biologist, as well as a computational support for the experimentalist. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26845596">26845596</a> Gene expression profiling reveals key genes and pathways related to the development of non-alcoholic fatty liver disease.<ul><li>Furthermore, protein-protein interaction (PPI) networks were constructed based on the PPI information from HPRD database, and then, networks were visualized through <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26840709">26840709</a> Gene expression profiling analysis of lung adenocarcinoma.<ul><li>Potential gene functions were predicted using functional and pathway enrichment analysis, and protein-protein interaction (PPI) networks obtained from the STRING database were constructed with <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26834985">26834985</a> iCTNet2: integrating heterogeneous biological interactions to understand complex traits.<ul><li>iCTNet (integrated Complex Traits Networks) version 2 is a <b>Cytoscape</b> app and database that allows researchers to build heterogeneous networks by integrating a variety of biological interactions, thus offering a systems-level view of human complex traits. <li>iCTNet2 is freely available at the <b>Cytoscape</b> app store.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26834984">26834984</a> PSFC: a Pathway Signal Flow Calculator App for <b>Cytoscape</b>.<ul><li>Here we present Pathway Signal Flow Calculator (PSFC), a <b>Cytoscape</b> app for calculation of a pathway signal flow based on the pathway topology and node input data. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26812286">26812286</a> Bioinformatics analyses combined microarray identify the differentially expressed microRNAs in periventricular leukomalacia.<ul><li>Furthermore, GO analysis and biological process interaction network was constructed using the BiNGO and <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26804550">26804550</a> Molecular mechanisms associated with breast cancer based on integrated gene expression profiling by bioinformatics analysis.<ul><li>In addition, co-expression network of DEGs was established using the <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26795696">26795696</a> Gene expression profiling analysis contributes to understanding the association between non-syndromic cleft lip and palate, and cancer.<ul><li>Subsequently, cancer genes were obtained from six cancer databases, and the cancer‑associated protein‑protein interaction network for the DEGs was visualized using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26793622">26793622</a> A Systems Biology Approach to Reveal Putative Host-Derived Biomarkers of Periodontitis by Network Topology Characterization of MMP-REDOX/NO and Apoptosis Integrated Pathways.<ul><li>Second, we determined the centrality values (closeness, stress, degree or connectivity, and betweenness) for the &quot;BIOMARK&quot; members by using the <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26784652">26784652</a> Non-canonical pathway network modelling and ubiquitination site prediction through homology modelling of NF-κB.<ul><li>For this we employed the <b>Cytoscape</b> and STRING database. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26782791">26782791</a> Identification of candidate target genes of pituitary adenomas based on the DNA microarray.<ul><li>Subsequently, the protein‑protein interaction (PPI) network was constructed using <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26781349">26781349</a> Identification of miRNAs and differentially expressed genes in early phase non-small cell lung cancer.<ul><li>The GeneCoDis3 database and <b>Cytoscape</b> software were used to construct the transcriptional regulatory network. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26779228">26779228</a> CoExpNetViz: Comparative Co-Expression Networks Construction and Visualization Tool.<ul><li>The CoExpNetViz algorithm proceeds in three main steps; (i) for every bait gene submitted, co-expression values are calculated using mutual information and Pearson correlation coefficients, (ii) non-bait (or target) genes are grouped based on cross-species orthology, and (iii) output files are generated and results can be visualized as network graphs in <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26779228">26779228</a> CoExpNetViz: Comparative Co-Expression Networks Construction and Visualization Tool.<ul><li>The CoExpNetViz tool is freely available both as a PHP web server (link: http://bioinformatics.psb.ugent.be/webtools/coexpr/) (implemented in C++) and as a <b>Cytoscape</b> plugin (implemented in Java). </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26770395">26770395</a> Screening feature genes of astrocytoma using a combined method of microarray gene expression profiling and bioinformatics analysis.<ul><li>Finally, corresponding protein-protein interaction (PPI) networks of DEGs was constructed using the <b>Cytoscape</b> based on the data collected from STRING online datasets. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26765468">26765468</a> Screening for the Key lncRNA Targets Associated With Metastasis of Renal Clear Cell Carcinoma.<ul><li>Construction of the co-expression network was accomplished using <b>Cytoscape</b>.A total of 3862 lncRNAs and 2935 mRNAs were deregulated in RCCC tissues, compared with paired NT tissues. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26737782">26737782</a> An approach for optimally extending mathematical models of signaling networks using omics data.<ul><li>Mathematical modeling is a key process in Systems Biology and the use of computational tools such as <b>Cytoscape</b> for omics data processing, need to be integrated in the modeling activity. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26732727">26732727</a> Seasonal proteome changes of nasal mucus reflect perennial inflammatory response and reduced defence mechanisms and plasticity in allergic rhinitis.<ul><li>Gene enrichment analysis was performed with <b>Cytoscape</b>/BINGO software.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26729218">26729218</a> Diffany: an ontology-driven framework to infer, visualise and analyse differential molecular networks.<ul><li>Diffany is freely available as open-source java library and <b>Cytoscape</b> plugin from http://bioinformatics.psb.ugent.be/supplementary_data/solan/diffany/.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26719042">26719042</a> Microarray-based gene expression profiling reveals genes and pathways involved in the oncogenic function of REG3A on pancreatic cancer cells.<ul><li>Ultimately, we constructed protein-protein interaction networks among the DEGs by <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26717154">26717154</a> Identification and characterization of microRNAs in Eucheuma denticulatum by high-throughput sequencing and bioinformatics analysis.<ul><li>A <b>Cytoscape</b> network was constructed to explore the interrelationships of miRNAs, miRNA-target genes and target genes. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26714915">26714915</a> [Screening genes related with leukocyte responses early after burn injury: analysis of differentially gene expression profiling data in mice].<ul><li>Biological annotation of the sub-networks was executed using the software <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26713275">26713275</a> Prediction of the anti-inflammatory mechanisms of curcumin by module-based protein interaction network analysis.<ul><li>The PIN of curcumin was constructed by <b>Cytoscape</b> and the function modules identified by gene ontology (GO) enrichment analysis based on molecular complex detection (MCODE). </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26709677">26709677</a> Tumor Suppressor Role of miR-363-3p in Gastric Cancer.<ul><li>MATERIAL AND METHODS The network of NOTCH1 and the involved molecules was constructed by use of <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26691584">26691584</a> In silico target fishing and pharmacological profiling for the isoquinoline alkaloids of Macleaya cordata (Bo Luo Hui).<ul><li>cordata was constructed and analyzed by <b>Cytoscape</b> 3.0.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26691002">26691002</a> Actions of Huangqi decoction against rat liver fibrosis: a gene expression profiling analysis.<ul><li>Pathway-gene and protein-protein interaction (PPI) networks were constructed with <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26674271">26674271</a> SLiMScape 3.x: a <b>Cytoscape</b> 3 app for discovery of Short Linear Motifs in protein interaction networks.<ul><li><b>Cytoscape</b> provides a user-friendly, interactive environment to explore interaction networks and select proteins based on common features, such as shared interaction partners. SLiMScape embeds tools of the SLiMSuite package for de novo SLiM discovery (SLiMFinder and QSLiMFinder) and identifying occurrences/enrichment of known SLiMs (SLiMProb) within this interactive framework. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26672762">26672762</a> CyREST: Turbocharging <b>Cytoscape</b> Access for External Tools via a RESTful API.<ul><li><b>Cytoscape</b> is a critical workflow component for executing network visualization, analysis, and publishing tasks, but it can be operated only manually via a point-and-click user interface. <li>In this paper, we present the new cyREST <b>Cytoscape</b> app and accompanying harmonization libraries. <li>We describe cyREST's API and overall construction, and present Python- and R-based examples that illustrate how <b>Cytoscape</b> can be integrated into large scale data analysis pipelines. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26672347">26672347</a> [Simulation and Prediction of Material Foundation of Rhei Radix et Rhizoma for Ischemic Stroke Based on Molecular Docking Technology].<ul><li>Meanwhile, multi-component protein target network was established by software <b>Cytoscape</b> 2. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26669930">26669930</a> CytoGEDEVO-global alignment of biological networks with <b>Cytoscape</b>.<ul><li>We have created CytoGEDEVO, a <b>Cytoscape</b> app for visual and user-assisted NA. <li>Our main focus was on the usability, even by non-programmers and the interpretability of the NA results with <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26662404">26662404</a> Construction of gene/protein interaction networks for primary myelofibrosis and KEGG pathway-enrichment analysis of molecular compounds.<ul><li>A gene/protein interaction network for primary myelofibrosis was obtained through <b>Cytoscape</b> with the literature mining performed using the Agilent Literature Search plugin. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26650608">26650608</a> Aberrant expression of long noncoding RNAs in cumulus cells isolated from PCOS patients.<ul><li>Furthermore, a lncRNA/mRNA co-expression network was constructed by using the <b>Cytoscape</b> software (V2.8.3, http://www.<b>cytoscape</b>.org/ ).</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26641843">26641843</a> Bioinformatics Analysis of microRNA Time-Course Expression in Brown Rat (Rattus norvegicus): Spinal Cord Injury Self-Repair.<ul><li>Furthermore, STRING database and <b>Cytoscape</b> software were used to construct interaction networks between time-course DEmiRNAs, and GenCLip2.0 software was applied to pathway enrichment for key genes associated with nervous system.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26636516">26636516</a> Expression data analysis to identify key target genes in visceral fat tissue associated with obstructive sleep apnea.<ul><li>Furthermore, a protein-protein interaction (PPI) network was constructed with STRING database and visualized with <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26600477">26600477</a> Analysis of the gene-protein interaction network in glioma.<ul><li>A literature search was performed using the Agilent Literature Search plugin, and <b>Cytoscape</b> was used to establish a protein-protein interaction network. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26594341">26594341</a> NetMatchStar: an enhanced <b>Cytoscape</b> network querying app.<ul><li>We present NetMatchStar, a <b>Cytoscape</b> app to find all the occurrences of a query graph in a network and check for its significance as a motif with respect to seven different random models. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26592840">26592840</a> Cytochrome c oxidase subunit VIIb as a potential target in familial hypercholesterolemia by bioinformatical analysis.<ul><li>Besides, the modules of network were analyzed with <b>Cytoscape</b> and screened out with Mcode plugin, and the functional annotation of the genes involved in the modules was analyzed with BiNGO (Biological Networks Gene Ontology).</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26566394">26566394</a> CyNetworkBMA: a <b>Cytoscape</b> app for inferring gene regulatory networks.<ul><li>We have developed a tool called CyNetworkBMA for inferring gene networks from expression data that integrates with <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26566394">26566394</a> CyNetworkBMA: a <b>Cytoscape</b> app for inferring gene regulatory networks.<ul><li>CyNetworkBMA is an easy-to-use tool that makes network inference accessible to non-programmers through seamless integration with <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26558755">26558755</a> Gene Network Rewiring to Study Melanoma Stage Progression and Elements Essential for Driving Melanoma.<ul><li>The web-based database resource consists of results from the analysis presented here, integrated with <b>cytoscape</b> web and user-friendly tools for visualization, retrieval and further analysis.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26525978">26525978</a> A bioinformatic survey of RNA-binding proteins in Plasmodium.<ul><li>Finally, <b>Cytoscape</b> was used to create protein-protein interaction network for CITH-Dozi and Caf1-CCR4-Not complexes.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26523131">26523131</a> The Construction of Regulatory Network for Insulin-Mediated Genes by Integrating Methods Based on Transcription Factor Binding Motifs and Gene Expression Variations.<ul><li>Based on a motif discovery method implemented by iRegulon, a <b>Cytoscape</b> app, we identified 25 candidate regulons, motifs of which were enriched among the promoters of 478 up-regulated genes and 82 down-regulated genes. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26518718">26518718</a> Identification of gene markers in the development of smoking-induced lung cancer.<ul><li>And protein-protein interaction (PPI) network was constructed using <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26502853">26502853</a> Identifying the biomarkers of multiple sclerosis based on non-coding RNA signature.<ul><li>Additionally, the integrated regulatory network of DEGs, DE-miRNAs and targets was constructed using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26498853">26498853</a> Identification of candidate target genes for human peripheral arterial disease using weighted gene co‑expression network analysis.<ul><li>In addition, the protein‑protein interaction network for the DEGs in the most significant module was constructed using <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26498776">26498776</a> Identification of differentially expressed genes associated with burn sepsis using microarray.<ul><li>Furthermore, the interaction associations of the proteins were obtained from the STRING database and the protein‑protein interaction (PPI) network was constructed using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26498477">26498477</a> Identification of key genes associated with the human abdominal aortic aneurysm based on the gene expression profile.<ul><li>Furthermore, a protein‑protein interaction (PPI) network based on the DEGs was constructed to detect highly connected regions using a <b>Cytoscape</b> plugin. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26497545">26497545</a> Further insight into molecular mechanism underlying thoracic spinal cord injury using bioinformatics methods.<ul><li>A protein‑protein interaction (PPI) network was constructed and transcription factors (TFs) were predicted using <b>cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26487783">26487783</a> Defining the three cell lineages of the human blastocyst by single-cell RNA-seq.<ul><li>In the legend, the text prior to ‘(A) <b>Cytoscape</b> enrichment map…’ should not have been included. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26481357">26481357</a> WikiPathways: capturing the full diversity of pathway knowledge.<ul><li>PathVisio and <b>Cytoscape</b>, web applications and common programming environments. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26477351">26477351</a> Network Pharmacology of Ayurveda Formulation Triphala with Special Reference to Anti-Cancer Property.<ul><li>We have developed pharmacology networks of Triphala based on the information gathered from different databases and using the software <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26469778">26469778</a> Systems pharmacology-based dissection of mechanisms of Chinese medicinal formula Bufei Yishen as an effective treatment for chronic obstructive pulmonary disease.<ul><li>The ClueGo assay, a <b>Cytoscape</b> plugin, revealed that most targets were related to activation of MAP kinase and matrix metalloproteinases. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26467653">26467653</a> Virtual pathway explorer (viPEr) and pathway enrichment analysis tool (PEANuT): creating and analyzing focus networks to identify cross-talk between molecules and pathways.<ul><li>We introduce two <b>Cytoscape</b> plug-ins, which support the generation and interpretation of experiment-based interaction networks. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26467653">26467653</a> Virtual pathway explorer (viPEr) and pathway enrichment analysis tool (PEANuT): creating and analyzing focus networks to identify cross-talk between molecules and pathways.<ul><li>The <b>Cytoscape</b> plug-in viPEr integrates -omics data with interactome data. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26458310">26458310</a> Identification of lung cancer oncogenes based on the mRNA expression and single nucleotide polymorphism profile data.<ul><li>Moreover, protein-protein interaction (PPI) network of the oncogenes was constructed using the HPRD database and then visualized using the <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26444528">26444528</a> Using the combined analysis of transcripts and metabolites to propose key genes for differential terpene accumulation across two regions.<ul><li>The gene co-expression networks were constructed with the <b>Cytoscape</b> software v 2.8.2 ( www.<b>cytoscape</b>.org).</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26439031">26439031</a> Identification of potential therapeutic target genes and mechanisms in non-small-cell lung carcinoma in non-smoking women based on bioinformatics analysis.<ul><li>Furthermore, the protein-protein interaction (PPI) network of the DEGs was constructed by <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26421278">26421278</a> mmnet: An R Package for Metagenomics Systems Biology Analysis.<ul><li>The result can be represented in an intuitive way and sent to <b>Cytoscape</b> for further exploration. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26415722">26415722</a> <b>Cytoscape</b>.js: a graph theory library for visualisation and analysis.<ul><li><b>Cytoscape</b>.js is an open-source JavaScript-based graph library. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26415722">26415722</a> <b>Cytoscape</b>.js: a graph theory library for visualisation and analysis.<ul><li><b>Cytoscape</b>.js is implemented in JavaScript. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26396508">26396508</a> Protein networks in induced sputum from smokers and COPD patients.<ul><li>Data reduction, logarithmic transformation of spectral counts, and <b>Cytoscape</b> network-interaction analysis were performed. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26379745">26379745</a> The Mechanism Research of Qishen Yiqi Formula by Module-Network Analysis.<ul><li>Next, the PINs of QSYQ were constructed by <b>Cytoscape</b> and were analyzed by gene ontology enrichment analysis based on Markov Cluster algorithm. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26357322">26357322</a> Building Transcriptional Association Networks in <b>Cytoscape</b> with RegNetC.<ul><li>The Regression Network plugin for <b>Cytoscape</b> (RegNetC) implements the RegNet algorithm for the inference of transcriptional association network from gene expression profiles. <li>Here, we present an integrated software suite, named RegNetC, as a <b>Cytoscape</b> plugin that can operate on its own as well. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26357321">26357321</a> ClusterViz: A <b>Cytoscape</b> APP for Cluster Analysis of Biological Network.<ul><li>In this paper, ClusterViz, an APP of <b>Cytoscape</b> 3 for cluster analysis and visualization, has been developed. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26354330">26354330</a> Molecular mechanisms underlying endometriosis pathogenesis revealed by bioinformatics analysis of microarray data.<ul><li>Thereafter, protein-protein interactions (PPIs) were analyzed using STRING (Search Tool for the Retrieval of Interacting Genes) database and visualized by <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26334995">26334995</a> Genes related to inflammation and bone loss process in periodontitis suggested by bioinformatics methods.<ul><li>Protein-protein interaction (PPI) network of the DEGs was constructed using <b>Cytoscape</b>, followed by module selection from the PPI network using MCODE plugin. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26329309">26329309</a> In silico analysis of the molecular mechanism of postmenopausal osteoporosis.<ul><li>In addition, the protein‑protein interaction network of DEGs was constructed via the search tool for the retrieval of interacting genes and then the functional modules were further analyzed via the clusterMaker package and The Biological Networks Gene Ontology package within the <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26328044">26328044</a> Protein-Protein Interaction Network could reveal the relationship between the breast and colon cancer.<ul><li>The genes were retrieved and compared from KEGG database and WikiPathway, and subsequently, protein-protein interaction (PPI) network was constructed and analyzed using <b>Cytoscape</b> v:3.2.1 software and related algorithms.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26324226">26324226</a> A systems biology approach to detect key pathways and interaction networks in gastric cancer on the basis of microarray analysis.<ul><li>A functional enrichment analysis was performed for all the DEGs with a BiNGO plug‑in in <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26317507">26317507</a> PhosphoPath: Visualization of Phosphosite-centric Dynamics in Temporal Molecular Networks.<ul><li>Here we present PhosphoPath, a <b>Cytoscape</b> app designed for the visualization and analysis of quantitative proteome and phosphoproteome data sets. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26297948">26297948</a> Identification of a PEAK1/ZEB1 signaling axis during TGFβ/fibronectin-induced EMT in breast cancer.<ul><li>Analysis of the <b>Cytoscape</b> interactome and Babelomics-derived gene ontologies for a novel gene set including PEAK1, CRK, ZEB1, IL11 and COL4A1 enabled us to hypothesize that PEAK1 may be regulating TGFβ-induced EMT via its interaction with or regulation of these other genes. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26276506">26276506</a> Integrative network-based analysis of mRNA and microRNA expression in 1,25-dihydroxyvitamin D3-treated cancer cells.<ul><li>The extendable network-based tools PathVisio and <b>Cytoscape</b> enable straightforward, in-depth and integrative analysis of mRNA and microRNA expression data in 1,25(OH)2D3-treated cancer cells.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26273408">26273408</a> Potential anti-cancer effect of curcumin in human lung squamous cell carcinoma.<ul><li>The protein-protein interaction (PPI) networks for up and downregulated DEGs were constructed by <b>Cytoscape</b> software using Search Tool for the Retrieval of Interacting Genes data to identify hub nodes.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26272981">26272981</a> cyNeo4j: connecting Neo4j and <b>Cytoscape</b>.<ul><li>We developed cyNeo4j, a <b>Cytoscape</b> App to link <b>Cytoscape</b> and Neo4j databases to utilize the performance and storage capacities Neo4j offers. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26272981">26272981</a> cyNeo4j: connecting Neo4j and <b>Cytoscape</b>.<ul><li>The app is available from the <b>Cytoscape</b> App Store at http://apps.<b>cytoscape</b>.org/apps/cyneo4j, the Neo4j plugins at www.github.com/gsummer/cyneo4j-parent and the community and commercial editions of Neo4j can be found at http://www.neo4j.com.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26272496">26272496</a> Integrated analysis of miRNA/mRNA network in placenta identifies key factors associated with labor onset of Large White and Qingping sows.<ul><li><b>Cytoscape</b> Network analysis of the functional genes found node mRNAs and that the regulatory network between the node mRNAs and miRNAs was established. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26271256">26271256</a> DeDaL: <b>Cytoscape</b> 3 app for producing and morphing data-driven and structure-driven network layouts.<ul><li>We developed a <b>Cytoscape</b> app, which allows constructing biological network layouts based on the data from molecular profiles imported as values of node attributes. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26271256">26271256</a> DeDaL: <b>Cytoscape</b> 3 app for producing and morphing data-driven and structure-driven network layouts.<ul><li>DeDaL is a convenient tool for applying data dimensionality reduction methods and for designing insightful data displays based on data-driven layouts of biological networks, built within <b>Cytoscape</b> environment. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26263550">26263550</a> Transcriptomic Analysis of Metabolic Pathways in Milkfish That Respond to Salinity and Temperature Changes.<ul><li>Basal metabolic pathways involved in hypothermal tolerance, such as glycolysis, fatty acid metabolism, amino acid catabolism and oxidative phosphorylation, were analyzed using PathVisio and <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26255378">26255378</a> Identifying susceptibility networks for drug-induced non-immune neutropenia.<ul><li><b>Cytoscape</b>-based analysis of detected signalling paths with statistically significant path expression scores reveals 'hub' proteins and a smaller sub-network of path proteins. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26250077">26250077</a> Screening genes associated with myocardial infarction and transverse aortic constriction using a combined analysis of miRNA and mRNA microarray.<ul><li>Common DEGs of different groups were conducted for protein-protein interaction (PPI) analysis using STRING and visualized in PPI network by <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27781081">27781081</a> Finding the shortest path with PesCa: a tool for network reconstruction.<ul><li>In this context, the PesCa app for the <b>Cytoscape</b> network analysis environment is specifically designed to help researchers infer and manipulate networks based on the shortest path principle. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26229487">26229487</a> Identification of the key genes connected with plasma cells of multiple myeloma using expression profiles.<ul><li>A protein-protein interaction (PPI) network and PPI subnetworks were constructed by <b>Cytoscape</b> software using the Search Tool for the Retrieval of Interacting Genes (STRING) database.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26229356">26229356</a> In-silico gene co-expression network analysis in Paracoccidioides brasiliensis with reference to haloacid dehalogenase superfamily hydrolase gene.<ul><li>Further co-expression (CE) study was performed individually and then a combination these genes were visualized in the <b>Cytoscape</b> 2. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26229356">26229356</a> In-silico gene co-expression network analysis in Paracoccidioides brasiliensis with reference to haloacid dehalogenase superfamily hydrolase gene.<ul><li>A significant network was selected from the <b>Cytoscape</b> network visualization that contains total 7 genes out of which 5 genes, which do not have significant protein hits, obtained from gene annotation of the expressed sequence tags by BLAST X. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26223925">26223925</a> Effects of β-catenin on differentially expressed genes in multiple myeloma.<ul><li>Furthermore, the protein-protein interaction (PPI) relationship was obtained from STRING and the protein-protein interaction network and the functional modules were visualized by <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26223573">26223573</a> Biological processes and pathway changes in isoflurane-induced anesthesia revealed by bioinformatics analysis of gene expression profiles.<ul><li>Protein-protein interactions (PPIs) among DEGs were obtained from STRING (Search Tool for the Retrieval of Interacting Genes/Proteins), and then visualized by constructing a network using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26207134">26207134</a> Microarray Analysis of the Major Depressive Disorder mRNA Profile Data.<ul><li>Protein-protein interactive network of DEGs was constructed using the <b>Cytoscape</b> software.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26205656">26205656</a> Circulating miRNAs profiles in Tourette syndrome: molecular data and clinical implications.<ul><li>Network and Gene-ontology analysis were performed by using <b>Cytoscape</b> and Babelomics server. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26191080">26191080</a> Network pharmacological identification of active compounds and potential actions of Erxian decoction in alleviating menopause-related symptoms.<ul><li>DAVID Bioinformatics Resources 6.7 and <b>Cytoscape</b> 3.01 with Jepetto plugin software were used to perform a network pharmacological analysis of EXD.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26189254">26189254</a> Identification of potential targets for ovarian cancer treatment by systematic bioinformatics analysis.<ul><li>Furthermore, the authors constructed the protein-protein interaction (PPI) network and co-expression networks by <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26187041">26187041</a> Screening of potential target genes for cataract by analyzing mRNA expression profile of mouse Hsf4-null lens.<ul><li>Furthermore, the protein-protein interaction (PPI) network of DEGs was constructed using <b>Cytoscape</b> and the key modules were selected from the PPI network based on the MCODE analysis.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26186710">26186710</a> The human urinary exosome as a potential metabolic effector cargo.<ul><li><b>Cytoscape</b> software analysis based on biological processes gene ontology database shows that metabolic pathways such as aerobic glycolysis and oxidative phosphorylation have a high probability (p ≤ 0.05) of being expressed and therefore functional. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26177842">26177842</a> Identification of genes and long non-coding RNAs associated with the pathogenesis of gastric cancer.<ul><li>The interaction relationships of the proteins encoded by DEGs that were obtained from the STRING database and protein‑protein interaction (PPI) network were visualized using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26163265">26163265</a> CytoASP: a <b>Cytoscape</b> app for qualitative consistency reasoning, prediction and repair in biological networks.<ul><li>Additionally, the <b>Cytoscape</b> platform can offer extensive functionality and visualisation capabilities. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26163265">26163265</a> CytoASP: a <b>Cytoscape</b> app for qualitative consistency reasoning, prediction and repair in biological networks.<ul><li>We report the implementation of CytoASP, an app that allows the use of BioASP for influence graph consistency checking, prediction and repair operations through <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26163265">26163265</a> CytoASP: a <b>Cytoscape</b> app for qualitative consistency reasoning, prediction and repair in biological networks.<ul><li>We have presented a user-friendly <b>Cytoscape</b> app for the analysis of regulatory networks using BioASP. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26137699">26137699</a> [Study on intersection and regulation mechanism of &quot;efficacy-toxicity network&quot; of aconite in combination environment of Sini decoction].<ul><li>The gene interaction network of Aconiti Lateralis Radix Praeparata, Glycyrrhizae Radix et Rhizoma, Zingiberis Rhizoma were mined and established with <b>Cytoscape</b> software and Agilent literature search plug-in. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26125802">26125802</a> Integrated miRNA-mRNA analysis of Epstein-Barr virus-positive nasopharyngeal carcinoma.<ul><li>Target genes associated with DEGs were used to construct the interaction network via <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26125269">26125269</a> Bioinformatics analysis of gene expression profiles in hepatocellular carcinoma.<ul><li>The resulting network was analyzed by <b>Cytoscape</b> software with CentiScaPe plugin to calculate the topological characteristics of the network and its individual node. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26114977">26114977</a> Follicular fluid alterations in endometriosis: label-free proteomics by MS(E) as a functional tool for endometriosis.<ul><li>Data was submitted to the biological network analysis using <b>Cytoscape</b> 2.8.2 with ClueGO plugin. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26104178">26104178</a> Analysis of Gene Expression During the Development of Congestive Heart Failure After Myocardial Infarction in Rat Models.<ul><li>A protein-protein interaction (PPI) network was constructed by Biomolecular Interaction Network Database and visualized by <b>Cytoscape</b>, and a subnetwork was analyzed using plugin ClusterONE in <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26103670">26103670</a> [The changes of microRNA in nasal mucosa after the specific immunotherapy for allergic rhinitis in mice].<ul><li>The MEV-4-6-0 and <b>Cytoscape</b>_v2. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26096802">26096802</a> Gene expression profiling analysis of osteosarcoma cell lines.<ul><li>Gene Ontology (GO) and pathway enrichment analysis of the DEGs was performed using the Database for Annotation, Visualization and Integrated Discovery, interactions between the proteins encoded by the DEGs were identified using STRING, and the protein‑protein interaction (PPI) network was visualized using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26064257">26064257</a> Interaction network analysis of differentially expressed genes and screening of cancer marker in the urine of patients with invasive bladder cancer.<ul><li>A total of 23 central nodes were screened with <b>Cytoscape</b> and are involved in multiple signaling pathways related to tumorigenesis. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26045708">26045708</a> Molecular mechanisms for alcoholic hepatitis based on analysis of gene expression profile.<ul><li>Both PPI network and subnetwork were visualized using the <b>Cytoscape</b> software.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26043775">26043775</a> Identification of key genes and pathways in renal cell carcinoma through expression profiling data.<ul><li>A multi-molecular regulation interaction network was constructed, and significant modules were subsequently isolated from the network by Molecular Complex Detection (Mcode) of <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26029219">26029219</a> Sequencing, de novo assembly and comparative analysis of Raphanus sativus transcriptome.<ul><li>Furthermore, a TFs-based regulation network was constructed using <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26021477">26021477</a> Profiling of urinary proteins in Karan Fries cows reveals more than 1550 proteins.<ul><li>Gene ontology (GO) using Panther7.0 grouped the majority of the proteins into cytoplasmic (location), catalytic activity (function), and metabolism (biological processes), while <b>Cytoscape</b> grouped proteins into complement and coagulation cascades; protease inhibitor activity and wound healing. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25982257">25982257</a> Latent Pathways Identification by Microarray Expression Profiles in Thyroid-Associated Ophthalmopathy Patients.<ul><li>Furthermore, protein-protein interaction (PPI) was obtained from the Search Tool for the Retrieval of Interacting Genes database, and the PPI network was visualized with <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25972860">25972860</a> Dissection of the cis-2-decenoic acid signaling network in Pseudomonas aeruginosa using microarray technique.<ul><li>A protein-protein interaction (PPI) network for differentially expressed (DE) genes with known function was then constructed by STRING and <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25955758">25955758</a> Screening biomarkers of bladder cancer using combined miRNA and mRNA microarray analysis.<ul><li>Associations between DEGs were investigated using STRING, and an interaction network was constructed using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25953442">25953442</a> Screening the molecular targets of ovarian cancer based on bioinformatics analysis.<ul><li>Furthermore, PPI network of top 10 upregulated DEGs and top 10 downregulated DEGs was constructed and visualized through <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25953080">25953080</a> CerebralWeb: a <b>Cytoscape</b>.js plug-in to visualize networks stratified by subcellular localization.<ul><li>CerebralWeb is a light-weight JavaScript plug-in that extends <b>Cytoscape</b>.js to enable fast and interactive visualization of molecular interaction networks stratified based on subcellular localization or other user-supplied annotation. <li>CerebralWeb also supports the automatic retrieval of Cerebral-compatible localizations for human, mouse and bovine genes via a web service and enables the automated parsing of <b>Cytoscape</b> compatible XGMML network files. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25945082">25945082</a> Comparative proteomic analysis of hypertrophic chondrocytes in osteoarthritis.<ul><li>ClueGO, a <b>Cytoscape</b> plug-in, is also used to compare groups of proteins and to visualize the functionally organized Gene Ontology (GO) terms and pathways in the form of dynamical network structures.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25940503">25940503</a> Novel equine conceptus?endometrial interactions on Day 16 of pregnancy based on RNA sequencing.<ul><li>In addition, <b>Cytoscape</b> interaction analysis revealed a plethora of interactions between genes expressed by the conceptus and endometrium, during a period when the former had substantial movement within the uterus. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25936657">25936657</a> Gene expression profiling via bioinformatics analysis reveals biomarkers in laryngeal squamous cell carcinoma.<ul><li>Module analysis was performed using ClusterONE (a software plugin from <b>Cytoscape</b>). </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25925016">25925016</a> Visual analysis of biological data-knowledge networks.<ul><li>An implementation is freely available as a <b>Cytoscape</b> app.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25922912">25922912</a> Metabonomic Study of the Effects of Acanthopanax senticosus on Peripheral System of Rats.<ul><li>The analysis of possible pathways influenced by A. senticosus was performed with MetaboAnalyst and <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25918880">25918880</a> Revealing the underlying mechanism of diabetic nephropathy viewed by microarray analysis.<ul><li>At last, the regulatory miRNAs for DEGs were calculated by WebGestalt, and DEGs-miRNAs network was visualized with <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25907256">25907256</a> Exploring the Molecular Mechanism and Biomakers of Liver Cancer Based on Gene Expression Microarray.<ul><li>Finally, protein-protein interactions (PPI) network of those DEGs were constructed using STRING database ( http://www.string-db.org/) and visualized by <b>Cytoscape</b> software, at the same time, genes with high degree were selected out. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25889429">25889429</a> Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<ul><li>Outstanding lncRNAs were organized to generate a TF-lncRNA-gene network using <b>Cytoscape</b> software.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25883154">25883154</a> CellWhere: graphical display of interaction networks organized on subcellular localizations.<ul><li>The <b>Cytoscape</b>.js JavaScript library is used in producing the graphical display. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25877637">25877637</a> DisGeNET: a discovery platform for the dynamical exploration of human diseases and their genes.<ul><li>It is an open access resource available through a web interface, a <b>Cytoscape</b> plugin and as a Semantic Web resource. <li>DisGeNET data can also be analysed via the DisGeNET <b>Cytoscape</b> plugin, and enriched with the annotations of other plugins of this popular network analysis software suite. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25865247">25865247</a> CyToStruct: Augmenting the Network Visualization of <b>Cytoscape</b> with the Power of Molecular Viewers.<ul><li>CyToStruct, introduced here, provides a transparent interface between the <b>Cytoscape</b> platform for network analysis and molecular viewers, including PyMOL, UCSF Chimera, VMD, and Jmol. <li>CyToStruct enriches the network tools of <b>Cytoscape</b> by adding a layer of structural analysis, offering all capabilities implemented in molecular viewers. <li>Given the coordinates of a molecular complex, our web server (http://trachel-srv.cs.haifa.ac.il/rachel/ppi/) automatically generates all files needed to visualize the complex as a <b>Cytoscape</b> network with CyToStruct bridging to PyMOL, UCSF Chimera, VMD, and Jmol.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25859558">25859558</a> Analysis and interpretation of transcriptomic data obtained from extended Warburg effect genes in patients with clear cell renal cell carcinoma.<ul><li>We developed a model of the extended Warburg effect and outlined the model using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25859558">25859558</a> Analysis and interpretation of transcriptomic data obtained from extended Warburg effect genes in patients with clear cell renal cell carcinoma.<ul><li>The <b>Cytoscape</b> network includes glycolysis, gluconeogenesis, the pentose phosphate pathway (PPP), the TCA cycle, the serine/glycine pathway, and partial glutaminolysis and fatty acid synthesis pathways. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25855937">25855937</a> Common genes in coronary artery disease from Europe, Asia and North America regardless of race and lifestyle.<ul><li>PPI networks of the common set of DEGs were established by <b>Cytoscape</b> software to understand their biological role in CAD.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25842907">25842907</a> [Analysis of metabolic profile of Chlamydomonas reinhardtii cultivated under autotrophic conditions].<ul><li>The mapping of metabolites by their chemical structure with the use of <b>Cytoscape</b> software, combined with quantitative interpretation, showed that the majority of organic matter was concentrated primarily in the carbon backbones of fatty acids and terpenes, as well as saccharides and structurally similar compounds.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25833953">25833953</a> Construction of phosphorylation interaction networks by text mining of full-length articles using the eFIP system.<ul><li>We enhance the website functionality to: (i) support searches based on protein roles (kinases, substrates, interacting partners) or using keywords; (ii) link protein entities to their corresponding UniProt identifiers if mapped and (iii) support visual exploration of phosphorylation interaction networks using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25832863">25832863</a> Identification of potential therapeutic target genes, key miRNAs and mechanisms in acute myeloid leukemia based on bioinformatics analysis.<ul><li>The functional enrichment analysis was performed, and protein-protein interaction (PPI) network of the DEGs was constructed by <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25824986">25824986</a> Identification of therapeutic targets for breast cancer using biological informatics methods.<ul><li>Subsequently, the protein-protein interaction (PPI) network was constructed using the <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25805103">25805103</a> Identification of dysfunctional biological pathways and their synergistic mechanism in hepatocellular carcinoma process.<ul><li>Furthermore, we performed collaborative analysis on these pathways through calculating the Jaccard index, and crosstalk networks were constructed and visualized by <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25786624">25786624</a> Systematic analysis to identify a key role of CDK1 in mediating gene interaction networks in cervical cancer development.<ul><li><b>Cytoscape</b> was utilized to construct biological networks.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25785089">25785089</a> Identification of B cells participated in the mechanism of postmenopausal women osteoporosis using microarray analysis.<ul><li>Protein-protein interaction network (PPI) of up- and down-regulated DEGs was respectively constructed by <b>Cytoscape</b> software using the STRING data. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25771989">25771989</a> Comprehensive transcriptome analysis of mesenchymal stem cells in elderly patients with osteoporosis.<ul><li>A protein-protein interaction (PPI) network of DEGs was constructed with STRING and then visualized with <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25769893">25769893</a> A network pharmacology study of Sendeng-4, a Mongolian medicine.<ul><li>We collected the data on the Sendeng-4 chemical composition corresponding targets through the literature and from DrugBank, SuperTarget, TTD (Therapeutic Targets Database) and other databases and the relevant signaling pathways from the KEGG (Kyoto Encyclopedia of Genes and Genomes) database and established models of the chemical composition-target network and chemical composition-target-disease network using <b>Cytoscape</b> software, the analysis indicated that the chemical composition had at least nine different types of targets that acted together to exert effects on the diseases, suggesting a &quot;multi-component, multi-target&quot; feature of the traditional Mongolian medicine. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25768320">25768320</a> Identification of hepatocellular carcinoma-associated hub genes and pathways by integrated microarray analysis.<ul><li>A protein-protein interaction network was constructed using the <b>Cytoscape</b> software; the netwerk served to find hub genes for HCC. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25757789">25757789</a> Constructing simple biological networks for understanding complex high-throughput data in plants.<ul><li>Step-by-step instructions for acquiring, integrating, analyzing, and visualizing genome-wide data are provided for two widely used open source platforms, R and <b>Cytoscape</b> platforms. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25756408">25756408</a> Gene expression profiling analysis of patients with ankylosing spondylitis.<ul><li>Besides, the interaction relationships of the proteins encoded by DEGs were searched by STRING, and the protein-protein interaction (PPI) network was visualized by <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25755273">25755273</a> BMRF-Net: a software tool for identification of protein interaction subnetworks by a bagging Markov random field-based method.<ul><li>A user friendly graphic user interface is developed as a <b>Cytoscape</b> plugin for the BMRF-Net software to deal with the input/output interface. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25755273">25755273</a> BMRF-Net: a software tool for identification of protein interaction subnetworks by a bagging Markov random field-based method.<ul><li>The package is tested under Ubuntu 12.04 (64-bit), Java 7, glibc 2.15 and <b>Cytoscape</b> 3.1.0.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25753876">25753876</a> Identifying hub genes and dysregulated pathways in hepatocellular carcinoma.<ul><li>Protein-protein interaction (PPI) network was constructed using <b>Cytoscape</b> 2.1. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25753874">25753874</a> Effect evaluation of cisplatin-gemcitabine combination chemotherapy for advanced non-small cell lung cancer patients using microarray data.<ul><li>Protein-protein interaction (PPI) analysis was to identify hub nodes and interacting pairs in dataset C and constructed into PPI network using <b>Cytoscape</b> software, followed by screening of small molecules using Connectivity Map.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25750651">25750651</a> CoVennTree: a new method for the comparative analysis of large datasets.<ul><li>We developed a new method called CoVennTree (Comparative weighted Venn Tree) that simultaneously compares up to three multifarious datasets by aggregating and propagating information from the bottom to the top level and produces a graphical output in <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25695524">25695524</a> Cross-platform meta-analysis of multiple gene expression profiles identifies novel expression signatures in acquired anthracycline-resistant breast cancer.<ul><li>A protein-protein interaction (PPI) network was also generated using <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25695263">25695263</a> Screening for characteristic microRNAs between pre-invasive and invasive stages of cervical cancer.<ul><li>The regulatory networks were visualized using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25660986">25660986</a> Protein-protein interaction network of gene expression in the hydrocortisone-treated keloid.<ul><li>Finally, the interactions of DEGs in samples of keloid treated with hydrocortisone were explored in a human protein-protein interaction (PPI) network, and sub-modules of the DEGs interaction network were analyzed using <b>Cytoscape</b> software.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25635527">25635527</a> Autophagy Regulatory Network - a systems-level bioinformatics resource for studying the mechanism and regulation of autophagy.<ul><li>The database can be downloaded in SQL, CSV, BioPAX, SBML, PSI-MI, and in a <b>Cytoscape</b> CYS file formats. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25604238">25604238</a> ocsESTdb: a database of oil crop seed EST sequences for comparative analysis and investigation of a global metabolic network and oil accumulation metabolism.<ul><li>Five genome-scale metabolic networks that contain different numbers of metabolites and gene-enzyme reaction-association entries were analysed and constructed using <b>Cytoscape</b> and yEd programs. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25592164">25592164</a> Gene expression profiling analysis of castration-resistant prostate cancer.<ul><li>We identified DEGs using limma Geoquery in R, the Robust Multi-array Average (RMA) method in Bioconductor, and Bias methods, followed by constructing an integrated regulatory network involving DEGs, miRNAs, and TFs using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25585621">25585621</a> Exploring the association between interleukin-1β and its interacting proteins in Alzheimer's disease.<ul><li>Proteins, which have been previously reported to be associated with IL-1β in AD, have been used in the present study as nodes to illustrate a net of protein interaction in <b>Cytoscape</b>. <li>The present study identified and subsequently compared two AD and six IL-1β pathways with the network produced in <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25580224">25580224</a> The <b>Cytoscape</b> app article collection.<ul><li>As a network visualization and analysis platform, <b>Cytoscape</b> relies on apps to provide domain-specific features and functions. <li>This article collection is another resource to help researchers find out more about relevant <b>Cytoscape</b> apps and to provide app developers with useful implementation tips. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25577435">25577435</a> ENViz: a <b>Cytoscape</b> App for integrated statistical analysis and visualization of sample-matched data with multiple data types.<ul><li>ENViz (Enrichment Analysis and Visualization) is a <b>Cytoscape</b> app that performs joint enrichment analysis of two types of sample matched datasets in the context of systematic annotations. <li>The results of the enrichment analysis are visualized as an interactive <b>Cytoscape</b> network.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25576219">25576219</a> MicroRNA expression profiling and functional annotation analysis of their targets associated with the malignant transformation of oral leukoplakia.<ul><li>Besides, potential transcription factors were predicted using CyTargetLinker plugin and the miRNA-mRNA regulatory network associated with the malignant pathogenesis was visualized in <b>Cytoscape</b> environment. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25573623">25573623</a> Identification of hub subnetwork based on topological features of genes in breast cancer.<ul><li>A PPI network was constructed using <b>Cytoscape</b> and clusters that focused on highly connected nodes were obtained using the molecular complex detection (MCODE) clustering algorithm. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25565839">25565839</a> Analysis of zinc oxide nanoparticles binding proteins in rat blood and brain homogenate.<ul><li>These proteins were further analyzed with ClueGO, a <b>Cytoscape</b> plugin, which provided gene ontology and the biological interaction processes of identified proteins. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25565838">25565838</a> Analysis of SiO2 nanoparticles binding proteins in rat blood and brain homogenate.<ul><li>The proteins identified as bound in the corona from SiO2 nanoparticles were further analyzed with ClueGO, a <b>Cytoscape</b> plugin used in protein ontology and for identifying biological interaction pathways. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25565391">25565391</a> Immune system-related differentially expressed genes, transcription factors and microRNAs in post-menopausal females with osteopenia.<ul><li>Enrichment analysis was performed using DAVID (Database for Annotation, Visualization and Integrated Discovery) and visualized using plug-in EnrichmentMap of <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25563367">25563367</a> Microarray data analysis reveals differentially expressed genes in prolactinoma.<ul><li>Protein complexes were predicted using ClusterONE from <b>Cytoscape</b> and then validated in terms of pathways, protein domain, cellular localization and literatures. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26949516">26949516</a> Social Network: a <b>Cytoscape</b> app for visualizing co-authorship networks.<ul><li>The Social Network <b>Cytoscape</b> app is capable of creating a visual summary of connected individuals automatically. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26937268">26937268</a> CyAnimator: Simple Animations of <b>Cytoscape</b> Networks.<ul><li>CyAnimator (http://apps.<b>cytoscape</b>.org/apps/cyanimator) is a <b>Cytoscape</b> app that provides a tool for simple animations of <b>Cytoscape</b> networks. <li>CyAnimator is available from within the <b>Cytoscape</b> App Manager or from the <b>Cytoscape</b> app store.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25541727">25541727</a> ReNE: a <b>cytoscape</b> plugin for regulatory network enhancement.<ul><li>In this context, we introduce ReNE, a <b>Cytoscape</b> 3.x plugin designed to automatically enrich a standard gene-based regulatory network with more detailed transcriptional, post transcriptional, and translational data, resulting in an enhanced network that more precisely models the actual biological regulatory mechanisms. <li>The merged network structure is normalized to guarantee a consistent and uniform description of the network nodes and edges and to enrich all integrated data with additional annotations retrieved from genome-wide databases like NCBI, thus producing a pathway fully manageable through the <b>Cytoscape</b> environment. <li>The resulting enhanced network is still a fully functional <b>Cytoscape</b> network where each regulatory element (transcription factor, miRNA, gene, protein) and regulatory mechanism (up-regulation/down-regulation) is clearly visually identifiable, thus enabling a better visual understanding of its role and the effect in the network behavior. <li>ReNE can be freely installed from the <b>Cytoscape</b> App Store (http://apps.<b>cytoscape</b>.org/apps/rene) and the full source code is freely available for download through a SVN repository accessible at http://www.sysbio.polito.it/tools_svn/BioInformatics/Rene/releases/. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25540777">25540777</a> Comprehensive reconstruction and visualization of non-coding regulatory networks in human.<ul><li>It is equipped with three interfaces: web based, command-line, and a <b>Cytoscape</b> app called ncINetView. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25527096">25527096</a> The Cyni framework for network inference in <b>Cytoscape</b>.<ul><li>Cyni allows the rapid transformation of Java-based network inference prototypes into apps of the popular open-source <b>Cytoscape</b> network analysis and visualization ecosystem. <li>In a case study, we illustrate the transformation of an ARACNE implementation into a <b>Cytoscape</b> app.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25522143">25522143</a> Plasmid flux in Escherichia coli ST131 sublineages, analyzed by plasmid constellation network (PLACNET), a new method for plasmid reconstruction from whole genome sequences.<ul><li>The resulting network is pruned by expert analysis, to eliminate confounding data, and implemented in a <b>Cytoscape</b>-based graphic representation. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25522063">25522063</a> TimeXNet: identifying active gene sub-networks using time-course gene expression profiles.<ul><li>The output of TimeXNet can be directly viewed in <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25521941">25521941</a> cytoHubba: identifying hub objects and sub-networks from complex interactome.<ul><li>We introduce a novel <b>Cytoscape</b> plugin cytoHubba for ranking nodes in a network by their network features. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25521941">25521941</a> cytoHubba: identifying hub objects and sub-networks from complex interactome.<ul><li>According to <b>cytoscape</b> plugin download statistics, the accumulated number of cytoHubba is around 6,700 times since 2010.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25520778">25520778</a> <b>Cytoscape</b> tools for the web age: D3.js and <b>Cytoscape</b>.js exporters.<ul><li>In this paper we present new data export modules for <b>Cytoscape</b> 3 that can generate network files for <b>Cytoscape</b>.js and D3.js. <li>These modules enable users to seamlessly export network and table data sets generated in <b>Cytoscape</b> to popular JavaScript library readable formats. <li>Example web applications created with these tools demonstrate how <b>Cytoscape</b> works in modern data visualization workflows built with traditional desktop tools and emerging web-based technologies. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25519357">25519357</a> Pathway analysis for genetic association studies: to do, or not to do? That is the question.<ul><li>At stage 2, networks and enriched pathways were estimated with <b>Cytoscape</b> for genes from stage 1, separately for the 2 phenotypes, then examining network overlap. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25515564">25515564</a> SMC3 may play an important role in atopic asthma development.<ul><li>Further, search tool for the retrieval of interacting genes/proteins (STRING) was used to select protein-protein interaction (PPI) for DEGs, and then the network was visualized by <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25512274">25512274</a> [Targeted effect of microRNA on nerve growth factor pathway and its functional network in patients with unstable angina].<ul><li>The functional network of microRNAs by <b>Cytoscape</b> was costructed.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25511031">25511031</a> Identification of differently expressed genes in leukemia using multiple microarray datasets.<ul><li>A protein-protein interaction network was constructed using the <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25501146">25501146</a> Protein-protein interaction network analysis of osteoarthritis-related differentially expressed genes.<ul><li>Meanwhile network module analysis was performed using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25488183">25488183</a> Enoxaparin sensitizes human non-small-cell lung carcinomas to gefitinib by inhibiting DOCK1 expression, vimentin phosphorylation, and Akt activation.<ul><li>Molecular interaction networks were visualized using the <b>Cytoscape</b> bioinformatics platform. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25475428">25475428</a> Database screening of herbal monomers regulating autophagy by constructing a &quot;disease-gene-drug&quot; network.<ul><li>In addition, we used the CFinder software to identify several functional modules, performed gene ontology-biological process (GO-BP) functional enrichment analysis using the DAVID software, constructed a herbal monomer-module gene regulatory network using literature search and the <b>Cytoscape</b> software, and then analyzed the relationships between autophagy, genes, and herbal monomers.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25469323">25469323</a> Visual analytics in healthcare education: exploring novel ways to analyze and represent big data in undergraduate medical education.<ul><li><b>Cytoscape</b> software was employed to build networks of the identified aspects and visualize them. <li>Further analysis and visualization of the identified aspects with <b>Cytoscape</b> resulted in building an abstract model of the examined data that presented three different approaches; (i) learning outcomes and teaching methods, (ii) examination and learning outcomes, and (iii) teaching methods, learning outcomes, examination results, and gap analysis. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25468726">25468726</a> Molecular mechanisms of osteoarthritis using gene microarrays.<ul><li>Next, gene-set enrichment analysis was performed using Enrichment map <b>Cytoscape</b> plug-in, followed by detecting sub-networks using clusterONE. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25451770">25451770</a> CytoNCA: a <b>cytoscape</b> plugin for centrality analysis and evaluation of protein interaction networks.<ul><li>Here we present CytoNCA, a <b>Cytoscape</b> plugin integrating calculation, evaluation and visualization analysis for multiple centrality measures.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25450280">25450280</a> Analysis of hepatocellular carcinoma and metastatic hepatic carcinoma via functional modules in a protein-protein interaction network.<ul><li>We used human protein interaction data to build a protein-protein interaction network with <b>Cytoscape</b> and then derived functional clusters using MCODE. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25445499">25445499</a> Microarray gene expression profiling and bioinformatics analysis of premature ovarian failure in a rat model.<ul><li>Potential regulatory targets associated with these DEGs were constructed using BisoGenet in <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25435164">25435164</a> Gene co-expression network and function modules in three types of glioma.<ul><li>After the co‑expression network of DEGs was constructed by <b>Cytoscape</b>, the functional modules were mined and enrichment analysis was performed, and then the similarities and differences between any two types of glioma were compared. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25434727">25434727</a> Regulatory network of differentially expressed genes in metastatic osteosarcoma.<ul><li>A differential co‑expression network was also constructed, and the submodules were screened using MCODE in <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25433696">25433696</a> LigDig: a web server for querying ligand-protein interactions.<ul><li>LigDig makes use of freely available databases, including ChEMBL, PubChem and SABIO-RK, and software programs, including <b>cytoscape</b>.js, PDB2PQR, ProBiS and Fconv. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25410188">25410188</a> Immune- and ribosome-related genes were associated with systemic vasculitis.<ul><li>Mcode in <b>Cytoscape</b> software was used to extract modules from the network (degree ≥ 2, K-core ≥ 2 and adjusted P-value &lt; 0.05) followed by pathway analysis using GenMAPP (false discovery rate &lt; 0.05). </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25409614">25409614</a> A comprehensive analysis of candidate genes and pathways in pancreatic cancer.<ul><li>Latent pathway identification analysis was applied to analyze the DEGs-related pathways crosstalk and the pathways with high weight value were included in the pathway crosstalk network using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25408372">25408372</a> Identification of differently expressed genes with specific SNP Loci for breast cancer by the integration of SNP and gene expression profiling analyses.<ul><li>With NetBox software, TRED (Transcriptional Regulatory Element Database) and UCSC (University of California Santa Cruz) database, genes-associated network and transcriptional regulatory network were constructed using <b>cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25400907">25400907</a> Development and use of the <b>Cytoscape</b> app GFD-Net for measuring semantic dissimilarity of gene networks.<ul><li>We present GFD-Net, a <b>Cytoscape</b> app for visualizing and analyzing the functional dissimilarity of gene networks.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25392417">25392417</a> CeCaFDB: a curated database for the documentation, visualization and comparative analysis of central carbon metabolic flux distributions explored by 13C-fluxomics.<ul><li>In addition to its comprehensive literature-derived data, the CeCaFDB supports a common text search function among the data and interactive visualization of the curated flux distributions with compartmentation information based on the <b>Cytoscape</b> Web API, which facilitates data interpretation. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25392406">25392406</a> VirHostNet 2.0: surfing on the web of virus/host molecular interactions data.<ul><li>The new interface of VirHostNet 2.0 is based on <b>Cytoscape</b> web library and provides a user-friendly access to the most complete and accurate resource of virus-virus and virus-host protein-protein interactions as well as their projection onto their corresponding host cell protein interaction networks. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25385480">25385480</a> Identification of genes associated with the differentiation potential of adipose-derived stem cells to osteocytes or myocytes.<ul><li>Besides, the interaction relationships of the proteins encoded by DEGs were obtained from STRING database, and protein-protein interaction (PPI) network was constructed using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25383185">25383185</a> CyKEGGParser: tailoring KEGG pathways to fit into systems biology analysis workflows.<ul><li>In this paper we present the CyKEGGParser app ( http://apps.<b>cytoscape</b>.org/apps/cykeggparser) for <b>Cytoscape</b> 3 that allows manipulation with KEGG pathway maps. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25380962">25380962</a> CytoCom: a <b>Cytoscape</b> app to visualize, query and analyse disease comorbidity networks.<ul><li>CytoCom is an interactive plugin for <b>Cytoscape</b> that can be used to search, explore, analyse and visualize human disease comorbidity network. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25378329">25378329</a> MatrixDB, the extracellular matrix interaction database: updated content, a new navigator and expanded functionalities.<ul><li>Networks may be exported under <b>Cytoscape</b> and tabular formats and as images, and may be saved for subsequent re-use.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25365344">25365344</a> Computational analysis of reciprocal association of metabolism and epigenetics in the budding yeast: a genome-scale metabolic model (GSMM) approach.<ul><li>Accordingly, we defined the centrality indices of the lowly expressed enzymes in the undirected enzyme-centric network of yeast by CytoHubba plug-in in <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25358439">25358439</a> Ranking candidate genes of esophageal squamous cell carcinomas based on differentially expressed genes and the topological properties of the co-expression network.<ul><li>Furthermore, the topological properties of the co-expression network were analyzed through NetworkAnalyzer (default settings) of <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25352980">25352980</a> setsApp for <b>Cytoscape</b>: Set operations for <b>Cytoscape</b> Nodes and Edges.<ul><li>setsApp ( http://apps.<b>cytoscape</b>.org/apps/setsapp) is a relatively simple <b>Cytoscape</b> 3 app for users to handle groups of nodes and/or edges. <li>Automatic set partitioning and layout functions are also provided. The sets functionality is also exposed to users and app developers in the form of a set of commands that can be used for scripting purposes or integrated in other <b>Cytoscape</b> apps.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25351486">25351486</a> Distinctive pathways characterize A. actinomycetemcomitans and P. gingivalis.<ul><li>gingivalis-related protein-protein interaction networks, and background network, which included average shortest path length (ASPL), degree, closeness centrality (CC), eccentricity (EC), betweenness centrality (BC) and topological coefficient (TC) were compared using network analysis plugin of <b>Cytoscape</b>, followed by pathway enrichment analysis (p value &lt;0.05) using FISHER hyper-geometric algorithm and calculation of pathway alter scores using LIMMA. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25348135">25348135</a> Gene expression analyses to explore the biomarkers and therapeutic targets for gliomas.<ul><li>Bingo plug-in in <b>Cytoscape</b> software was applied to analyze Gene Ontology functional enrichment analysis for up- and down-regulated DEGs in network, respectively. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25339452">25339452</a> Co-expression analysis of differentially expressed genes in hepatitis C virus-induced hepatocellular carcinoma.<ul><li>Subsequently, MCODE in <b>Cytoscape</b> software was applied to conduct module analysis of the constructed co-expression networks. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25324964">25324964</a> GASOLINE: a <b>Cytoscape</b> app for multiple local alignment of PPI networks.<ul><li>In this paper we present GASOLINE, a <b>Cytoscape</b> app for multiple local alignments of PPI (protein-protein interaction) networks. <li>GASOLINE is a <b>Cytoscape</b> app for computing and visualizing local alignments, without requiring any post-processing operations. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25309732">25309732</a> ReactomeFIViz: a <b>Cytoscape</b> app for pathway and network-based data analysis.<ul><li>We have developed a <b>Cytoscape</b> app called &quot;ReactomeFIViz&quot;, which utilizes a highly reliable gene functional interaction network combined with human curated pathways derived from Reactome and other pathway databases. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25285206">25285206</a> enhancedGraphics: a <b>Cytoscape</b> app for enhanced node graphics.<ul><li>enhancedGraphics ( http://apps.<b>cytoscape</b>.org/apps/enhancedGraphics) is a <b>Cytoscape</b> app that implements a series of enhanced charts and graphics that may be added to <b>Cytoscape</b> nodes. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25275790">25275790</a> Affinity purification-mass spectrometry and network analysis to understand protein-protein interactions.<ul><li>The protocol also explores methods available in <b>Cytoscape</b> to visualize and analyze these types of interaction data. <li>Similarly, the network analysis and visualization protocol in <b>Cytoscape</b> takes 2-4 h to complete with the provided sample data, but we recommend taking days or even weeks to explore one's data and find the right questions.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25275465">25275465</a> GIANT: a <b>cytoscape</b> plugin for modular networks.<ul><li>Here we propose a <b>Cytoscape</b> plugin (GIANT) to perform network clustering and characterize nodes at the light of a modified Guimerà-Amaral cartography. <li>The plugin has been already approved and uploaded on the <b>Cytoscape</b> APP store.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25273109">25273109</a> KDDN: an open-source <b>Cytoscape</b> app for constructing differential dependency networks with significant rewiring.<ul><li>This knowledge-fused differential dependency networks (KDDN) method, implemented as a Java <b>Cytoscape</b> app, can be used to optimally integrate prior biological knowledge with measured data to simultaneously construct both common and differential networks, to quantitatively assign model parameters and significant rewiring p-values and to provide user-friendly graphical results. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25260698">25260698</a> PINBPA: <b>cytoscape</b> app for network analysis of GWAS data.<ul><li>Protein interaction network-based pathway analysis (PINBPA) for genome-wide association studies (GWAS) has been developed as a <b>Cytoscape</b> app, to enable analysis of GWAS data in a network fashion. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25260698">25260698</a> PINBPA: <b>cytoscape</b> app for network analysis of GWAS data.<ul><li>PINBPA app is freely available in <b>Cytoscape</b> app store.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25257390">25257390</a> Gene expression profiles analysis identifies key genes for acute lung injury in patients with sepsis.<ul><li>Moreover, the gene regulatory network was constructed with information from Kyoto Encyclopedia of Genes and Genomes (KEGG) and then visualized by <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25254104">25254104</a> GeneMANIA: Fast gene network construction and function prediction for <b>Cytoscape</b>.<ul><li>The GeneMANIA <b>Cytoscape</b> app enables users to construct a composite gene-gene functional interaction network from a gene list. <li>The app is compatible with <b>Cytoscape</b> versions 2 and 3.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25254103">25254103</a> WikiPathways App for <b>Cytoscape</b>: Making biological pathways amenable to network analysis and visualization.<ul><li>In this paper we present the open-source WikiPathways app for <b>Cytoscape</b> ( http://apps.<b>cytoscape</b>.org/apps/wikipathways) that can be used to import biological pathways for data visualization and network analysis. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25227403">25227403</a> Renal agenesis in Kallmann syndrome: a network approach.<ul><li>We created a STRING protein interaction network from KS-related genes and renal-agenesis-associated genes and analyzed it with <b>Cytoscape</b> 3.0.1 network software. <li>In addition, STRING and <b>Cytoscape</b> 3.0.1 software identified new potential KS renal-aplasia-associated genes (PAX2, BMP4, and SOX10). </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25227254">25227254</a> Analyzing the differentially expressed genes and pathway cross-talk in aggressive breast cancer.<ul><li>The gene ontology (GO) term enrichment and pathway cross-talk analysis were performed with the online tool of DAVID and <b>Cytoscape</b> software.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25216767">25216767</a> Looking for complexity in quantitative semiology of frontal and temporal lobe seizures using neuroethology and graph theory.<ul><li>The same series were used for graph analysis (<b>CYTOSCAPE</b>). </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25212755">25212755</a> cddApp: a <b>Cytoscape</b> app for accessing the NCBI conserved domain database.<ul><li>cddApp is a <b>Cytoscape</b> extension that supports the annotation of protein networks with information about domains and specific functional sites from the National Center for Biotechnology Information's conserved domain database (CDD). <li>cddApp integrates with the <b>Cytoscape</b> apps structureViz2 and enhancedGraphics. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25212755">25212755</a> cddApp: a <b>Cytoscape</b> app for accessing the NCBI conserved domain database.<ul><li>cddApp is written in Java and freely available for download from the <b>Cytoscape</b> app store (http://apps.<b>cytoscape</b>.org). </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25211130">25211130</a> Characterization of transcription factor networks involved in umbilical cord blood CD34+ stem cells-derived erythropoiesis.<ul><li>Both GSEA datasets were analyzed using the MIMI <b>Cytoscape</b> platform, which discovered TFNs centered on KLF4 and GATA2 (Profile-1) and KLF1 and GATA1 for Profile-2 genes. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25205120">25205120</a> Text mining and network analysis of molecular interaction in non-small cell lung cancer by using natural language processing.<ul><li>We automatically extracted the human-to-mouse orthologous interactions using the GeneWays system by natural language processing and further constructed molecular (gene and its products) networks by mapping the human-to-mouse interactions to NSCLC-related mammalian phenotypes, followed by module analysis using ClusterONE of <b>Cytoscape</b> and pathway enrichment analysis using the database for annotation, visualization and integrated discovery (DAVID) successively. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25199793">25199793</a> Biological network exploration with <b>Cytoscape</b> 3.<ul><li><b>Cytoscape</b> is one of the most popular open-source software tools for the visual exploration of biomedical networks composed of protein, gene, and other types of interactions. <li><b>Cytoscape</b> provides core functionality to load, visualize, search, filter, and save networks, and hundreds of Apps extend this functionality to address specific research needs. <li>This protocol aims to jump-start new users with specific protocols for basic <b>Cytoscape</b> functions, such as installing <b>Cytoscape</b> and <b>Cytoscape</b> Apps, loading data, visualizing and navigating the networks, visualizing network associated data (attributes), and identifying clusters. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25199469">25199469</a> Screening of diagnostic markers for osteosarcoma.<ul><li>Subsequently, the protein interaction network was decomposed and Gene Otology annotation using <b>Cytoscape</b>, Mcode and Bingo, was conducted on the function modules. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25189652">25189652</a> Investigation of key miRNAs and target genes in bladder cancer using miRNA profiling and bioinformatic tools.<ul><li>To construct potential miRNA regulatory networks, the overlapping parts of three selected KEGG pathways were visualized by <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25183411">25183411</a> Network analysis of ChIP-Seq data reveals key genes in prostate cancer.<ul><li>Protein-protein interaction (PPI) network analysis was performed with BioGRID and constructed with <b>Cytoscape</b>, following which the highly connected genes were screened.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25182125">25182125</a> Mapping knowledge translation and innovation processes in Cancer Drug Development: the case of liposomal doxorubicin.<ul><li>In order to map the processes, a literature network analysis was made through <b>Cytoscape</b> and semantic analysis was performed by GOPubmed which is based in the controlled vocabularies MeSH (Medical Subject Headings) and GO (Gene Ontology). </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25177485">25177485</a> KEGGscape: a <b>Cytoscape</b> app for pathway data integration.<ul><li>In this paper, we present KEGGscape a pathway data integration and visualization app for <b>Cytoscape</b> ( http://apps.<b>cytoscape</b>.org/apps/keggscape). <li>KEGGscape utilizes the database to reproduce the corresponding hand-drawn pathway diagrams with as much detail as possible in <b>Cytoscape</b>. <li>coli chips are used as an example to demonstrate how users can integrate pathways, annotations, and experimental data sets to create complex visualizations that clarify biological systems using KEGGscape and other <b>Cytoscape</b> apps.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25172542">25172542</a> Identification of key genes affecting disease free survival time of pediatric acute lymphoblastic leukemia based on bioinformatic analysis.<ul><li>Then, based on latent pathway identification analysis (LPIA), latent pathways were identified, and pathway-pathway interaction network was constructed and visualized by <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25165537">25165537</a> <b>Cytoscape</b>: the network visualization tool for GenomeSpace workflows.<ul><li>One such tool is <b>Cytoscape</b> 3, a popular application that enables analysis and visualization of graph-oriented genomic networks. <li>In this paper, we present the design and operation of the <b>Cytoscape</b> GenomeSpace app, which accomplishes this integration, thereby providing critical analysis and visualization functionality for GenomeSpace users. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25153667">25153667</a> FunMod: a <b>Cytoscape</b> plugin for identifying functional modules in undirected protein-protein networks.<ul><li>In this paper we present FunMod, an innovative <b>Cytoscape</b> version 2.8 plugin that is able to mine undirected protein-protein networks and to infer sub-networks of interacting proteins intimately correlated with relevant biological pathways. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25151173">25151173</a> Hox protein interactions: screening and network building.<ul><li>In this chapter we propose a strategy and methods to map Hox interaction networks, from yeast two-hybrid and high-throughput yeast two-hybrid interaction screening to bioinformatic analyses based on the software platform <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25149502">25149502</a> Biological network inferences for a protection mechanism against familial Creutzfeldt-Jakob disease with E200K pathogenic mutation.<ul><li>Biological networks were created using <b>Cytoscape</b> (v2.8.3 and v3.0.2) and Pathway Studio 9.0 software.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25138169">25138169</a> Pepper: <b>cytoscape</b> app for protein complex expansion using protein-protein interaction networks.<ul><li>We introduce Pepper (Protein complex Expansion using Protein-Protein intERactions), a <b>Cytoscape</b> app designed to identify protein complexes as densely connected subnetworks from seed lists of proteins derived from proteomic studies. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25138169">25138169</a> Pepper: <b>cytoscape</b> app for protein complex expansion using protein-protein interaction networks.<ul><li>Pepper is available from the <b>Cytoscape</b> plug-in manager or online (http://apps.<b>cytoscape</b>.org/apps/pepper) and released under GNU General Public License version 3.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25134827">25134827</a> KeyPathwayMiner 4.0: condition-specific pathway analysis by combining multiple omics studies and networks with <b>Cytoscape</b>.<ul><li>Finally the new release now also provides a set of novel visualization features and has been implemented as an app for the standard bioinformatics network analysis tool: <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25134827">25134827</a> KeyPathwayMiner 4.0: condition-specific pathway analysis by combining multiple omics studies and networks with <b>Cytoscape</b>.<ul><li>With KeyPathwayMiner 4.0, we publish a <b>Cytoscape</b> app for multi-omics based sub-network extraction. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25125975">25125975</a> Serum cytokine profiling and enrichment analysis reveal the involvement of immunological and inflammatory pathways in stable patients with chronic obstructive pulmonary disease.<ul><li>Enrichment pathways and network analyses were implemented, using a list of cytokines showing differential expression between healthy controls and patients with COPD by <b>Cytoscape</b> and GeneGo Metacore™ software (Thomson-Reuters Corporation, New York, NY, USA). </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25118911">25118911</a> Differential co-expression analysis of rheumatoid arthritis with microarray data.<ul><li>Finally, the obtained TFs were mapped to the known TF-targets to construct the network using <b>cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25116327">25116327</a> Grouping annotations on the subcellular layered interactome demonstrates enhanced autophagy activity in a recurrent experimental autoimmune uveitis T cell line.<ul><li>That is, the subcellular layered interactome was established by the <b>Cytoscape</b> and Cerebral plugin based on differential expression, global interactome, and subcellular localization information. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25110584">25110584</a> BridgeDb app: unifying identifier mapping services for <b>Cytoscape</b>.<ul><li>The BridgeDb app for <b>Cytoscape</b> allows users to map and annotate identifiers of genes, proteins and metabolites in the context of biological networks. <li>The app also provides a programming interface via <b>Cytoscape</b> Commands that can be utilized for identifier mapping by other <b>Cytoscape</b> apps. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25080304">25080304</a> An extended gene protein/products Boolean network model including post-transcriptional regulation.<ul><li>The open-source toolkit is compatible with available visualization tools like <b>Cytoscape</b> and allows to run detailed analysis of the network topology as well as of its attractors, trajectories, and state-space. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25075307">25075307</a> Porting and using PanGIA for <b>Cytoscape</b> 3: challenges and solutions.<ul><li>Much of the biologically significant functionality in <b>Cytoscape</b> is contained within third-party add-ons, called plugins in <b>Cytoscape</b> 2 and apps in <b>Cytoscape</b> 3. <li>PanGIA is a <b>Cytoscape</b> add-on (http://apps.<b>cytoscape</b>.org/apps/pangia) designed to analyze and visualize genetic interaction data in light of physical interaction data. <li>Most authors of <b>Cytoscape</b> 2 plugins will face similar challenges porting their software to work with <b>Cytoscape</b> 3, and this paper gives details of how the PanGIA port addressed them.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25075306">25075306</a> Enrichment Map - a <b>Cytoscape</b> app to visualize and explore OMICs pathway enrichment results.<ul><li>We have updated Enrichment Map to support <b>Cytoscape</b> 3, and have added additional features including new data formats and command line access.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25070821">25070821</a> Identification crucial genes in peripheral neuropathic pain induced by spared nerve injury.<ul><li>Protein-protein interaction (PPI) network of DEGs were constructed by STRING online database, and co-expression networks were constructed through <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25058310">25058310</a> PANET: a GPU-based tool for fast parallel analysis of robustness dynamics and feed-forward/feedback loop structures in large-scale biological networks.<ul><li>A <b>Cytoscape</b> plugin named NetDS was recently proposed to analyze the robustness-related dynamics and feed-forward/feedback loop structures of biological networks. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25058159">25058159</a> iRegulon: from a gene list to a gene regulatory network using large motif and track collections.<ul><li>iRegulon is available as a <b>Cytoscape</b> plugin from http://iregulon.aertslab.org.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25057483">25057483</a> The human plasma membrane peripherome: visualization and analysis of interactions.<ul><li>The interaction network created from this dataset has been visualized using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25055025">25055025</a> Integrative analysis of ocular complications in atherosclerosis unveils pathway convergence and crosstalk.<ul><li>Network topological analysis was carried out using <b>Cytoscape</b>, while network nodes were annotated using database for annotation, visualization and integrated discovery in order to identify the most enriched pathways and processes. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25051057">25051057</a> Visualized gene network reveals the novel target transcripts Sox2 and Pax6 of neuronal development in trans-placental exposure to bisphenol A.<ul><li>Visualized gene network of <b>Cytoscape</b> analysis showed that Sox2 and Pax6 which were contributed to neural precursor cell proliferation and neuronal differentiation might be down-regulated through sonic hedgehog (Shh), vascular endothelial growth factor A (VEGFA) and Notch signaling.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25040996">25040996</a> Construction of a protein-protein interaction network for chronic myelocytic leukemia and pathway prediction of molecular complexes.<ul><li>Literature mining was performed by Agilent Literature Search plugin and a protein-protein interaction network of chronic myelocytic leukemia was established by <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25003364">25003364</a> Identification of biological targets of therapeutic intervention for diabetic nephropathy with bioinformatics approach.<ul><li>Then we predicted the protein-protein interaction (PPI) pairs regulated by significant miRNAs and constructed the PPI pairs-miRNA network using <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24993113">24993113</a> Key genes associated with osteoporosis revealed by genome wide gene expression analysis.<ul><li>Gene ontology enrichment analysis and Kyoto encyclopedia of genes and genomes pathway analysis were performed with plug-ins of <b>cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24969630">24969630</a> Identification of key genes and crucial modules associated with coronary artery disease by bioinformatics analysis.<ul><li>In addition, expression-activated subnetworks (crucial modules) of the constructed PPI networks were identified using the jActiveModule plug-in, and their topological properties were analyzed using NetworkAnalyzer, both available from <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24959249">24959249</a> Bioinformatics analyses combined microarray identify the deregulated microRNAs in oral cancer.<ul><li>Furthermore, a protein-protein interaction network was constructed using the Search Tools for the Retrieval of Interacting Genes database and <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24949626">24949626</a> IIS--Integrated Interactome System: a web-based platform for the annotation, analysis and visualization of protein-metabolite-gene-drug interactions by integrating a variety of data sources and tools.<ul><li>This module generates a XGMML file that can be imported into <b>Cytoscape</b> or be visualized directly on the web.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24895587">24895587</a> Network of microRNAs-mRNAs interactions in pancreatic cancer.<ul><li>MiRNA-mRNA interactions data analysis and related networks were explored using MAGIA tool and <b>Cytoscape</b> 3 software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24886659">24886659</a> Network analysis identifies protein clusters of functional importance in juvenile idiopathic arthritis.<ul><li>Reactome FI <b>Cytoscape</b> 2.83 Plugin and the Disease Association Protein-Protein Link Evaluator (Dapple) algorithm were used to assess the functionality of the biological pathways within the MEN and to statistically rank the proteins. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24867915">24867915</a> Mechanisms for the adverse effects of late gestational increases in maternal cortisol on the heart revealed by transcriptomic analyses of the fetal septum.<ul><li>To investigate effects of maternal cortisol on the heart, we performed transcriptomic analyses on the septa using ovine microarrays and Webgestalt and <b>Cytoscape</b> programs for pathway inference. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24859971">24859971</a> Screening of critical genes in lung adenocarcinoma via network analysis of gene expression profile.<ul><li>Network clustering was performed with MCL in <b>cytoscape</b> to identify sub-networks and several hub genes were obtained: CDC25C, ICT1, TK1 and EZH2. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24838641">24838641</a> Transduction motif analysis of gastric cancer based on a human signaling network.<ul><li>Topological characteristics of the network were analyzed by <b>CytoScape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24830558">24830558</a> The G protein-coupled receptor heterodimer network (GPCR-HetNet) and its hub components.<ul><li>The interaction data were used to build and analyze the network using <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24829448">24829448</a> ToppMiR: ranking microRNAs and their mRNA targets based on biological functions and context.<ul><li>Results of these analyses are provided as downloadable tables or network file formats usable in <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24788790">24788790</a> ModuleRole: a tool for modulization, role determination and visualization in protein-protein interaction networks.<ul><li>In order to gain insight into the organization and structure of the resultant large complex networks formed by interacting molecules, using simulated annealing, a method based on the node connectivity, we developed ModuleRole, a user-friendly web server tool which finds modules in PPI network and defines the roles for every node, and produces files for visualization in <b>Cytoscape</b> and Pajek. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24763884">24763884</a> Identified differently expressed genes in renal cell carcinoma by using multiple microarray datasets running head: differently expressed genes in renal cell carcinoma.<ul><li>Protein-protein interaction network was constructed used <b>Cytoscape</b> software.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24754179">24754179</a> [Network pharmacology study on major active compounds of siwu decoction analogous formulae for treating primary dysmenorrhea of gynecology blood stasis syndrome].<ul><li>The targets information access was used to construct and visualize components-targets-pathways network model using the kyoto encyclopedia of genes and genomes (KEGG) pathway database and <b>Cytoscape</b> software.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24743993">24743993</a> Identifying stem cell gene expression patterns and phenotypic networks with AutoSOME.<ul><li>Here, we present a facile primer demonstrating the use of AutoSOME for identification and characterization of stem cell gene expression signatures and for visualization of transcriptome networks using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24736319">24736319</a> Network analysis of inflammatory genes and their transcriptional regulators in coronary artery disease.<ul><li>Initially, we obtained 124 candidate genes associated with inflammation and CAD using Polysearch and CADgene database for which protein-protein interaction network was generated using STRING 9.0 (Search Tool for the Retrieval of Interacting Genes) and visualized using <b>Cytoscape</b> v 2.8.3. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24728565">24728565</a> Uncovering the pathogenesis and identifying novel targets of pancreatic cancer using bioinformatics approach.<ul><li>Followed with protein-protein interaction (PPI) network construction by STRING and <b>Cytoscape</b>, module analysis was conducted using ClusterONE. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24724603">24724603</a> Meningioma protein-protein interaction network.<ul><li>The networks of related differentially expressed genes were explored using <b>cytoscape</b> and the PPI analysis methods such as MCODE and ClueGO. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24718810">24718810</a> Module-based functional pathway enrichment analysis of a protein-protein interaction network to study the effects of intestinal microbiota depletion in mice.<ul><li>A protein-protein interaction (PPI) network was constructed for the DEGs using the <b>Cytoscape</b> software, and the network was divided into several modules using the MCODE plugin. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24713438">24713438</a> MetDisease--connecting metabolites to diseases via literature.<ul><li>We present a new <b>Cytoscape</b> 3 plug-in, MetDisease, which uses an alternative approach to link metabolites to disease information. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24713438">24713438</a> MetDisease--connecting metabolites to diseases via literature.<ul><li>MetDisease can be downloaded from http://apps.<b>cytoscape</b>.org/apps/metdisease or installed via the <b>Cytoscape</b> app manager. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24713437">24713437</a> HyperModules: identifying clinically and phenotypically significant network modules with disease mutations for biomarker discovery.<ul><li>HyperModules is available in <b>Cytoscape</b> App Store and as a command line tool at www.baderlab.org/Sofware/HyperModules.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24700371">24700371</a> CDK2 and CDK4 play important roles in promoting the proliferation of SKOV3 ovarian carcinoma cells induced by tumor-associated macrophages.<ul><li>Hence, 31 genes in module I were analyzed with <b>Cytoscape</b> software to generate a gene-function network, which revealed that four gene proteins (E2F1, RB1, CDK2 and CDK4) were functional. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24695521">24695521</a> RNA-Seq and microarrays analyses reveal global differential transcriptomes of Mesorhizobium huakuii 7653R between bacteroids and free-living cells.<ul><li>For a global analysis, we reconstructed a protein-protein interaction (PPI) network for 7653R and integrated gene expression data into the network using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24668670">24668670</a> Module function and two-way clustering analysis of Epstein-Barr virus-related nasopharyngeal cancer.<ul><li>The co-expression coefficient was calculated to construct a co-expression network using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24615040">24615040</a> Screening of key genes of unruptured intracranial aneurysms by using DNA microarray data analysis techniques.<ul><li>We then used the plugins of the <b>Cytoscape</b> software, Mcode and Bingo, to conduct a module analysis. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24603189">24603189</a> Revealing the underlying mechanism of ischemia reperfusion injury using bioinformatics approach.<ul><li>miRNA-target gene network pairs were predicted using WebGestalt, and protein-protein interactions (PPI) were identified based on STRING database, followed by the networks construction using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24586448">24586448</a> Node interference and robustness: performing virtual knock-out experiments on biological networks: the case of leukocyte integrin activation network.<ul><li>A new <b>Cytoscape</b> plug-in calculating these measures was developed and applied to a protein-protein interaction network specifically regulating integrin activation in human primary leukocytes. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24578538">24578538</a> PARK2 and proinflammatory/anti-inflammatory cytokine gene interactions contribute to the susceptibility to leprosy: a case-control study of North Indian population.<ul><li>Protein-protein interaction networks were constructed using <b>cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24577727">24577727</a> Modular analysis of bioinformatics demonstrates a critical role for NF-κB in macrophage activation.<ul><li>A network was constructed using the <b>Cytoscape</b> software. <li>Small molecules capable of regulating these five genes were identified via the CMap software, and a network diagram was generated using the <b>Cytoscape</b> software to provide a reference for the development of new drugs that regulate macrophage activation.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24573512">24573512</a> RASSF1A promotes apoptosis and suppresses the proliferation of ovarian cancer cells.<ul><li>A RASSF1A related protein-protein interaction (PPI) network was then constructed using STRING and <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24452951">24452951</a> Identification of typical miRNAs and target genes in hepatocellular carcinoma by DNA microarray technique.<ul><li>Finally, we made module analysis by using <b>Cytoscape</b> software and its plugins--MCODE and BiNGO.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24452942">24452942</a> Screening of genes related to acute coronary syndrome and identification of functional modules in the PPI network.<ul><li>At last, we used two plugins of <b>Cytoscape</b> to map the functional modules on the base of the PPI network.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24443383">24443383</a> ProbMetab: an R package for Bayesian probabilistic annotation of LC-MS-based metabolomics.<ul><li>This graph can be overlaid with other graph-based analysis, such as partial correlation networks, in a visualization scheme exported to <b>Cytoscape</b>, with web and stand-alone versions.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24430173">24430173</a> A gene pathway analysis highlights the role of cellular adhesion molecules in multiple sclerosis susceptibility.<ul><li>Using SNP association P-values from eight multiple sclerosis (MS) GWAS data sets, we performed a candidate pathway analysis for MS susceptibility by considering genes interacting in the cell adhesion molecule (CAMs) biological pathway using <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24418523">24418523</a> Colorectal cancer candidate biomarkers identified by tissue secretome proteome profiling.<ul><li>Subsequent data analysis was based on label-free spectral counting, Ingenuity Pathway Analysis, Secretome/SignalP, STRING and <b>Cytoscape</b>, resulting in 2703 protein identifications in the tissue secretomes, of which 409 proteins were significantly more present in CRC samples than in controls. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24398765">24398765</a> Revealing the molecular mechanism of colorectal cancer by establishing LGALS3-related protein-protein interaction network and identifying signaling pathways.<ul><li>The LGALS3-related protein-protein interaction network (PPI) was established by STRING and <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26594322">26594322</a> Biological network analysis with CentiScaPe: centralities and experimental dataset integration.<ul><li>Here we present CentiScaPe, a <b>Cytoscape</b> app specifically designed to calculate centrality indexes used for the identification of the most important nodes in a network. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25948119">25948119</a> Identification by microarray technology of key genes involved in the progression of carotid atherosclerotic plaque.<ul><li>A gene coexpression network was constructed with information from COXPRESdb and then visualized with <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24366584">24366584</a> Screening for transcription factors and their regulatory small molecules involved in regulating the functions of CL1-5 cancer cells under the effects of macrophage-conditioned medium.<ul><li>GO and KEGG enrichment analyses were carried out using DAVID software, and visualization networks were constructed using <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24363376">24363376</a> JEPETTO: a <b>Cytoscape</b> plugin for gene set enrichment and topological analysis based on interaction networks.<ul><li>JEPETTO (Java Enrichment of Pathways Extended To TOpology) is a <b>Cytoscape</b> 3.x plugin performing integrative human gene set analysis. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24363376">24363376</a> JEPETTO: a <b>Cytoscape</b> plugin for gene set enrichment and topological analysis based on interaction networks.<ul><li>Installable directly via <b>Cytoscape</b> plugin manager. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24340000">24340000</a> CyTargetLinker: a <b>cytoscape</b> app to integrate regulatory interactions in network analysis.<ul><li>We developed CyTargetLinker (http://projects.bigcat.unimaas.nl/cytargetlinker), a <b>Cytoscape</b> app, for integrating regulatory interactions in network analysis. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24340000">24340000</a> CyTargetLinker: a <b>cytoscape</b> app to integrate regulatory interactions in network analysis.<ul><li>Importantly the incorporation of our tool into the <b>Cytoscape</b> framework allows the application of CyTargetLinker in combination with a wide variety of other apps for state-of-the-art network analysis.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24338419">24338419</a> Screening for feature genes associated with hereditary hemochromatosis and functional analysis with DNA microarrays.<ul><li>Finally, we used the plugins of the <b>Cytoscape</b> software, i.e., Mcode and Bingo, to conduct module analysis. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24304957">24304957</a> [Expression and bioinformatic analysis of miR-29 family, target gene IFN-γ in CD4(+) T cells from subjects with latent tuberculosis infection].<ul><li>Target genes of miR-29 family were predicted with both TargetScan and PicTar.GO annotation and pathway overrepresentation were further analyzed with David database and <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24290289">24290289</a> Nasal mucus proteomic changes reflect altered immune responses and epithelial permeability in patients with allergic rhinitis.<ul><li>Gene enrichment analysis was performed by using <b>Cytoscape</b>/BINGO software tools. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24280356">24280356</a> Ets-1 global gene expression profile reveals associations with metabolism and oxidative stress in ovarian and breast cancers.<ul><li>Methods included over-representation analysis, functional class scoring and pathway topology, and network representations were visualized in <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24265686">24265686</a> PANADA: protein association network annotation, determination and analysis.<ul><li>Here we present PANADA, a novel toolkit for the visualization and analysis of protein similarity networks in <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24254565">24254565</a> Gene expression signature analysis and protein-protein interaction network construction of spinal cord injury.<ul><li>Protein-Protein interaction (PPI) network was constructed by STRING and visualized in <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24233784">24233784</a> Integrating in silico resources to map a signaling network.<ul><li>Furthermore, we provide a simple step-by-step protocol for building customized protein-protein interaction networks in <b>Cytoscape</b>, a powerful network assembly and visualization program, integrating data retrieved from these various databases. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24225050">24225050</a> 3DScapeCS: application of three dimensional, parallel, dynamic network visualization in <b>Cytoscape</b>.<ul><li>We developed a 3D <b>Cytoscape</b> Client/Server (3DScapeCS) plugin, which adopted <b>Cytoscape</b> in interpreting different types of data, and UbiGraph for three-dimensional visualization. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24222250">24222250</a> Protein-protein interaction network and significant gene analysis of osteoporosis.<ul><li>We used the <b>Cytoscape</b> plug-in unit, Cluster ONE, to perform cluster module analysis to find hub proteins of the network module and to analyze their Gene Ontology (GO) functions. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24217020">24217020</a> Construction and application of a protein interaction map for white spot syndrome virus (WSSV).<ul><li>The program <b>Cytoscape</b> was employed to create a WSSV protein-protein interaction (PPI) network. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24206562">24206562</a> CellFateScout - a bioinformatics tool for elucidating small molecule signaling pathways that drive cells in a specific direction.<ul><li>The CellFateScout <b>Cytoscape</b> plugin, a tutorial and the Human Small Molecule Mechanisms Database are available at https://sourceforge.net/projects/cellfatescout/ under LGPLv2 license.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24198161">24198161</a> Study of differential effects of TGF-beta3/BMP2 on chondrogenesis in MSC cells by gene microarray data analysis.<ul><li>The protein-protein interaction (PPI) networks of the DEGs were constructed using <b>cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24156242">24156242</a> VAN: an R package for identifying biologically perturbed networks via differential variability analysis.<ul><li>Additionally, VAN supports the visualization of changes to network modules in R and <b>Cytoscape</b> (visualizeNetwork and obtainPairSubset, respectively). </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24155070">24155070</a> Effect of curcumin on aged Drosophila melanogaster: a pathway prediction analysis.<ul><li>Two <b>Cytoscape</b> plugins including BisoGenet and molecular complex detection (MCODE) were used to establish the protein-protein interaction (PPI) network based upon differential genes in order to detect highly connected regions. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24148319">24148319</a> Comparative analysis of resistant and susceptible macrophage gene expression response to Leishmania major parasite.<ul><li>Using GeneMapp <b>Cytoscape</b>, we showed that the induced genes unique to BALB/c or C57BL/6 BMdMs are interconnected. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24146757">24146757</a> SIDD: a semantically integrated database towards a global view of human disease.<ul><li>Furthermore, a network visualization tool using <b>Cytoscape</b> Web plugin has been implemented in SIDD. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24109950">24109950</a> Comparison of functional network integrity in TBI and orthopedic controlpatientsusing graph-theoretical analysis.<ul><li>Network graphs using a spring-embedded layout (<b>Cytoscape</b>) and a 3D layout integrated into the anatomical space (Paraview) were created. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24089221">24089221</a> Bioinformatic analysis of microarray data reveals several key genes related to heart failure.<ul><li>The overlapping part of the network was selected out using <b>Cytoscape</b>, for which functional enrichment analysis was applied with DAVID tools.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24083701">24083701</a> Screening of differentially expressed genes related to bladder cancer and functional analysis with DNA microarray.<ul><li>Finally, by using plugins of software <b>Cytoscape</b>, Mcode and Bingo, module analysis of hub-genes was performed.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24051510">24051510</a> Molecular and gene network analysis of thyroid transcription factor 1 (TTF1) and enhanced at puberty (EAP1) genes in patients with GnRH-dependent pubertal disorders.<ul><li>Association of TTF1 and EAP1 to genes implicated in timing of puberty was investigated by meta-network framework GeneMANIA and <b>Cytoscape</b> software.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24009617">24009617</a> Navigating the transcriptional roadmap regulating plant secondary cell wall deposition.<ul><li>We include a manipulable <b>Cytoscape</b> representation of the Arabidopsis SCW transcriptional network to aid in future studies, along with a list of supporting literature for each documented interaction. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23994140">23994140</a> Immune signal transduction in leishmaniasis from natural to artificial systems: role of feedback loop insertion.<ul><li><b>Cytoscape</b> and adjacency matrix were used to calculate network topology. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25509360">25509360</a> [[Screening of differentially expressed miRNAs related to muscle strain and their target genes].<ul><li>Finally, <b>Cytoscape</b> and its plugins were used to identify and analyze the modules in this network. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23967151">23967151</a> Meta-analysis of genetic programs between idiopathic pulmonary fibrosis and sarcoidosis.<ul><li>Gene enrichments with their corresponding functions were analyzed by the online analysis engine &quot;Database for Annotation, Visualization and Integrated Discovery&quot; (DAVID) 6.7, and genes interactions and functional networks were further analyzed by STRING 9.0 and <b>Cytoscape</b> 3.0.0 Beta1. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23964737">23964737</a> Unraveling the phosphoproteome dynamics in mammal mitochondria from a network perspective.<ul><li>To integrate data from distinct research papers, we performed network analysis, with bioinformatic tools like <b>Cytoscape</b>, String, and PANTHER taking into consideration variables such as tissue specificity, biological processes, molecular functions, and pathophysiological conditions. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23956138">23956138</a> Characterization of the EGFR interactome reveals associated protein complex networks and intracellular receptor dynamics.<ul><li>The newly developed <b>Cytoscape</b> plugin ModuleGraph allowed retrieving and visualizing 93 well-described protein complexes that contained at least one of the proteins found to interact with EGFR in our experiments. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23948412">23948412</a> [Expression and function prediction of miR-618 in sputa and sera of patients with pulmonary tuberculosis].<ul><li>Target genes of miR-618 were predicted with TargetSean, and GO annotation and biological pathway of the target genes were analyzed with DAVID database and BINGO of <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23908262">23908262</a> SPNConverter: a new link between static and dynamic complex network analysis.<ul><li>However, analysis of complex networks represented using other systems biology formatting languages (on which popular software, such as CellDesigner and <b>Cytoscape</b>, are based) requires manual manipulation, a step that is prone to error and limits the use of the SPN simulator in BioLayout Express(3D). </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23908249">23908249</a> Facilitating the use of large-scale biological data and tools in the era of translational bioinformatics.<ul><li>We review recent applications of <b>Cytoscape</b>, BioGPS and DAVID for data visualization, integration and functional enrichment. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23894138">23894138</a> GALANT: a <b>Cytoscape</b> plugin for visualizing data as functional landscapes projected onto biological networks.<ul><li>In a continuing effort to create clear and integrated visualizations that facilitate the gathering of novel biological insights despite the overwhelming complexity of data, we present here the GrAph LANdscape VisualizaTion (GALANT), a <b>Cytoscape</b> plugin that builds functional landscapes onto biological networks. <li>As a <b>Cytoscape</b> plugin, GALANT is further improved by the functionalities of <b>Cytoscape</b>, the popular bioinformatics package for biological network visualization and data integration.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23859268">23859268</a> Reconn: a <b>cytoscape</b> plug-in for exploring and visualizing reactome.<ul><li>ReConn (Reactome Connector) is an open source extension to <b>Cytoscape</b> which allows user friendly interaction with the Reactome database. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23856141">23856141</a> [Expression of miR-29a in serum of patients with pulmonary tuberculosis and prediction of its function with bioinformatics analysis].<ul><li>GO overrepresentation was further analyzed by BINGO of <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23855714">23855714</a> SLiMScape: a protein short linear motif analysis plugin for <b>Cytoscape</b>.<ul><li><b>Cytoscape</b> provides a visual interface for protein networks but there is no streamlined way to rapidly visualize motifs in a network of proteins, or to integrate computational discovery with such visualizations.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23855714">23855714</a> SLiMScape: a protein short linear motif analysis plugin for <b>Cytoscape</b>.<ul><li>We present SLiMScape, a <b>Cytoscape</b> plugin, which enables both de novo motif discovery and searches for instances of known motifs. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23847684">23847684</a> Pivotal regulatory network and genes in osteosarcoma.<ul><li>We explored the regulation relationship between transcription factors and target genes using <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23846596">23846596</a> T2D@ZJU: a knowledgebase integrating heterogeneous connections associated with type 2 diabetes mellitus.<ul><li>A <b>Cytoscape</b> Web-based interactive network browser is available to visualize the corresponding network relationships between T2D-related entities. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23837656">23837656</a> R<b>Cytoscape</b>: tools for exploratory network analysis.<ul><li>R<b>Cytoscape</b> integrates R (an open-ended programming environment rich in statistical power and data-handling facilities) and <b>Cytoscape</b> (powerful network visualization and analysis software). <li>To illustrate the power of R<b>Cytoscape</b>, a portion of the Glioblastoma multiforme (GBM) data set from the Cancer Genome Atlas (TCGA) is examined. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23837656">23837656</a> R<b>Cytoscape</b>: tools for exploratory network analysis.<ul><li>Network visualizations--molecular maps--created from an open-ended programming environment rich in statistical power and data-handling facilities, such as R<b>Cytoscape</b>, will play an essential role in this progression.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23771539">23771539</a> Bioinformatics analyses combined microarray identify the desregulated microRNAs in lung cancer.<ul><li>Further, a protein-protein interaction (PPI) network was constructed used STRING and <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23749961">23749961</a> NOA: a <b>cytoscape</b> plugin for network ontology analysis.<ul><li>The Network Ontology Analysis (NOA) plugin for <b>Cytoscape</b> implements the NOA algorithm for network-based enrichment analysis, which extends Gene Ontology annotations to network links, or edges. <li>In addition to tables, the NOA plugin also presents results in the form of heatmaps and overview networks in <b>Cytoscape</b>, which can be exported for publication figures.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23749961">23749961</a> NOA: a <b>cytoscape</b> plugin for network ontology analysis.<ul><li>The NOA plugin is an open source, Java program for <b>Cytoscape</b> version 2.8 available via the <b>Cytoscape</b> App Store (http://apps.<b>cytoscape</b>.org/apps/noa) and plugin manager. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23749465">23749465</a> Construction of protein phosphorylation networks by data mining, text mining and ontology integration: analysis of the spindle checkpoint.<ul><li>The integrated approach involves (i) text mining guided by RLIMS-P, a tool that identifies phosphorylation-related information in scientific literature; (ii) data mining from curated PPI databases; (iii) protein form and complex representation using the Protein Ontology (PRO); (iv) functional annotation using the Gene Ontology (GO); and (v) network visualization and analysis with <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23743336">23743336</a> Clust&amp;See: a <b>Cytoscape</b> plugin for the identification, visualization and manipulation of network clusters.<ul><li>We developed Clust&amp;See, a <b>Cytoscape</b> plugin dedicated to the identification, visualization and analysis of clusters extracted from such networks.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23731710">23731710</a> Screening of osteoprotegerin-related feature genes in osteoporosis and functional analysis with DNA microarray.<ul><li>The interactions among the osteoprotegerin gene (OPG) and differentially expressed genes were searched and visualized by <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23731466">23731466</a> Microarray analysis of microRNA expression in peripheral blood mononuclear cells of critically ill patients with influenza A (H1N1).<ul><li>Furthermore, an integrative network of miRNA-mediated host-influenza virus protein interactions was constructed by integrating the predicted and validated miRNA-gene interaction data with influenza virus and host-protein-protein interaction information using <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23725466">23725466</a> PMTED: a plant microRNA target expression database.<ul><li>Networks are displayed through a <b>Cytoscape</b> Web-based graphical interface. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23724694">23724694</a> [Chemoinformatics study on antibacterial activity of traditional Chinese medicine compounds].<ul><li><b>Cytoscape</b> and its plug-in ChemViz were applied to compute the 2D chemical structure similarity and topological parameter TPSA (topological molecular polar surface area), which measures cell permeability of chemicals, between TCM compounds and clinical antibacterials. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23715980">23715980</a> Visualization and analysis of biological networks.<ul><li>We will be making use of the popular open source tool <b>Cytoscape</b> and of other resources such as : the PSICQUIC client to access several protein interaction repositories and the BiNGO plugin to perform GO enrichment analysis of the resulting network.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23715979">23715979</a> Integration of genomic information with biological networks using <b>Cytoscape</b>.<ul><li><b>Cytoscape</b> is an open-source software for visualizing, analyzing, and modeling biological networks. <li>The chapter is divided into five parts: (1) obtaining information about the functional effect of sequence variation in a <b>Cytoscape</b> readable format, (2) loading and displaying different types of biological networks in <b>Cytoscape</b>, (3) integrating the genomic information (SNPs and mutations) with the biological networks, and (4) analyzing the effect of the genomic perturbation onto the network structure using <b>Cytoscape</b> built-in functions. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23710289">23710289</a> NRF2-ome: an integrated web resource to discover protein interaction and regulatory networks of NRF2.<ul><li>The database can be downloaded in SQL, CSV, BioPAX, SBML, PSI-MI, and in a <b>Cytoscape</b> CYS file formats. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23627759">23627759</a> ContigScape: a <b>Cytoscape</b> plugin facilitating microbial genome gap closing.<ul><li>We have developed a <b>Cytoscape</b> plugin to facilitate gap closing for high-throughput sequencing data from microbial genomes. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23627759">23627759</a> ContigScape: a <b>Cytoscape</b> plugin facilitating microbial genome gap closing.<ul><li>Displaying relationships between contigs using graphs in <b>Cytoscape</b> rather than tables provides a more straightforward visual representation. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23626000">23626000</a> OCSANA: optimal combinations of interventions from network analysis.<ul><li>The latest version of OCSANA, implemented as a plugin for <b>Cytoscape</b> and distributed under LGPL license, is available together with source code at http://bioinfo.curie.fr/projects/ocsana.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23625999">23625999</a> NetworkTrail--a web service for identifying and visualizing deregulated subnetworks.<ul><li>NetworkTrail uses a state-of-the-art integer linear programming-based approach for this task and offers interfaces to the Biological Network Analyzer (BiNA) and <b>Cytoscape</b> Web for visualizing the resulting subnetworks. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23620364">23620364</a> HOMECAT: consensus homologs mapping for interspecific knowledge transfer and functional genomic data integration.<ul><li>With HOMECAT, we aim to facilitate cross-species relationship identification and data mapping, by combining orthology predictions from several publicly available sources, a convenient interface for high-throughput data download and automatic identifier conversion into a <b>Cytoscape</b> plug-in, that provides both an integration with a large set of bioinformatics tools, as well as a user-friendly interface.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23616008">23616008</a> A guide to CORNET for the construction of coexpression and protein-protein interaction networks.<ul><li>Networks and associated evidence of the majority of the currently available data types are visualized in <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23597302">23597302</a> Reaction schemes visualized in network form: the syntheses of strychnine as an example.<ul><li>An example of data formatting that allows construction of network maps in <b>Cytoscape</b> is presented, along with maps that illustrate the comparison of multiple reaction sequences, comparison of scaffold changes within sequences, and consolidation to highlight common key intermediates used across sequences. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23595664">23595664</a> <b>Cytoscape</b> app store.<ul><li><b>Cytoscape</b> is an open source software tool for biological network visualization and analysis, which can be extended with independently developed apps. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23595661">23595661</a> NetworkPrioritizer: a versatile tool for network-based prioritization of candidate disease genes or other molecules.<ul><li>Therefore, we developed the <b>Cytoscape</b> plugin NetworkPrioritizer that particularly supports the integrative network-based prioritization of candidate disease genes or other molecules. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23567935">23567935</a> Health-related effects reported by electronic cigarette users in online forums.<ul><li>Data were collected into Microsoft Access databases and analyzed using <b>Cytoscape</b> association graphics, frequency distributions, and interactomes to determine the number and type of health effects reported, the organ systems affected the frequency of specific effects, and systems interactions.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23552858">23552858</a> PodNet, a protein-protein interaction network of the podocyte.<ul><li>These networks do not exist for the podocyte; therefore, we built PodNet, a literature-based mouse podocyte network in <b>Cytoscape</b> format. <li>Transcriptomes of podocytes in 10 different states were mapped on XPodNet and analyzed with the <b>Cytoscape</b> plugin ExprEssence, based on the law of mass action. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23543444">23543444</a> The PPI network and cluster ONE analysis to explain the mechanism of bladder cancer.<ul><li>In our study, we constructed the Protein-Protein Interactions (PPI) network using the <b>Cytoscape</b> and detected some network modeling clusters. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23453054">23453054</a> BiNoM 2.0, a <b>Cytoscape</b> plugin for accessing and analyzing pathways using standard systems biology formats.<ul><li>We have developed BiNoM (Biological Network Manager), a <b>Cytoscape</b> plugin, which provides functions for the import-export of some standard systems biology file formats (import from CellDesigner, BioPAX Level 3 and CSML; export to SBML, CellDesigner and BioPAX Level 3), and a set of algorithms to analyze and reduce the complexity of biological networks. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23443765">23443765</a> [Differences in peripheral blood gene expression profiles between liver and kidney transplantation recipients].<ul><li>By combining gene set enrichment analysis (GSEA) and biological network analysis of the differentially expressed genes using <b>Cytoscape</b> software, we analyzed the core genes closely related to liver or kidney transplantations.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23443765">23443765</a> [Differences in peripheral blood gene expression profiles between liver and kidney transplantation recipients].<ul><li>Fourteen hub nodes (gene) for liver transplantation and 13 for kidney transplantation were identified by <b>cytoscape</b> software using interaction network analysis. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23427985">23427985</a> Computational construction of specialized biological networks.<ul><li>NcDocReSy is implemented as a <b>Cytoscape</b> plugin.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23410028">23410028</a> Pivotal role of the muscle-contraction pathway in cryptorchidism and evidence for genomic connections with cardiomyopathy pathways in RASopathies.<ul><li><b>Cytoscape</b> plug-in PiNGO 1.11 was employed for protein-network-based prediction of novel candidate genes. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23398688">23398688</a> A survey of protein interaction data and multigenic inherited disorders.<ul><li>Disease group overlaps can be further explored using the iRefscape plugin for <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23368786">23368786</a> Visualization of protein interaction networks: problems and solutions.<ul><li><b>Cytoscape</b>), that may be incrementally enriched by the interactomics community with novel and more powerful functions for PIN analysis, through the development of plug-ins. <li>From a technological point of view, open-source, free and extensible tools, like <b>Cytoscape</b>, guarantee a long term sustainability due to the largeness of the developers and users communities, and provide a great flexibility since new functions are continuously added by the developer community through new plug-ins, but the emerging parallel, often closed-source tools like NAViGaTOR, can offer near real-time response time also in the analysis of very huge PINs.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23368516">23368516</a> Expanding the boundaries of local similarity analysis.<ul><li>We computationally compare our approach to previous implementations of LSA, demonstrate broad applicability by analyzing time series data from public health, microbial ecology, and social media, and visualize resulting networks using the <b>Cytoscape</b> software.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23390349">23390349</a> Identification of conserved drought stress responsive gene-network across tissues and developmental stages in rice.<ul><li>In the present study, we analyzed the temporal-spatial coordination of gene expression in root, leaf and panicle of rice under drought stress and constructed network using WGCNA and <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23363777">23363777</a> Visualizing multidimensional cancer genomics data.<ul><li>The review covers visualization methods employed by tools such as Circos, Gitools, the Integrative Genomics Viewer, <b>Cytoscape</b>, Savant Genome Browser, StratomeX and platforms such as cBio Cancer Genomics Portal, IntOGen, the UCSC Cancer Genomics Browser, the Regulome Explorer and the Cancer Genome Workbench.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23347958">23347958</a> Bioinformatics and systems biology analysis of genes network involved in OLP (Oral Lichen Planus) pathogenesis.<ul><li>This study was complemented by a topological analysis of the network, carried out using <b>Cytoscape</b>, BinGO and FANMOD software.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23325622">23325622</a> CluePedia <b>Cytoscape</b> plugin: pathway insights using integrated experimental and in silico data.<ul><li>The CluePedia <b>Cytoscape</b> plugin is a search tool for new markers potentially associated to pathways. <li>The CluePedia <b>Cytoscape</b> plugin is user-friendly and has an expressive and intuitive visualization.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23325622">23325622</a> CluePedia <b>Cytoscape</b> plugin: pathway insights using integrated experimental and in silico data.<ul><li>http://www.ici.upmc.fr/cluepedia/ and via the <b>Cytoscape</b> plugin manager. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23324051">23324051</a> CySBGN: a <b>Cytoscape</b> plug-in to integrate SBGN maps.<ul><li>However, support for SBGN in <b>Cytoscape</b>, one of the most widely used platforms in biology to visualise and analyse networks, is limited, and in particular it is not possible to import SBGN diagrams.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23324051">23324051</a> CySBGN: a <b>Cytoscape</b> plug-in to integrate SBGN maps.<ul><li>We have developed CySBGN, a <b>Cytoscape</b> plug-in that extends the use of <b>Cytoscape</b> visualisation and analysis features to SBGN maps. <li>To illustrate the interoperability with other <b>Cytoscape</b> functionalities, we present two analysis examples, shortest path calculation, and motif identification in a metabolic network.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23324051">23324051</a> CySBGN: a <b>Cytoscape</b> plug-in to integrate SBGN maps.<ul><li>CySBGN imports, modifies and analyzes SBGN diagrams in <b>Cytoscape</b>, and thus allows the application of the large palette of tools and plug-ins in this platform to networks and pathways in SBGN format.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23281970">23281970</a> Integration of interactive, multi-scale network navigation approach with <b>Cytoscape</b> for functional genomics in the big data era.<ul><li><b>Cytoscape</b> is a de facto standard platform for network visualization and analysis, which has many users around the world. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23281970">23281970</a> Integration of interactive, multi-scale network navigation approach with <b>Cytoscape</b> for functional genomics in the big data era.<ul><li>We developed NaviClusterCS, which enables researchers to interactively navigate large biological networks of ~100,000 nodes in a &quot;Google Maps-like&quot; manner in the <b>Cytoscape</b> environment. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23297033">23297033</a> RamiGO: an R/Bioconductor package providing an AmiGO visualize interface.<ul><li>Given a list of GO terms, RamiGO uses the AmiGO visualize API to import Graphviz-DOT format files into R, and export these either as images (SVG, PNG) or into <b>Cytoscape</b> for extended network analyses. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23292740">23292740</a> GESTODIFFERENT: a <b>Cytoscape</b> plugin for the generation and the identification of gene regulatory networks describing a stochastic cell differentiation process.<ul><li>GeStoDifferent is a <b>Cytoscape</b> plugin aimed at the generation and the identification of gene regulatory networks (GRNs) describing an arbitrary stochastic cell differentiation process. <li>Matched networks can then be analyzed by <b>Cytoscape</b> network analysis algorithms or, for instance, used to define (multiscale) models of cellular dynamics.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23292740">23292740</a> GESTODIFFERENT: a <b>Cytoscape</b> plugin for the generation and the identification of gene regulatory networks describing a stochastic cell differentiation process.<ul><li>Freely available at http://bimib.disco.unimib.it/index.php/Retronet#GESTODifferent or at the <b>Cytoscape</b> App Store http://apps.<b>cytoscape</b>.org/.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23274719">23274719</a> Proteomic identification network analysis of haptoglobin as a key regulator associated with liver fibrosis.<ul><li>HP protein was subjected to functional pathway analysis using STRING and <b>Cytoscape</b> software for better understanding of the protein-protein interaction (PPI) networks in biological context. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24715956">24715956</a> Cyrface: An interface from <b>Cytoscape</b> to R that provides a user interface to R packages.<ul><li>Here, we present a <b>Cytoscape</b> app termed Cyrface that allows <b>Cytoscape</b> apps to connect to any function and package developed in R. <li>Moreover, it can link R packages with the capabilities of <b>Cytoscape</b> and its apps, in particular network visualization and analysis. <li>Download links, installation instructions and user guides can be accessed from the Cyrface's homepage ( http://www.ebi.ac.uk/saezrodriguez/cyrface/) and from the <b>Cytoscape</b> app store ( http://apps.<b>cytoscape</b>.org/apps/cyrface).</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23261424">23261424</a> Discovering gene-environment interactions in glioblastoma through a comprehensive data integration bioinformatics method.<ul><li>This comparison produced a list of genes responsive to the environment and important to GBM that was then further analyzed using gene networking tools such as RSpider, <b>Cytoscape</b>, and DAVID. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23228038">23228038</a> Clarifying off-target effects for torcetrapib using network pharmacology and reverse docking approach.<ul><li><b>Cytoscape</b> with three plugins including BisoGenet, NetworkAnalyzer and ClusterONE was utilized to establish a context-specific drug-gene interaction network. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23228031">23228031</a> A network module-based method for identifying cancer prognostic signatures.<ul><li>A <b>Cytoscape</b> plugin implementation of the signature discovery method is available at http://wiki.reactome.org/index.php/Reactome_FI_<b>Cytoscape</b>_Plugin.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23203989">23203989</a> The BioGRID interaction database: 2013 update.<ul><li>BioGRID records can also be interrogated and analyzed with a series of new bioinformatics tools, which include a post-translational modification viewer, a graphical viewer, a REST service and a <b>Cytoscape</b> plugin.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23203868">23203868</a> COXPRESdb: a database of comparative gene coexpression networks of eleven species for mammals.<ul><li>Second, the network drawing function was updated, to implement automatic cluster analyses with enrichment analyses in Gene Ontology and in cis elements, along with interactive network analyses with <b>Cytoscape</b> Web. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23180784">23180784</a> RiceFREND: a platform for retrieving coexpressed gene networks in rice.<ul><li>A user-friendly web interface facilitates visualization and interpretation of gene coexpression networks in HyperTree, <b>Cytoscape</b> Web and Graphviz formats. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23152744">23152744</a> NetView: a high-definition network-visualization approach to detect fine-scale population structures from genome-wide patterns of variation.<ul><li>Furthermore, we optimised an analysis pipeline called NetView, a high-definition network visualization, starting with computation of genetic distance, followed clustering using Spc and finally visualization of clusters with <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23143270">23143270</a> The ConsensusPathDB interaction database: 2013 update.<ul><li>To facilitate the interactive, visual interpretation of interaction and pathway data, we have re-implemented the graph visualization feature of ConsensusPathDB using the <b>Cytoscape</b>.js library.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23132118">23132118</a> A travel guide to <b>Cytoscape</b> plugins.<ul><li><b>Cytoscape</b> is open-source software for integration, visualization and analysis of biological networks. <li>We present a travel guide to the world of plugins, covering the 152 publicly available plugins for <b>Cytoscape</b> 2.5-2.8. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23129297">23129297</a> DvD: An R/<b>Cytoscape</b> pipeline for drug repurposing using public repositories of gene expression data.<ul><li>Drug versus Disease (DvD) provides a pipeline, available through R or <b>Cytoscape</b>, for the comparison of drug and disease gene expression profiles from public microarray repositories. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23129297">23129297</a> DvD: An R/<b>Cytoscape</b> pipeline for drug repurposing using public repositories of gene expression data.<ul><li>R package (submitted to Bioconductor) under GPL 3 and <b>Cytoscape</b> plug-in freely available for download at www.ebi.ac.uk/saezrodriguez/DVD/.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23112803">23112803</a> Mining and visualization of microarray and metabolomic data reveal extensive cell wall remodeling during winter hardening in Sitka spruce (Picea sitchensis).<ul><li>Automated Layout Pipeline for Inferred NEtworks (ALPINE), an in-house plugin for the <b>Cytoscape</b> visualization environment that utilizes the existing GeneMANIA and Mosaic plugins, together with the use of visualization tools, provided images of proposed signaling processes that became active over the time course of winter hardening, particularly at later time points in the process. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23093600">23093600</a> The Comparative Toxicogenomics Database: update 2013.<ul><li>To improve data visualization, we added a <b>Cytoscape</b> Web view to our ChemComps feature, included color-coded interactions and created a 'slim list' for our MEDIC disease vocabulary (allowing diseases to be grouped for meta-analysis, visualization and better data management). </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23079107">23079107</a> CellNOptR: a flexible toolkit to train protein signaling networks to data using multiple logic formalisms.<ul><li>We provide in addition an interface via <b>Cytoscape</b> (CytoCopteR) to facilitate use and integration with <b>Cytoscape</b> network-based capabilities.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23071747">23071747</a> TRIP database 2.0: a manually curated information hub for accessing TRP channel interaction network.<ul><li>Here, we present the TRIP Database 2.0 (http://www.trpchannel.org) in which many helpful, user-friendly web interfaces have been developed to facilitate knowledge acquisition and inspire new approaches to studying TRP channel functions: 1) the PPI information found in the supplementary data of referred articles was curated; 2) the PPI summary matrix enables users to intuitively grasp overall PPI information; 3) the search capability has been expanded to retrieve information from 'PubMed' and 'PIE the search' (a specialized search engine for PPI-related articles); and 4) the PPI data are available as sif files for network visualization and analysis using '<b>Cytoscape</b>'. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23055628">23055628</a> Network analysis of gene lists for finding reproducible prognostic breast cancer gene signatures.<ul><li>The method is implemented in BiNoM <b>Cytoscape</b> plugin (http://binom.curie.fr).</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23042589">23042589</a> A signal transduction score flow algorithm for cyclic cellular pathway analysis, which combines transcriptome and ChIP-seq data.<ul><li>Implementation of the algorithm as a <b>Cytoscape</b> plug-in allows interactive visualization and analysis of KEGG pathways as well as user-generated and curated <b>Cytoscape</b> pathways. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22954630">22954630</a> SAPIN: a framework for the structural analysis of protein interaction networks.<ul><li>Finally, the results are displayed using <b>Cytoscape</b> Web.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22952659">22952659</a> CytoKavosh: a <b>cytoscape</b> plug-in for finding network motifs in large biological networks.<ul><li>CytoKavosh is a <b>Cytoscape</b> plug-in which supports us in finding motifs of given size in a network that is formerly loaded into the <b>Cytoscape</b> work-space (directed or undirected). <li>CytoKavosh is implemented in a visual environment <b>Cytoscape</b> that is convenient for the users to interact and create visual options to analyze the structural behavior of a network. <li>There is no specific <b>Cytoscape</b> plug-in, specific for finding the network motifs, based on original concept. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22900683">22900683</a> HTRIdb: an open-access database for experimentally verified human transcriptional regulation interactions.<ul><li>The HTRIdb is an open-access database that can be searched via a user-friendly web interface and the retrieved TF-TG interactions data and the associated protein-protein interactions can be downloaded or interactively visualized as a network through the web version of the popular <b>Cytoscape</b> visualization tool, the <b>Cytoscape</b> Web. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22854123">22854123</a> Development of an integrated network visualisation and graph analysis tool for biological networks.<ul><li>To that end, this paper presents a newly developed plugin for integrated network visualisation and graph analysis within the <b>Cytoscape</b> framework. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22782548">22782548</a> Gemma: a resource for the reuse, sharing and meta-analysis of expression profiling data.<ul><li>Web services and integration with third-party resources further increase the scope of the tools, which include a <b>Cytoscape</b> plugin.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22772946">22772946</a> CySBML: a <b>Cytoscape</b> plugin for SBML.<ul><li>CySBML is a plugin designed to work with Systems Biology Markup Language (SBML) in <b>Cytoscape</b> having the following features: SBML import, support of the SBML layout and qualitative model packages, navigation in network layouts based on SBML structure, access to MIRIAM and SBO-based annotations and SBML validation. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22772946">22772946</a> CySBML: a <b>Cytoscape</b> plugin for SBML.<ul><li>Freely available for non-commercial purposes through the <b>Cytoscape</b> plugin manager or for download at http://sourceforge.net/projects/cysbml/.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22724003">22724003</a> Molecular effects of doxycycline treatment on pterygium as revealed by massive transcriptome sequencing.<ul><li>Acquired data were subjected to quantile normalization and analyzed using <b>cytoscape</b> plugin software to explore the pathways involved. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22718784">22718784</a> ModuLand plug-in for <b>Cytoscape</b>: determination of hierarchical layers of overlapping network modules and community centrality.<ul><li>The ModuLand plug-in provides <b>Cytoscape</b> users an algorithm for determining extensively overlapping network modules. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22704053">22704053</a> MicroRNA expression signature in human abdominal aortic aneurysms.<ul><li>Networks from the target gene set were generated and examined using the network analysis programs, <b>CytoScape</b>® and Ingenuity Pathway Core Analysis®.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22689386">22689386</a> ETAscape: analyzing protein networks to predict enzymatic function and substrates in <b>Cytoscape</b>.<ul><li>To address this problem, we implemented the Evolutionary Trace Annotation (ETA) method in the <b>Cytoscape</b> network visualization environment. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22651224">22651224</a> CORNET 2.0: integrating plant coexpression, protein-protein interactions, regulatory interactions, gene associations and functional annotations.<ul><li>Networks and associated evidence of the majority of currently available data types are visualized in <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22591066">22591066</a> MetaMapp: mapping and visualizing metabolomic data by integrating information from biochemical pathways and chemical and mass spectral similarity.<ul><li>MetaMapp graphs in <b>Cytoscape</b> showed much clearer metabolic modularity and complete content visualization compared to conventional biochemical mapping approaches. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22591066">22591066</a> MetaMapp: mapping and visualizing metabolomic data by integrating information from biochemical pathways and chemical and mass spectral similarity.<ul><li>MetaMapp graphs efficiently visualizes mass spectrometry based metabolomics datasets as network graphs in <b>Cytoscape</b>, and highlights metabolic alterations that can be associated with higher rate of pulmonary diseases and infections in children prenatally exposed to ETS. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22587372">22587372</a> CytoITMprobe: a network information flow plugin for <b>Cytoscape</b>.<ul><li><b>Cytoscape</b> is a well-developed flexible platform for visualization, integration and analysis of network data. <li>However, plugins for <b>Cytoscape</b> with these features do not yet exist. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22587372">22587372</a> CytoITMprobe: a network information flow plugin for <b>Cytoscape</b>.<ul><li>The input, consisting of a <b>Cytoscape</b> network, together with the desired origins and/or destinations of information and a dissipation coefficient, is specified through a query form. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22587372">22587372</a> CytoITMprobe: a network information flow plugin for <b>Cytoscape</b>.<ul><li>It allows the user to specify as input any <b>Cytoscape</b> network, rather than being restricted to the pre-compiled protein-protein interaction networks available through the ITM Probe web service. <li>It also enables seamless integration of ITM Probe results with other <b>Cytoscape</b> plugins having complementary functionality for data analysis.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22576176">22576176</a> Mosaic: making biological sense of complex networks.<ul><li>We present a <b>Cytoscape</b> plugin called Mosaic to support interactive network annotation, partitioning, layout and coloring based on gene ontology or other relevant annotations.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22576176">22576176</a> Mosaic: making biological sense of complex networks.<ul><li>Mosaic is distributed for free under the Apache v2.0 open source license and can be downloaded via the <b>Cytoscape</b> plugin manager. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22570439">22570439</a> Genomics of estradiol-3-sulfate action in the ovine fetal hypothalamus.<ul><li>Network and enrichment analyses were performed using the <b>Cytoscape</b>/Bingo software, and the results validated by quantitative real-time PCR. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22517619">22517619</a> Conceptus-endometrium crosstalk during maternal recognition of pregnancy in cattle.<ul><li>Using <b>Cytoscape</b> software, a total of 133 conceptus ligands that interact with corresponding receptors on the endometrium and 121 endometrium ligands that interact with corresponding receptors on the conceptus were identified. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22458412">22458412</a> Screening and functional analysis of differentially expressed genes in EBV-transformed lymphoblasts.<ul><li>LIMMA, String, <b>Cytoscape</b> and other softwares were used to screen and analyze differentially expressed genes. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22458412">22458412</a> Screening and functional analysis of differentially expressed genes in EBV-transformed lymphoblasts.<ul><li>According to the results of Generank, String and <b>Cytoscape</b> analyses, 38 genes may be key controlled genes related to EBV-transformed lymphocytes, including 22 up-regulated genes(PLK1, E2F1, AURKB, CDK2, PLCG2, CD80, PIK3R3, CDC20, CDC6, AURKA, CENPA, BUB1B, NUP37, MAD2L1, BIRC5, CDC25A, CCNB1, RPA3, HJURP, KIF2C, CDK1, CDCA8) and 16 down-regulated genes(FYN, CD3D, CD4, CD3G, ZAP70, FOS, HCK, CD247, PRKCQ, ITK, LCP2, CXCL1, CD8A, ITGB5, VAV3, CXCR4), which primarily control biological processes such as cell cycle, mitosis, cytokine-cytokine pathway, immunity response and so on.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22448851">22448851</a> A multilevel layout algorithm for visualizing physical and genetic interaction networks, with emphasis on their modular organization.<ul><li>While a number of layout algorithms are available in popular network analysis platforms, such as <b>Cytoscape</b>, it remains poorly understood how well their solutions reflect the underlying biological processes that give rise to the network connectivity structure. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22448851">22448851</a> A multilevel layout algorithm for visualizing physical and genetic interaction networks, with emphasis on their modular organization.<ul><li>Using a wide variety of real life biological networks, we carried out a systematic evaluation of the method in comparison with other layout algorithms in <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22448851">22448851</a> A multilevel layout algorithm for visualizing physical and genetic interaction networks, with emphasis on their modular organization.<ul><li>The multilevel approach provided both biologically relevant and visually pleasant layout solutions in most network types, hence complementing the layout options available in <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22442671">22442671</a> miR-181a post-transcriptionally downregulates oncogenic RalA and contributes to growth inhibition and apoptosis in chronic myelogenous leukemia (CML).<ul><li>Using the KEGG database combined with recent publications, downstream signaling pathway of RalA was graphed by <b>cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22433437">22433437</a> Predicting new molecular targets for rhein using network pharmacology.<ul><li>We used <b>Cytoscape</b> software for prediction rhein-target interactions, to facilitate the drug discovery pipeline.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22433437">22433437</a> Predicting new molecular targets for rhein using network pharmacology.<ul><li>Results showed that 3 differentially expressed genes confirmed by <b>Cytoscape</b> as the central nodes of the complicated interaction network (99 nodes, 153 edges). </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22430954">22430954</a> GPEC: a <b>Cytoscape</b> plug-in for random walk-based gene prioritization and biomedical evidence collection.<ul><li>Here, we introduce a novel <b>Cytoscape</b> plug-in, GPEC, to help identify putative genes likely to be associated with specific diseases or pathways. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22422314">22422314</a> Topological analysis and interactive visualization of biological networks and protein structures.<ul><li>This protocol describes three workflows based on the NetworkAnalyzer and RINalyzer plug-ins for <b>Cytoscape</b>, a popular software platform for networks. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22411911">22411911</a> How much can we learn about the function of bacterial rRNA modification by mining large-scale experimental datasets?<ul><li>Results were visualized as graphs using <b>Cytoscape</b> and analysed. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22390940">22390940</a> CentiLib: comprehensive analysis and exploration of network centralities.<ul><li>Its architecture allows for easy adaption to Java-based network analysis, simulation and visualization tools, which is demonstrated by providing the plug-in for two popular network analysis tools-<b>Cytoscape</b> and Vanted. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22385417">22385417</a> Analyzing protein-protein interaction networks.<ul><li>Using the molecular interaction database IntAct, the software platform <b>Cytoscape</b>, and its plugins BiNGO and clusterMaker, and taking as a starting point a list of proteins identified in a mass spectrometry-based proteomics experiment, we show how to build, visualize, and analyze a protein-protein interaction network.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22356618">22356618</a> miRTrail--a comprehensive webserver for analyzing gene and miRNA patterns to enhance the understanding of regulatory mechanisms in diseases.<ul><li>For interactively visualizing obtained results, it relies on the network analyzers and viewers BiNA or <b>Cytoscape</b>-web, also enabling direct access to relevant literature. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22353882">22353882</a> Efficient key pathway mining: combining networks and OMICS data.<ul><li>KeyPathwayMiner release 2 is available as a <b>Cytoscape</b> plugin and online at http://keypathwayminer.mpi-inf.mpg.de.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22345616">22345616</a> CytoSaddleSum: a functional enrichment analysis plugin for <b>Cytoscape</b> based on sum-of-weights scores.<ul><li>CytoSaddleSum provides <b>Cytoscape</b> users with access to the functionality of SaddleSum, a functional enrichment tool based on sum-of-weight scores. <li>Furthermore, query results are written as <b>Cytoscape</b> attributes allowing easy saving, retrieval and integration into network-based data analysis workflows.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22342379">22342379</a> Systems biology approach to transplant tolerance: proof of concept experiments using RNA interference (RNAi) to knock down hub genes in Jurkat and HeLa cells in vitro.<ul><li>The molecular interaction networks were then analyzed and visualized with <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22340758">22340758</a> Global gene expression profiling displays a network of dysregulated genes in non-atherosclerotic arterial tissue from patients with type 2 diabetes.<ul><li>Gene expression changes were integrated with GO-Elite, GSEA, and <b>Cytoscape</b> to identify significant biological pathways and networks.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22308675">22308675</a> [Tools for network pharmacology study: network visualization and network analysis].<ul><li>In this paper, we generally showed the work flow of network pharmacology research, and introduced concepts of network visualization and network analysis, as well the commonly-used tools, such as <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22308672">22308672</a> [Network pharmacology study on major active compounds of Fufang Danshen formula].<ul><li>Combined with cardiovascular related diseases and genes from OMIM database, the corresponding multi-compound-multi- target-multi-disease network was constructed and visualized by <b>Cytoscape</b> software.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22307715">22307715</a> Identifying gene interaction networks.<ul><li>As a case study, we used <b>Cytoscape</b>, an open source and easy-to-use network visualization and analysis tool to first gather and visualize a small network. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22305245">22305245</a> A network-based analysis of traditional Chinese medicine cold and hot patterns in rheumatoid arthritis.<ul><li><b>Cytoscape</b> software was used to search the existing literature and databases for protein-protein interaction information for genes of interest that were identified from this analysis. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22292669">22292669</a> TranscriptomeBrowser 3.0: introducing a new compendium of molecular interactions and a new visualization tool for the study of gene regulatory networks.<ul><li>It is thus compatible with the network analysis software <b>Cytoscape</b> and with the Gene Interaction Network simulation software (GINsim). </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22287994">22287994</a> Associating Drugs, Targets and Clinical Outcomes into an Integrated Network Affords a New Platform for Computer-Aided Drug Repurposing.<ul><li>From the ADL corresponding to 988 unique drugs, the &quot;adverse reactions&quot; section was mapped onto 174 SE, then clustered via principal component analysis into a 5x5 self-organizing map that was integrated into a <b>Cytoscape</b> network of SE-D-T-CO. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22210867">22210867</a> CytoSEED: a <b>Cytoscape</b> plugin for viewing, manipulating and analyzing metabolic models created by the Model SEED.<ul><li>CytoSEED is a <b>Cytoscape</b> plugin for viewing, manipulating and analyzing metabolic models created using the Model SEED. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22155862">22155862</a> Computing graphlet signatures of network nodes and motifs in <b>Cytoscape</b> with GraphletCounter.<ul><li>For this purpose we have developed GraphletCounter, an open-source software tool for computing graphlet degree signatures that can operate on its own or as a plug-in to the network analysis environment <b>Cytoscape</b>. <li>GraphletCounter displays graphlet signatures for visual inspection within <b>Cytoscape</b>, and can output graphlet data for integration with larger workflows.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22155862">22155862</a> Computing graphlet signatures of network nodes and motifs in <b>Cytoscape</b> with GraphletCounter.<ul><li>It can be downloaded from the <b>Cytoscape</b> plugin repository, and is also available at http://sonmezsysbio.org/software/ graphletcounter.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22139938">22139938</a> Gene3D: a domain-based resource for comparative genomics, functional annotation and protein network analysis.<ul><li>Searching has been simplified with more sophisticated filtering of results, along with new tools based on <b>Cytoscape</b> Web, for visualizing protein-protein interaction networks, differences in domain composition between genomes and the taxonomic distribution of individual superfamilies.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22135418">22135418</a> Metscape 2 bioinformatics tool for the analysis and visualization of metabolomics and gene expression data.<ul><li>It can be downloaded from http://<b>cytoscape</b>.org or installed via <b>Cytoscape</b> plugin manager.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22135298">22135298</a> PhosphoSitePlus: a comprehensive resource for investigating the structure and function of experimentally determined post-translational modifications in man and mouse.<ul><li>Features designed to facilitate proteomic analyses include downloads of modification sites, kinase-substrate data sets, sequence logo generators, a <b>Cytoscape</b> plugin and BioPAX download to enable pathway visualization of the kinase-substrate interactions in PhosphoSitePlus®.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22070249">22070249</a> clusterMaker: a multi-algorithm clustering plugin for <b>Cytoscape</b>.<ul><li>Here we present clusterMaker, a <b>Cytoscape</b> plugin that implements several clustering algorithms and provides network, dendrogram, and heat map views of the results. <li>clusterMaker is the first <b>Cytoscape</b> plugin to implement such a wide variety of clustering algorithms and visualizations, including the only implementations of hierarchical clustering, dendrogram plus heat map visualization (tree view), k-means, k-medoid, SCPS, AutoSOME, and native (Java) MCL.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22070249">22070249</a> clusterMaker: a multi-algorithm clustering plugin for <b>Cytoscape</b>.<ul><li><b>Cytoscape</b> session files for all three scenarios are provided in the Additional Files section.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22070249">22070249</a> clusterMaker: a multi-algorithm clustering plugin for <b>Cytoscape</b>.<ul><li>The <b>Cytoscape</b> plugin clusterMaker provides a number of clustering algorithms and visualizations that can be used independently or in combination for analysis and visualization of biological data sets, and for confirming or generating hypotheses about biological function. <li>clusterMaker is available via the <b>Cytoscape</b> plugin manager.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22032859">22032859</a> gViz, a novel tool for the visualization of co-expression networks.<ul><li>Several programs (<b>Cytoscape</b>, yEd) have already been developed for network analysis; however, to our knowledge, there are no available GraphML compatible programs.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22028466">22028466</a> ANAT: a tool for constructing and analyzing functional protein networks.<ul><li>ANAT is an interactive software tool, implemented as a <b>Cytoscape</b> plug-in, for elucidating functional networks of proteins. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22027258">22027258</a> An in silico approach combined with in vivo experiments enables the identification of a new protein whose overexpression can compensate for specific respiratory defects in Saccharomyces cerevisiae.<ul><li>We collected all the physical protein-protein interactions (PPI) involving the core subunits, the supernumerary subunits and the assembly factors of complex III and used <b>Cytoscape</b> 2.6.3 and its plugins to construct a network. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22016407">22016407</a> PRINCIPLE: a tool for associating genes with diseases via network propagation.<ul><li>PRINCIPLE is a Java application implemented as a <b>Cytoscape</b> plug-in, based on a previously published algorithm, PRINCE. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22016407">22016407</a> PRINCIPLE: a tool for associating genes with diseases via network propagation.<ul><li>Implemented in Java, PRINCIPLE runs over <b>Cytoscape</b> 2.7 or newer versions. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21994228">21994228</a> Visualization and quality assessment of de novo genome assemblies.<ul><li>We present a method to evaluate genome scaffolding by aligning independently obtained transcriptome sequences to the genome and visually summarizing the alignments using the <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21994228">21994228</a> Visualization and quality assessment of de novo genome assemblies.<ul><li>Scripts that generate tables for visualization in <b>Cytoscape</b> from FASTA sequence and scaffolding information files are publicly available at https://github.com/ksanao/TGNet.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21975162">21975162</a> iRefScape. A <b>Cytoscape</b> plug-in for visualization and data mining of protein interaction data from iRefIndex.<ul><li>The iRefScape plug-in for the <b>Cytoscape</b> graphical viewer addresses these challenges. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21943367">21943367</a> iCTNet: a <b>Cytoscape</b> plugin to produce and analyze integrative complex traits networks.<ul><li>We describe a new open-source <b>Cytoscape</b> plugin named iCTNet (integrated Complex Traits Networks). </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21943367">21943367</a> iCTNet: a <b>Cytoscape</b> plugin to produce and analyze integrative complex traits networks.<ul><li>We argue this tool is a key instrument that facilitates systematic integration of disparate large-scale data through network visualization, ultimately allowing the identification of disease similarities and the design of novel therapeutic approaches.The online database and <b>Cytoscape</b> plugin are freely available for academic use at: http://www.cs.queensu.ca/ictnet.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21886098">21886098</a> Assembling global maps of cellular function through integrative analysis of physical and genetic networks.<ul><li>PanGIA (for physical and genetic interaction alignment) is a plug-in for the bioinformatics platform <b>Cytoscape</b>, designed to integrate physical and genetic interactions into hierarchical module maps. <li>This protocol begins with importing interaction networks into <b>Cytoscape</b>, followed by filtering and basic network visualization. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21877291">21877291</a> Displaying chemical information on a biological network using <b>Cytoscape</b>.<ul><li><b>Cytoscape</b> is an open-source software package that is widely used to integrate and visualize diverse data sets in biology. <li>Herein, two different protocols are explained in detail, with no prior knowledge of <b>Cytoscape</b> assumed, which demonstrate how to incorporate data from the ChEMBL database with either a gene-gene or a protein-protein interaction network. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21877289">21877289</a> Statistical analysis of dynamic transcriptional regulatory network structure.<ul><li>This is done by visualizing interactive data using a network analysis tool, such as <b>Cytoscape</b>, clustering DNA targets of the transcription factors based on their network topologies, and statistically analyzing each cluster based on its size and properties of its members. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21877285">21877285</a> Visualizing gene-set enrichment results using the <b>Cytoscape</b> plug-in enrichment map.<ul><li>&quot;Enrichment Map&quot; is a <b>Cytoscape</b> plug-in that helps overcome gene-set redundancy and aids in the interpretation of enrichment results. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21828085">21828085</a> NetDS: a <b>Cytoscape</b> plugin to analyze the robustness of dynamics and feedforward/feedback loop structures of biological networks.<ul><li>NetDS is a novel <b>Cytoscape</b> plugin that conveniently simulates dynamics related to robustness, and examines structural properties with respect to feedforward/feedback loops. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21756334">21756334</a> Construct and Compare Gene Coexpression Networks with DAPfinder and DAPview.<ul><li>Network results are easily displayed in <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21731702">21731702</a> Intuitive visualization and analysis of multi-omics data and application to Escherichia coli carbon metabolism.<ul><li>Graphical representations of complex biological networks can be generated using <b>Cytoscape</b> where all molecular and functional components could be explicitly represented using a set of dedicated symbols. <li>A <b>Cytoscape</b> plugin was developed to increase the possibilities of both multi-omic data representation and interpretation. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21712249">21712249</a> <b>Cytoscape</b>RPC: a plugin to create, modify and query <b>Cytoscape</b> networks from scripting languages.<ul><li><b>Cytoscape</b>RPC is a plugin for <b>Cytoscape</b> which allows users to create, query and modify <b>Cytoscape</b> networks from any programming language which supports XML-RPC. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21712249">21712249</a> <b>Cytoscape</b>RPC: a plugin to create, modify and query <b>Cytoscape</b> networks from scripting languages.<ul><li>Install through the <b>Cytoscape</b> plugin manager or visit the web page: http://wiki.nbic.nl/index.php/<b>Cytoscape</b>RPC for the user tutorial and download.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21622961">21622961</a> CoPub update: CoPub 5.0 a text mining system to answer biological questions.<ul><li>With CoPub 5.0, it is possible to create, annotate and analyze networks using the layout and highlight options of <b>Cytoscape</b> web, allowing for literature based systems biology. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21622212">21622212</a> Crosstalk between viruses and PML nuclear bodies: a network-based approach.<ul><li>Based on extensive literature data mining, we created a comprehensive PML-NB/virus crosstalk <b>Cytoscape</b> network, which groups not only the most common relations but also less well described findings. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21595958">21595958</a> Modeling microRNA-mRNA interactions using PLS regression in human colon cancer.<ul><li>Integrative mRNA:miRNA predictive networks from our analysis were constructed with the aid of <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511714">21511714</a> CytoModeler: a tool for bridging large-scale network analysis and dynamic quantitative modeling.<ul><li>CytoModeler is an open-source Java application based on the <b>Cytoscape</b> platform. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511714">21511714</a> CytoModeler: a tool for bridging large-scale network analysis and dynamic quantitative modeling.<ul><li>Implemented in Java, CytoModeler runs with <b>Cytoscape</b> 2.6 and 2.7. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21509594">21509594</a> Differential expression of a set of genes in follicular and classic variants of papillary thyroid carcinoma.<ul><li>Computational gene function analyses via <b>Cytoscape</b>, FuncBASE, and GeneMANIA led us to a functional network of 17 genes in which a core sub-network of five genes coexists. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21505034">21505034</a> Systems biology model repository for macrophage pathway simulation.<ul><li>MACPAK comprises a total of 24 009 entities and 12 774 processes and is described in the Cell System Markup Language (CSML), an XML format that runs on the Cell Illustrator platform and can be visualized with a customized <b>Cytoscape</b> for further analysis.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21493660">21493660</a> RING: networking interacting residues, evolutionary information and energetics in protein structures.<ul><li>Here we present a novel web server, RING, to construct physico-chemically valid RINs interactively from PDB files for subsequent visualization in the <b>Cytoscape</b> platform. <li>Different visualization styles are provided to facilitate visualization and standard plugins can be used to calculate topological parameters in <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21478195">21478195</a> CyClus3D: a <b>Cytoscape</b> plugin for clustering network motifs in integrated networks.<ul><li>We developed CyClus3D, a <b>Cytoscape</b> plugin for clustering composite three-node network motifs using a 3D spectral clustering algorithm.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21478195">21478195</a> CyClus3D: a <b>Cytoscape</b> plugin for clustering network motifs in integrated networks.<ul><li>Via the <b>Cytoscape</b> plugin manager or http://bioinformatics.psb.ugent.be/software/details/CyClus3D.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21473782">21473782</a> WordCloud: a <b>Cytoscape</b> plugin to create a visual semantic summary of networks.<ul><li>The WordCloud <b>Cytoscape</b> plugin generates a visual summary of these annotations by displaying them as a tag cloud, where more frequent words are displayed using a larger font size. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21414991">21414991</a> Customizable views on semantically integrated networks for systems biology.<ul><li>The technique was implemented in a plug-in for <b>Cytoscape</b>, called OndexView. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21414991">21414991</a> Customizable views on semantically integrated networks for systems biology.<ul><li>The Ondex yeast dataset and the OndexView plug-in for <b>Cytoscape</b> are accessible at http://bsu.ncl.ac.uk/ondexview.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21383412">21383412</a> Exact and approximate area-proportional circular Venn and Euler diagrams.<ul><li>An R function venneuler() is available as a package in CRAN and a plugin is available in <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21372810">21372810</a> Comprehensive cluster analysis with Transitivity Clustering.<ul><li>To facilitate this, Transitivity Clustering is accessible online and offers three user-friendly interfaces: a powerful stand-alone version, a web interface, and a collection of <b>Cytoscape</b> plug-ins. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21334471">21334471</a> A workflow for peptide-based proteomics in a poorly sequenced plant: a case study on the plasma membrane proteome of banana.<ul><li>The main steps of this workflow are (i) optimization of the peptide separation, (ii) performing de novo sequencing to allow a sequence homology search and (iii) visualization of identified peptide-protein associations using <b>Cytoscape</b> to remove redundancy and wrongly assigned peptides, based on species-specific information. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21300700">21300700</a> BioGRID REST Service, BiogridPlugin2 and BioGRID WebGraph: new tools for access to interaction data at BioGRID.<ul><li>We also describe two visualization tools that interface with the REST service, the BiogridPlugin2 for <b>Cytoscape</b> and the BioGRID WebGraph.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21296752">21296752</a> DDN: a caBIG® analytical tool for differential network analysis.<ul><li>Besides the standalone Java application, we have also developed a <b>Cytoscape</b> plug-in, CytoDDN, to integrate network analysis and visualization seamlessly.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21278188">21278188</a> PiNGO: a <b>Cytoscape</b> plugin to find candidate genes in biological networks.<ul><li>PiNGO is implemented as a plugin for <b>Cytoscape</b>, a popular network visualization platform.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21278188">21278188</a> PiNGO: a <b>Cytoscape</b> plugin to find candidate genes in biological networks.<ul><li>PiNGO is distributed as an open-source Java package under the GNU General Public License (http://www.gnu.org/), and can be downloaded via the <b>Cytoscape</b> plugin manager. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21257609">21257609</a> OrthoNets: simultaneous visual analysis of orthologs and their interaction neighborhoods across different organisms.<ul><li>We present OrthoNets, a <b>Cytoscape</b> plugin that displays protein-protein interaction (PPI) networks from two organisms simultaneously, highlighting orthology relationships and aggregating several types of biomedical annotations. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21217125">21217125</a> ATTED-II updates: condition-specific gene coexpression to extend coexpression analyses and applications to a broad range of flowering plants.<ul><li>We also developed an interactive visualization system, using the <b>Cytoscape</b> web system, to improve the network representation. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21179244">21179244</a> The PluriNetWork: an electronic representation of the network underlying pluripotency in mouse, and its applications.<ul><li>We have assembled a network of 574 molecular interactions, stimulations and inhibitions, based on a collection of research data from 177 publications until June 2010, involving 274 mouse genes/proteins, all in a standard electronic format, enabling analyses by readily available software such as <b>Cytoscape</b> and its plugins. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21156070">21156070</a> HIVBrainSeqDB: a database of annotated HIV envelope sequences from brain and other anatomical sites.<ul><li>Using <b>Cytoscape</b>, we visualized relationships between studies, patients and sequences, illustrating interconnections between studies and the varying depth of sequencing, patient number, and tissue representation across studies. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21149340">21149340</a> <b>Cytoscape</b> 2.8: new features for data integration and network visualization.<ul><li><b>Cytoscape</b> is a popular bioinformatics package for biological network visualization and data integration. <li>Attribute Equations provide <b>Cytoscape</b> with spreadsheet-like functionality in which the value of an attribute is computed dynamically as a function of other attributes and network properties.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21149340">21149340</a> <b>Cytoscape</b> 2.8: new features for data integration and network visualization.<ul><li><b>Cytoscape</b> is a desktop Java application released under the Library Gnu Public License (LGPL). </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21123224">21123224</a> GLay: community structure analysis of biological networks.<ul><li>GLay provides <b>Cytoscape</b> users an assorted collection of versatile community structure algorithms and graph layout functions for network clustering and structured visualization. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21121031">21121031</a> Dynamic, multi-level network models of clinical trials.<ul><li>We constructed networks of diseases and interventions for visualization and analysis using <b>Cytoscape</b> software. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21118823">21118823</a> Improving the quality of protein similarity network clustering algorithms using the network edge weight distribution.<ul><li>Python code for implementing the automated threshold selection heuristic, together with the datasets used in our analysis, are available at http://www.rbvi.ucsf.edu/Research/<b>cytoscape</b>/threshold_scripts.zip.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21118483">21118483</a> ExprEssence--revealing the essence of differential experimental data in the context of an interaction/regulation net-work.<ul><li>To address this challenge, we developed a method, implemented as a <b>Cytoscape</b> plugin called ExprEssence. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21097782">21097782</a> DIMA 3.0: Domain Interaction Map.<ul><li>The DIMA Web site allows to calculate domain interaction networks either for a domain of interest or for entire organisms, and to explore them interactively using the Flash-based <b>Cytoscape</b> Web software.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21085593">21085593</a> Enrichment map: a network-based method for gene-set enrichment visualization and interpretation.<ul><li>Enrichment Map is implemented as a freely available and user friendly plug-in for the <b>Cytoscape</b> network visualization software (http://baderlab.org/Software/EnrichmentMap/).</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21063955">21063955</a> <b>Cytoscape</b>: software for visualization and analysis of biological networks.<ul><li>Utilization of visualization software is exemplified using the widely used <b>Cytoscape</b> tool. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20946599">20946599</a> HPIDB--a unified resource for host-pathogen interactions.<ul><li>The output is provided in a tab delimited text file format that is compatible with <b>Cytoscape</b>, an open source resource for PPI visualization. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20947524">20947524</a> OmicsAnalyzer: a <b>Cytoscape</b> plug-in suite for modeling omics data.<ul><li>OmicsAnalyzer is a <b>Cytoscape</b> plug-in for visual omics-based network analysis that (i) integrates hetero-omics data for one or more species; (ii) performs statistical tests on the integrated datasets; and (iii) visualizes results in a network context.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20947524">20947524</a> OmicsAnalyzer: a <b>Cytoscape</b> plug-in suite for modeling omics data.<ul><li>Implemented in Java, OmicsAnalyzer runs with <b>Cytoscape</b> 2.6 and 2.7. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20926419">20926419</a> GeneMANIA <b>Cytoscape</b> plugin: fast gene function predictions on the desktop.<ul><li>The GeneMANIA <b>Cytoscape</b> plugin brings fast gene function prediction capabilities to the desktop. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20926419">20926419</a> GeneMANIA <b>Cytoscape</b> plugin: fast gene function predictions on the desktop.<ul><li>The GeneMANIA <b>Cytoscape</b> plugin is implemented in Java and is freely available at http://www.genemania.org/plugin/.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20861033">20861033</a> SpotXplore: a <b>Cytoscape</b> plugin for visual exploration of hotspot expression in gene regulatory networks.<ul><li>SpotXplore is a plugin for <b>Cytoscape</b> for extraction and visualization of differentially expressed subnetworks (hotspots) from gene networks. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20861032">20861032</a> DisGeNET: a <b>Cytoscape</b> plugin to visualize, integrate, search and analyze gene-disease networks.<ul><li>DisGeNET is a plugin for <b>Cytoscape</b> to query and analyze human gene-disease networks. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20861032">20861032</a> DisGeNET: a <b>Cytoscape</b> plugin to visualize, integrate, search and analyze gene-disease networks.<ul><li>DisGeNET is compatible with <b>Cytoscape</b> 2.6.3 and 2.7.0, please visit http://ibi.imim.es/DisGeNET/DisGeNETweb.html for installation guide, user tutorial and download.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20852260">20852260</a> MatrixDB, the extracellular matrix interaction database.<ul><li>MatrixDB allows researchers to perform customized queries and to build tissue- and disease-specific interaction networks that can be visualized and analyzed with <b>Cytoscape</b> or Medusa.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20851835">20851835</a> Protein-protein interaction and pathway databases, a graphical review.<ul><li>The graphical representation is made available in cooperation with the team maintaining www.pathguide.org and can be accessed at http://www.pathguide.org/interactions.php in a new <b>Cytoscape</b> web implementation. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20847220">20847220</a> Evidence mining and novelty assessment of protein-protein interactions with the ConsensusPathDB plugin for <b>Cytoscape</b>.<ul><li>To facilitate the task of evidence mining and novelty assessment of protein-protein interactions, we have developed a <b>Cytoscape</b> plugin that automatically mines publication references, database references, interaction detection method descriptions and pathway annotation for a user-supplied network of interactions. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20847220">20847220</a> Evidence mining and novelty assessment of protein-protein interactions with the ConsensusPathDB plugin for <b>Cytoscape</b>.<ul><li>The ConsensusPathDB plugin for <b>Cytoscape</b> (version 2.7.0 or later) can be installed within <b>Cytoscape</b> on a major operating system (Windows, Mac OS, Unix/Linux) with Sun Java 1.5 or later installed through <b>Cytoscape</b>'s Plugin manager (category 'Network and Attribute I/O'). </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20813661">20813661</a> [Analysis and verification of the interaction network of differentially expressed genes in invasive bladder cancer].<ul><li>The interaction data were imported into <b>Cytoscape</b> 2.6.2 software for screening the central nodes of the network. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20813661">20813661</a> [Analysis and verification of the interaction network of differentially expressed genes in invasive bladder cancer].<ul><li>Twenty-six nodes involved in several signal pathways were confirmed by <b>Cytoscape</b> as the central nodes of the network, among which UBE2C, VEGF, TGFBR2, and CAV1 nodes were verified by real-time RT-PCR as the genes with maximal differential expressions between the bladder cancer and normal tissues, and the 2(-delta delta Ct) of these genes were 9.45, 4.17, 0.13 and 0.18 (GAPHD as the internal control), respectively.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20810536">20810536</a> Large-scale proteome investigation in wild relatives (A, B, and D genomes) of wheat.<ul><li>Here, we reported genome-specific protein interaction network using <b>Cytoscape</b> software, which provides further insight into the molecular functions and mechanism of biochemical pathways. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20801912">20801912</a> Ontology- and graph-based similarity assessment in biological networks.<ul><li>We present SimTrek, a <b>Cytoscape</b> plugin for ontology-based similarity assessment in biological networks.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20719001">20719001</a> Chaperone expression profiles correlate with distinct physiological states of Plasmodium falciparum in malaria patients.<ul><li>Chaperone networks operational in the distinct physiological clusters in clinical malaria parasites were constructed using <b>cytoscape</b> by utilizing their clinical expression profiles.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20684769">20684769</a> Open source tool for prediction of genome wide protein-protein interaction network based on ortholog information.<ul><li>Results from our predictions can be visualized using the <b>Cytoscape</b> visualization software, and can be piped to downstream processing algorithms. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20664907">20664907</a> Factor XIII-A is involved in the regulation of gene expression in alternatively activated human macrophages.<ul><li>Functional clustering of the target genes carried out using <b>Cytoscape</b>/BiNGO and Ingenuity Pathways Analysis programs showed that, in the absence of FXIII-A, the most prominent differences are related to immune functions and to wound response. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20656902">20656902</a> <b>Cytoscape</b> Web: an interactive web-based network browser.<ul><li><b>Cytoscape</b> Web is a web-based network visualization tool-modeled after <b>Cytoscape</b>-which is open source, interactive, customizable and easily integrated into web sites. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20656902">20656902</a> <b>Cytoscape</b> Web: an interactive web-based network browser.<ul><li><b>Cytoscape</b> Web is implemented in Flex/ActionScript with a JavaScript API and is freely available at http://<b>cytoscape</b>web.<b>cytoscape</b>.org/.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20650822">20650822</a> A survey of current software for network analysis in molecular biology.<ul><li>The three major software packages for network analysis, <b>CYTOSCAPE</b>, INGENUITY and PATHWAY STUDIO, and their associated databases, are compared in detail. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20644733">20644733</a> Jerarca: efficient analysis of complex networks using hierarchical clustering.<ul><li>Outputs compatible with the phylogenetic software MEGA and the <b>Cytoscape</b> package are generated, allowing the results to be easily visualized.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20644733">20644733</a> Jerarca: efficient analysis of complex networks using hierarchical clustering.<ul><li>THE FOUR MAIN ADVANTAGES OF JERARCA IN RESPECT TO UVCLUSTER ARE: 1) Improved speed of a novel UVCluster algorithm; 2) Additional, alternative strategies to perform iterative hierarchical clustering; 3) Automatic evaluation of the hierarchical trees to obtain optimal partitions; and, 4) Outputs compatible with popular software such as MEGA and <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20594316">20594316</a> Flexible network reconstruction from relational databases with <b>Cytoscape</b> and CytoSQL.<ul><li>Molecular interaction networks can be efficiently studied using network visualization software such as <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20594316">20594316</a> Flexible network reconstruction from relational databases with <b>Cytoscape</b> and CytoSQL.<ul><li>A new <b>Cytoscape</b> plugin 'CytoSQL' is developed to connect <b>Cytoscape</b> to any relational database. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20594316">20594316</a> Flexible network reconstruction from relational databases with <b>Cytoscape</b> and CytoSQL.<ul><li>CytoSQL offers a unified approach to let <b>Cytoscape</b> interact with relational databases. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22081592">22081592</a> Fluxviz - <b>Cytoscape</b> plug-in for visualization of flux distributions in networks.<ul><li>We present Flux Viz, an open-source <b>Cytoscape</b> plug-in for the visualization of flux distributions in molecular interaction networks. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20513647">20513647</a> AltAnalyze and DomainGraph: analyzing and visualizing exon expression data.<ul><li>Here, we present our new software workflow composed of the open-source application AltAnalyze and the <b>Cytoscape</b> plugin DomainGraph. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20507894">20507894</a> Integrative platform to translate gene sets to networks.<ul><li>Moksiskaan networks can be downloaded to <b>Cytoscape</b> for further analysis. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20507893">20507893</a> MODEVO: exploring modularity and evolution of protein interaction networks.<ul><li>We developed two independent <b>Cytoscape</b> plugins that can be cooperatively used to map evolving protein interaction networks at the module level. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20507893">20507893</a> MODEVO: exploring modularity and evolution of protein interaction networks.<ul><li>The plugins are also available from the <b>Cytoscape</b> plugin repository.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20500839">20500839</a> Reconstruction of the yeast protein-protein interaction network involved in nutrient sensing and global metabolic regulation.<ul><li>The annotated information is readily available utilizing the functionalities of network modelling tools such as <b>Cytoscape</b> and CellDesigner.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20495000">20495000</a> FuncBase: a resource for quantitative gene function annotation.<ul><li>For example, a custom <b>Cytoscape</b> viewer shows functional linkage graphs relevant to the gene or function of interest. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20484371">20484371</a> ToppCluster: a multiple gene list feature analyzer for comparative enrichment clustering and network-based dissection of biological systems.<ul><li>ToppCluster provides users with choices of tabular outputs, hierarchical clustering and heatmap generation, or the ability to interactively select features from the functional enrichment matrix to be transformed into XGMML or GEXF network format documents for use in <b>Cytoscape</b> or Gephi applications, respectively. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20308225">20308225</a> Arsenic exposure perturbs epithelial-mesenchymal cell transition and gene expression in a collagen gel assay.<ul><li><b>Cytoscape</b> analysis of the microarray data using the chicken interactome identified four clusters of altered genes based on published relationships and pathways. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20230615">20230615</a> Characterizing regulatory path motifs in integrated networks using perturbational data.<ul><li>We introduce Pathicular http://bioinformatics.psb.ugent.be/software/details/Pathicular, a <b>Cytoscape</b> plugin for studying the cellular response to perturbations of transcription factors by integrating perturbational expression data with transcriptional, protein-protein and phosphorylation networks. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20222723">20222723</a> Quantitative analysis of HGF and EGF-dependent phosphotyrosine signaling networks.<ul><li>Using available protein-interaction data the identified proteins have been assembled into a highly connected network that can be visualized using the <b>Cytoscape</b> tool.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20214776">20214776</a> SCPS: a fast implementation of a spectral method for detecting protein families on a genome-wide scale.<ul><li>Besides the spectral method, SCPS also implements connected component analysis and hierarchical clustering, it integrates TribeMCL, it provides different cluster quality tools, it can extract human-readable protein descriptions using GI numbers from NCBI, it interfaces with external tools such as BLAST and <b>Cytoscape</b>, and it can produce publication-quality graphical representations of the clusters obtained, thus constituting a comprehensive and effective tool for practical research in computational biology. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20122237">20122237</a> NeMo: Network Module identification in <b>Cytoscape</b>.<ul><li>An implementation of our method is freely available for download as a plugin to <b>Cytoscape</b> through our website and through <b>Cytoscape</b> itself.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20172945">20172945</a> DASS-GUI: a user interface for identification and analysis of significant patterns in non-sequential data.<ul><li>Different export options allow easy usage of additional tools such as <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20163717">20163717</a> BisoGenet: a new tool for gene network building, visualization and analysis.<ul><li>The client tier is a <b>Cytoscape</b> plugin, which manages user input, communication with the Web Service, visualization and analysis of the resulting network.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20163717">20163717</a> BisoGenet: a new tool for gene network building, visualization and analysis.<ul><li>This feature facilitates the integration of our software with other tools available in the <b>Cytoscape</b> platform. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20139469">20139469</a> Metscape: a <b>Cytoscape</b> plug-in for visualizing and interpreting metabolomic data in the context of human metabolic networks.<ul><li>Metscape is a plug-in for <b>Cytoscape</b>, used to visualize and interpret metabolomic data in the context of human metabolic networks. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20139469">20139469</a> Metscape: a <b>Cytoscape</b> plug-in for visualizing and interpreting metabolomic data in the context of human metabolic networks.<ul><li>Metscape can be installed from within <b>Cytoscape</b> 2.6.x under 'Network and Attribute I/O' category. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20117829">20117829</a> Low malignant potential tumors with micropapillary features are molecularly similar to low-grade serous carcinoma of the ovary.<ul><li>Integration of differentially expressed genes into the protein interaction database <b>CytoScape</b> highlighted gene products in the MAPK pathway as differentially regulated between LMP and LGSC. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20109149">20109149</a> PlasmoID: A P. falciparum protein information discovery tool.<ul><li>The graphical user interface is designed as a plug-in to <b>Cytoscape</b>, an open-source network visualization tool. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20105306">20105306</a> Biana: a software framework for compiling biological interactions and analyzing networks.<ul><li>BIANA is also provided as a plugin for <b>Cytoscape</b>, which allows users to visualize and interactively manage the data. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20094647">20094647</a> The use of network analyses for elucidating mechanisms in cardiovascular disease.<ul><li>Data from 47 microarrays were examined using a combination of Bioconductor modules and the <b>Cytoscape</b> resource with several associated plugins to analyze the transcriptomics data and create a combined gene association and correlation network of atherosclerosis. <li>The utility and functionality of network analysis and the different <b>Cytoscape</b> plugins employed are discussed. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20053712">20053712</a> CORNET: a user-friendly tool for data mining and integration.<ul><li>Moreover, networks and associated evidence are visualized in <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19914922">19914922</a> PerturbationAnalyzer: a tool for investigating the effects of concentration perturbation on protein interaction networks.<ul><li>In order to facilitate this type of study, PerturbationAnalyzer, which is an open source plugin for <b>Cytoscape</b>, has been developed. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19914922">19914922</a> PerturbationAnalyzer: a tool for investigating the effects of concentration perturbation on protein interaction networks.<ul><li>PerturbationAnalyzer can be accessed from the <b>Cytoscape</b> web site http://www.<b>cytoscape</b>.org/plugins/index.php or http://biotech.bmi.ac.cn/PerturbationAnalyzer.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19887508">19887508</a> SimBoolNet--a <b>Cytoscape</b> plugin for dynamic simulation of signaling networks.<ul><li>SimBoolNet is an open source <b>Cytoscape</b> plugin that simulates the dynamics of signaling transduction using Boolean networks. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19877594">19877594</a> Finding key members in compound libraries by analyzing networks of molecules assembled by structural similarity.<ul><li>In the frame of the study, a Jchem plug-in has been developed to the <b>Cytoscape</b> network visualization software, which makes the visual observation of molecular networks more convenient. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19758470">19758470</a> From SNPs to pathways: integration of functional effect of sequence variations on models of cell signalling pathways.<ul><li>We generated attribute files containing phenotypic and genotypic annotations to the nodes of biological networks, which can be imported into network visualization tools such as <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19729372">19729372</a> Analyzing biological network parameters with CentiScaPe.<ul><li>In order to enrich experimental data with network topological parameters, we have developed the <b>Cytoscape</b> plug-in CentiScaPe. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19729372">19729372</a> Analyzing biological network parameters with CentiScaPe.<ul><li>CentiScaPe can be downloaded via the <b>Cytoscape</b> web site: http://chianti.ucsd.edu/cyto_web/plugins/index.php. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19723303">19723303</a> Construction of a polycystic ovarian syndrome (PCOS) pathway based on the interactions of PCOS-related proteins retrieved from bibliomic data.<ul><li><b>Cytoscape</b> 2.6.1 was used to build a network of protein and then to analyze it. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19654116">19654116</a> Genoscape: a <b>Cytoscape</b> plug-in to automate the retrieval and integration of gene expression data and molecular networks.<ul><li>Genoscape is an open-source <b>Cytoscape</b> plug-in that visually integrates gene expression data sets from GenoScript, a transcriptomic database, and KEGG pathways into <b>Cytoscape</b> networks. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19629191">19629191</a> Layered functional network analysis of gene expression in human heart failure.<ul><li>Four <b>Cytoscape</b> plugins including BionetBuilder, NetworkAnalyzer, Cerebral and GenePro were used to establish the layered PPI network, which was based upon validated subcellular protein localization data retrieved from the HRPD and Entrez Gene databases. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19607657">19607657</a> BioNetBuilder2.0: bringing systems biology to chicken and other model organisms.<ul><li>Systems Biology research tools, such as <b>Cytoscape</b>, have greatly extended the reach of genomic research. <li>BioNetBuilder is an open-source client-server <b>Cytoscape</b> plugin that automatically integrates molecular interactions from all major public interaction databases and serves them directly to the user's <b>Cytoscape</b> environment. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19607657">19607657</a> BioNetBuilder2.0: bringing systems biology to chicken and other model organisms.<ul><li>Using BioNetBuilder, we constructed a chicken interactome possessing 72,000 interactions among 8,140 genes directly in the <b>Cytoscape</b> environment. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19607657">19607657</a> BioNetBuilder2.0: bringing systems biology to chicken and other model organisms.<ul><li>BioNetBuilder 2.0, the tools for maintaining its data bases, standard operating procedures for creating local copies of its back-end data bases, as well as all of the Gaggle and <b>Cytoscape</b> codes required, are open-source and freely available at http://err.bio.nyu.edu/<b>cytoscape</b>/bionetbuilder/.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19597788">19597788</a> <b>Cytoscape</b>: a community-based framework for network modeling.<ul><li><b>Cytoscape</b> is a general network visualization, data integration, and analysis software package. <li><b>Cytoscape</b>'s flexibility has encouraged many users to adopt it and adapt it to their own research by using the plugin framework offered to specialize data analysis, data integration, or visualization. <li>This community of users and developers has worked together since <b>Cytoscape</b>'s initial release to improve the basic project through contributions to the core code and public offerings of plugin modules. <li>It also describes the development of a plugin used to investigate a particular research question in systems biology and walks through an example analysis using <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19470162">19470162</a> BioQuali <b>Cytoscape</b> plugin: analysing the global consistency of regulatory networks.<ul><li>We have developed a plugin for the <b>Cytoscape</b> environment, designed to facilitate automatic reasoning on regulatory networks. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19470162">19470162</a> BioQuali <b>Cytoscape</b> plugin: analysing the global consistency of regulatory networks.<ul><li>The BioQuali plugin facilitates in silico exploration of large-scale regulatory networks by combining the user-friendly tools of the <b>Cytoscape</b> environment with high-performance automatic reasoning algorithms. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19381531">19381531</a> Biological Network Inference and analysis using SEBINI and CABIN.<ul><li>The collection of edges in a public database, along with the confidence held in each edge (if available), can be fed into CABIN as one &quot;evidence network,&quot; using the <b>Cytoscape</b> SIF file format. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19348639">19348639</a> A network inference workflow applied to virulence-related processes in Salmonella typhimurium.<ul><li>The associated analysis pipeline automatically stores the inferred edges from the CLR runs within SEBINI and, upon request, transfers the inferred edges into either <b>Cytoscape</b> or the plug-in Collective Analysis of Biological Interaction Networks (CABIN) tool for further postanalysis of the inferred regulatory edges. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19273534">19273534</a> Construction and application of a protein and genetic interaction network (yeast interactome).<ul><li><b>Cytoscape</b> is a bioinformatic data analysis and visualization platform that is well-suited to the analysis of gene expression data. <li>These data were formatted and imported into <b>Cytoscape</b> using semi-automated methods, including Linux-based scripts, that simplified the process while minimizing the introduction of processing errors. <li>Using <b>Cytoscape</b>, we illustrate the use of this interactome through the analysis of expression data from a recent yeast diauxic shift experiment. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19237447">19237447</a> ClueGO: a <b>Cytoscape</b> plug-in to decipher functionally grouped gene ontology and pathway annotation networks.<ul><li>We have developed ClueGO, an easy to use <b>Cytoscape</b> plug-in that strongly improves biological interpretation of large lists of genes. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19193966">19193966</a> The interactions of the largest subunit of RNA polymerase II with other cellular proteins: a bioinformatic approach.<ul><li>Here, we have analyzed bioinformatically the interactions of Rpb1p (the largest subunit of RNA Polymerase II) with other proteins in yeast, using <b>Cytoscape</b> software and Biogrid/Biomart database. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19144180">19144180</a> BioMart--biological queries made easy.<ul><li>BioMart's capabilities are extended by integration with several widely used software packages such as BioConductor, DAS, Galaxy, <b>Cytoscape</b>, Taverna. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19102773">19102773</a> RPPAML/RIMS: a metadata format and an information management system for reverse phase protein arrays.<ul><li>The RIMS platform was designed to interoperate with other data analysis and data visualization tools such as <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18978014">18978014</a> Michigan molecular interactions r2: from interacting proteins to pathways.<ul><li>New functionality includes extension from proteins to genes and to pathways; identification of highlighted sentences in source publications; seamless two-way linkage with <b>Cytoscape</b>; query facilities based on MeSH/GO terms and other concepts; approximate graph matching to find relevant pathways; support for querying in bulk; and a user focus-group driven interface design. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18840285">18840285</a> DroID: the Drosophila Interactions Database, a comprehensive resource for annotated gene and protein interactions.<ul><li>All data in DroID are freely available and can be searched, explored, and downloaded through three different interfaces, including a text based web site, a Java applet with dynamic graphing capabilities (IM Browser), and a <b>Cytoscape</b> plug-in. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18819078">18819078</a> Exploring biological networks with <b>Cytoscape</b> software.<ul><li><b>Cytoscape</b> is a free software package for visualizing, modeling, and analyzing molecular and genetic interaction networks. <li>This unit explains how to use <b>Cytoscape</b> to load and navigate biological network information and view mRNA expression profiles and other functional genomics and proteomics data in the context of the network obtained for genes of interest. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18812364">18812364</a> Integrating and annotating the interactome using the MiMI plugin for <b>cytoscape</b>.<ul><li>Here, we describe a <b>Cytoscape</b> plugin that retrieves interaction and annotation data from MiMI and links out to multiple data sources and tools. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18812364">18812364</a> Integrating and annotating the interactome using the MiMI plugin for <b>cytoscape</b>.<ul><li>The plugin is also preloaded when <b>Cytoscape</b> is launched using Java WebStart at http://mimi.ncibi.org by querying a gene and clicking 'View in MiMI Plugin for <b>Cytoscape</b>' link.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18779234">18779234</a> OmicsViz: <b>Cytoscape</b> plug-in for visualizing omics data across species.<ul><li>OmicsViz is a <b>Cytoscape</b> plug-in for mapping and visualizing large-scale omics datasets across species, including those with many-to-many mappings between homologs. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18769717">18769717</a> Network inference algorithms elucidate Nrf2 regulation of mouse lung oxidative stress.<ul><li>Inferred networks were analyzed, compared, and integrated using the Collective Analysis of Biological Interaction Networks (CABIN) plug-in of <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18755046">18755046</a> FERN - a Java framework for stochastic simulation and evaluation of reaction networks.<ul><li>To illustrate how FERN can be easily integrated into other systems biology applications, plugins to <b>Cytoscape</b> and CellDesigner are included. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18710874">18710874</a> Integrating expression data with domain interaction networks.<ul><li>We have developed the <b>Cytoscape</b> plugin DomainGraph that enables the visualization and detailed study of domain-domain interactions forming protein interaction networks. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18678589">18678589</a> VistaClara: an expression browser plug-in for <b>Cytoscape</b>.<ul><li>VistaClara is a plug-in for <b>Cytoscape</b> which provides a more flexible means to visualize gene and protein expression within a network context. <li>This extended browser permits a variety of display options and interactions not currently available in <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18658180">18658180</a> DrugViz: a <b>Cytoscape</b> plugin for visualizing and analyzing small molecule drugs in biological networks.<ul><li>DrugViz is a <b>Cytoscape</b> plugin that is designed to visualize and analyze small molecules within the framework of the interactome. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18653520">18653520</a> Cytoprophet: a <b>Cytoscape</b> plug-in for protein and domain interaction networks inference.<ul><li>It is implemented as a plug-in of <b>Cytoscape</b>, an open source software framework for analysis and visualization of molecular networks. <li>Cytoprophet was designed to take advantage of the visual capabilities of <b>Cytoscape</b> and be simple to use. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18445605">18445605</a> <b>Cytoscape</b> ESP: simple search of complex biological networks.<ul><li><b>Cytoscape</b> enhanced search plugin (ESP) enables searching complex biological networks on multiple attribute fields using logical operators and wildcards. <li>ESP is implemented as a <b>Cytoscape</b> plugin and complements existing search functions in the <b>Cytoscape</b> network visualization and analysis software, allowing users to easily identify nodes, edges and subgraphs of interest, even for very large networks. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18430989">18430989</a> NetAtlas: a <b>Cytoscape</b> plugin to examine signaling networks based on tissue gene expression.<ul><li>We have developed NetAtlas, an open-source, Java-based <b>Cytoscape</b> plugin for examining signaling networks in the context of tissue gene expression patterns. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18428770">18428770</a> Searching, viewing, and visualizing data in the Biomolecular Interaction Network Database (BIND).<ul><li>Furthermore, a protocol is provided for visualizing biomolecular interactions within BIND or for transferring this information to the visualization tools <b>Cytoscape</b> and Cn3D.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18426593">18426593</a> Linking <b>Cytoscape</b> and the corynebacterial reference database CoryneRegNet.<ul><li><b>Cytoscape</b> is an open-source software project whose aim is to provide graph-based visualization and analysis for biological networks. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18426593">18426593</a> Linking <b>Cytoscape</b> and the corynebacterial reference database CoryneRegNet.<ul><li>We introduce two novel plugins for the <b>Cytoscape</b> environment designed to enhance in silico studies of procaryotic transcriptional regulatory networks. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18426593">18426593</a> Linking <b>Cytoscape</b> and the corynebacterial reference database CoryneRegNet.<ul><li>coli, by combining the knowledge from the corynebacterial reference database with the graph analysis capabilities of <b>Cytoscape</b>, which is one of the most-widely used tools for biological network analyses.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20559448">20559448</a> Proteomics cataloging analysis of human expressed prostatic secretions reveals rich source of biomarker candidates.<ul><li>To help prioritize candidate biomarkers, we constructed a protein-protein interaction network of the EPS proteins using <b>Cytoscape</b> (www.<b>cytoscape</b>.org), and overlaid the expression level changes from the Oncomine database onto the network.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20134061">20134061</a> Integrative visual analysis of the effects of alternative splicing on protein domain interaction networks.<ul><li>Therefore, we have developed the versatile <b>Cytoscape</b> plugin DomainGraph that allows for the visual analysis of protein domain interaction networks and their integration with exon expression data. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18024474">18024474</a> BiNoM: a <b>Cytoscape</b> plugin for manipulating and analyzing biological networks.<ul><li>BiNoM comes as a library and as a <b>Cytoscape</b> plugin which adds a rich set of operations to <b>Cytoscape</b> such as path and cycle analysis, clustering sub-networks, decomposition of network into modules, clipboard operations and others.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18021453">18021453</a> The Firegoose: two-way integration of diverse data from different bioinformatics web resources with desktop applications.<ul><li>As a component of the Gaggle integration framework, Firegoose can also exchange data with <b>Cytoscape</b>, the R statistical package, Multiexperiment Viewer (MeV), and several other popular desktop software tools. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18006545">18006545</a> Computing topological parameters of biological networks.<ul><li>In order to gain insight into the organization and structure of the resultant large complex networks formed by the interacting molecules, we have developed the versatile <b>Cytoscape</b> plugin NetworkAnalyzer. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18003642">18003642</a> MetaNetter: inference and visualization of high-resolution metabolomic networks.<ul><li>We present a <b>Cytoscape</b> plugin for the inference and visualization of networks from high-resolution mass spectrometry metabolomic data. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17953753">17953753</a> Genes involved in TGF beta1-driven epithelial-mesenchymal transition of renal epithelial cells are topologically related in the human interactome map.<ul><li>By mapping the differentially expressed genes in the Human Interactome Map using <b>Cytoscape</b> software, we identified a single scale-free network consisting of 2630 interacting proteins and containing 449 differentially expressed proteins. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17947979">17947979</a> Integration of biological networks and gene expression data using <b>Cytoscape</b>.<ul><li><b>Cytoscape</b> is a free software package for visualizing, modeling and analyzing molecular and genetic interaction networks. <li>These steps provide a broad sample of the types of analyses performed by <b>Cytoscape</b>.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17720984">17720984</a> Tools for visually exploring biological networks.<ul><li>Many tools exist for visually exploring biological networks including well-known examples such as <b>Cytoscape</b>, VisANT, Pathway Studio and Patika. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17644818">17644818</a> APID2NET: unified interactome graphic analyzer.<ul><li>APID2NET is a new tool that works with <b>Cytoscape</b> to allow surfing unified interactome data by querying APID server (http://bioinfow.dep.usal.es/apid/) to provide interactive analysis of protein-protein interaction (PPI) networks. <li>The tool provides interactive graphical representation of the networks with all <b>Cytoscape</b> capabilities, plus new automatic tools to find concurrent functional and structural attributes along all protein pairs in a network.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17623706">17623706</a> structureViz: linking <b>Cytoscape</b> and UCSF Chimera.<ul><li>structureViz is a <b>Cytoscape</b> plug-in that links the visualization of biological networks provided by <b>Cytoscape</b> with the visualization and analysis of macromolecular structures and sequences provided by UCSF Chimera. <li>When a structure is opened, structureViz provides an alternative interface to Chimera: the <b>Cytoscape</b> Molecular Structure Navigator. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17553858">17553858</a> The MiSink Plugin: <b>Cytoscape</b> as a graphical interface to the Database of Interacting Proteins.<ul><li>The MiSink Plugin converts <b>Cytoscape</b>, an open-source bioinformatics platform for network visualization, to a graphical interface for the database of interacting proteins (DIP: http://dip.doe-mbi.ucla.edu). </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17500038">17500038</a> CABIN: collective analysis of biological interaction networks.<ul><li>CABIN has been written in Java and is available as a plugin for <b>Cytoscape</b>--an open source network visualization tool.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17392333">17392333</a> Cyclone: java-based querying and computing with Pathway/Genome databases.<ul><li>In addition, Cyclone interfaces easily with Java software including the Eclipse IDE for HQL edition, the Jung API for graph algorithms or <b>Cytoscape</b> for graph visualization.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17309895">17309895</a> Cerebral: a <b>Cytoscape</b> plugin for layout of and interaction with biological networks using subcellular localization annotation.<ul><li>Cerebral (Cell Region-Based Rendering And Layout) is an open-source Java plugin for the <b>Cytoscape</b> biomolecular interaction viewer. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17277332">17277332</a> NetMatch: a <b>Cytoscape</b> plugin for searching biological networks.<ul><li>NetMatch is a <b>Cytoscape</b> plugin which allows searching biological networks for subcomponents matching a given query. <li><b>Cytoscape</b> is a bioinformatics software platform for the visualization and analysis of biological networks.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17172291">17172291</a> The rice kinase database. A phylogenomic database for the rice kinome.<ul><li>Interaction maps are easily accessed through links and displayed using <b>Cytoscape</b>, an open source software platform. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17169983">17169983</a> T1DBase: integration and presentation of complex data for type 1 diabetes research.<ul><li>(2002) Genome Res., 10, 1599-1610] displays genes and other features in their genomic context, and <b>Cytoscape</b> [Shannon et al. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17138585">17138585</a> BioNetBuilder: automatic integration of biological networks.<ul><li>BioNetBuilder is an open-source client-server <b>Cytoscape</b> plugin that offers a user-friendly interface to create biological networks integrated from several databases. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17127678">17127678</a> GOlorize: a <b>Cytoscape</b> plug-in for network visualization with Gene Ontology-based layout and coloring.<ul><li>It can be used in conjunction with BiNGO plug-in to <b>Cytoscape</b>, which finds the GO categories over-represented in a given network. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17101041">17101041</a> cPath: open source software for collecting, storing, and querying biological pathways.<ul><li>cPath makes it easy to aggregate custom pathway data sets available in standard exchange formats from multiple databases, present pathway data to biologists via a customizable web interface, and export pathway data via a web service to third-party software, such as <b>Cytoscape</b>, for visualization and analysis. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17021160">17021160</a> The tYNA platform for comparative interactomics: a web tool for managing, comparing and mining multiple networks.<ul><li>tYNA can also be accessed from the <b>Cytoscape</b> software using a plugin.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16939803">16939803</a> Visualizing networks.<ul><li>It also presents a practical introduction to Osprey and <b>Cytoscape</b>, two software platforms that are powerful tools for visualizing, integrating, and manipulating networks.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16921162">16921162</a> GenePro: a <b>Cytoscape</b> plug-in for advanced visualization and analysis of interaction networks.<ul><li>The public-domain open software environment <b>Cytoscape</b> has been developed with the goal of facilitating the design and development of such software tools by the scientific community.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16921162">16921162</a> GenePro: a <b>Cytoscape</b> plug-in for advanced visualization and analysis of interaction networks.<ul><li>We present GenePro, a plugin to <b>Cytoscape</b> featuring a set of versatile tools that greatly facilitates the visualization and analysis of protein networks derived from high-throughput interactions data and the validation of various methods for parsing these networks into meaningful functional modules.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16756676">16756676</a> SBEAMS-Microarray: database software supporting genomic expression analyses for systems biology.<ul><li>It is interoperable with the <b>Cytoscape</b> network integration, visualization, analysis, and modeling software platform.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16569235">16569235</a> The Gaggle: an open-source software system for integrating bioinformatics software and data sources.<ul><li>We have integrated diverse databases (for example, KEGG, BioCyc, String) and software (<b>Cytoscape</b>, DataMatrixViewer, R statistical environment, and TIGR Microarray Expression Viewer). </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16204107">16204107</a> Decomposing protein networks into domain-domain interactions.<ul><li>We have designed a <b>Cytoscape</b> plugin that decomposes interacting proteins into their respective domains and computes a putative network of corresponding domain-domain interactions. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=15972284">15972284</a> BiNGO: a <b>Cytoscape</b> plugin to assess overrepresentation of gene ontology categories in biological networks.<ul><li>BiNGO can be used either on a list of genes, pasted as text, or interactively on subgraphs of biological networks visualized in <b>Cytoscape</b>. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=15608258">15608258</a> T1DBase, a community web-based resource for type 1 diabetes research.<ul><li>Tools on the site include the GBrowse genome browser, site-wide context dependent search, Connect-the-Dots for connecting gene and other identifiers from multiple data sources, <b>Cytoscape</b> for visualizing and analyzing biological networks, and the GESTALT workbench for genome annotation. </ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=15287976">15287976</a> System-based proteomic analysis of the interferon response in human liver cells.<ul><li>To analyze this proteomics dataset, we utilized several systems-biology data-mining tools, including Gene Ontology via the GoMiner program and the <b>Cytoscape</b> bioinformatics platform.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=15287976">15287976</a> System-based proteomic analysis of the interferon response in human liver cells.<ul><li>Integration of the quantitative proteomics with global protein interaction data using the <b>Cytoscape</b> platform led to the identification of several novel and liver-specific key regulatory components of the IFN response, which may be important in regulating the interplay between HCV, interferon and the host response to virus infection.</ul></p><p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=14597658">14597658</a> <b>Cytoscape</b>: a software environment for integrated models of biomolecular interaction networks.<ul><li><b>Cytoscape</b> is an open source software project for integrating biomolecular interaction networks with high-throughput expression data and other molecular states into a unified conceptual framework. <li><b>Cytoscape</b>'s software Core provides basic functionality to layout and query the network; to visually integrate the network with expression profiles, phenotypes, and other molecular states; and to link the network to databases of functional annotations. <li>Several case studies of <b>Cytoscape</b> plug-ins are surveyed, including a search for interaction pathways correlating with changes in gene expression, a study of protein complexes involved in cellular recovery to DNA damage, inference of a combined physical/functional interaction network for Halobacterium, and an interface to detailed stochastic/kinetic gene regulatory models.</ul></p></body>
